diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/AGATHBED.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/AGATHBED.msg
new file mode 100644
index 000000000..0c4c20f20
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/AGATHBED.msg
@@ -0,0 +1,3 @@
+{100}{}{The bed looks neatly made despite its tattered condition.}
+{101}{}{You see a tattered bed.}
+{102}{}{Agatha would not be happy if you messed up her bed.}
