diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/AIRLOCK.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/AIRLOCK.msg
new file mode 100644
index 000000000..85fe2caf6
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/AIRLOCK.msg
@@ -0,0 +1 @@
+{100}{}{This Vault looks particularly scavenged. You think to yourself that nothing of value could possibly remain.}
