diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/BLAST.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/BLAST.msg
new file mode 100644
index 000000000..758a63bfa
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/BLAST.msg
@@ -0,0 +1,4 @@
+{100}{}{The explosion closes off the tunnel with a mighty blast!}
+{101}{}{You drop the explosives and they go off!}
+{102}{}{The explosion just makes the hole larger. It is impossible to close it now.}
+{103}{}{You earn 10 experience for successful use of explosives.}
