diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/BLKROM.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/BLKROM.msg
new file mode 100644
index 000000000..650904d88
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/BLKROM.msg
@@ -0,0 +1,2 @@
+{100}{}{The rubble has piled up high in the entrance way. It is too heavy to move. You wonder if the Water Chip might have once been there. If it was, it is no longer available.}
+{101}{}{Big, heavy rocks are blocking the way to the room. You strain to move a rock, but it fails to budge.}
