diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/BOBDOUG.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/BOBDOUG.msg
new file mode 100644
index 000000000..77f2b5f75
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/BOBDOUG.msg
@@ -0,0 +1,4 @@
+{100}{}{You see a Knight.}
+{101}{}{The learning room's right over there if that's what you're looking for.}
+{102}{}{Are you lost? The learning room is right over there.}
+{103}{}{The medical lab is right around the corner.}
