diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/BONEBODY.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/BONEBODY.msg
new file mode 100644
index 000000000..f2fe65be1
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/BONEBODY.msg
@@ -0,0 +1,3 @@
+{100}{}{Whatever attacked this person cleaned them down to the bones.}
+{101}{}{The wind has bleached these bones clean.}
+{102}{}{These corpses appear to have been here for quite some time.}
