diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/BOOKCASE.MSG b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/BOOKCASE.MSG
index 6cbbf7ebc..c935ad555 100644
--- a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/BOOKCASE.MSG
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/BOOKCASE.MSG
@@ -6,3 +6,7 @@
 {105}{}{You fumble with the bookcase for a moment but fail to accomplish anything.}
 {200}{}{The bookshelf is filled with religious and philosophical books.}
 {201}{}{The latch will not budge.}
+
+# Fo1in2
+{202}{}{ finds a hidden latch.}
+{203}{}{ fumbles with the bookcase for a moment but fails to accomplish anything.}
\ No newline at end of file
