diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/BOSASIST.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/BOSASIST.msg
new file mode 100644
index 000000000..1034c7952
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/BOSASIST.msg
@@ -0,0 +1,9 @@
+{100}{}{You see the Doctor's assistant.}
+{179}{}{You should see the Doctor.}
+{180}{}{She's the Doctor, not me.}
+{181}{}{I'm only in training.}
+{182}{}{I can't help you, officially.}
+{183}{}{Lorri's the Doctor.}
+{184}{}{You have to ask the Doctor.}
+{185}{}{Don't worry, she's just a kidder.}
+{186}{}{She just likes to tease, don't worry.}
