diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/BOXGUARD.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/BOXGUARD.msg
new file mode 100644
index 000000000..0c1ea71af
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/BOXGUARD.msg
@@ -0,0 +1,8 @@
+{100}{}{Holy crap! Kill it!}
+{101}{}{Shoot that beast before it eats him!}
+{102}{}{Guns!}
+{103}{}{Get him out of there!}
+{104}{}{Don't go inside the ring, chump.}
+{105}{}{Hey, where do you think you're going?}
+{106}{}{Talk to Gustofer if you want to bet.}
+{107}{}{Boxing matches at 2:00 every third day.}
