diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/BVENT.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/BVENT.msg
new file mode 100644
index 000000000..84c60f32f
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/BVENT.msg
@@ -0,0 +1,2 @@
+{100}{}{This is where the main entrance to the Vault should be.}
+{101}{}{The collapse looks natural. You cannot find any evidence of foul play or explosives.}
