diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/BVLIB.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/BVLIB.msg
new file mode 100644
index 000000000..c489bf662
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/BVLIB.msg
@@ -0,0 +1,2 @@
+{100}{}{This is the library. You've seen all this equipment before in your Vault library.}
+{101}{}{There looks to be little of value left in this room.}
