diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/CALLY.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/CALLY.msg
new file mode 100644
index 000000000..a8882a467
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/CALLY.msg
@@ -0,0 +1,2 @@
+{100}{}{You see: Cally.}
+{101}{}{You're good enough to sneak in here, but dumb enough to show yourself? Too bad!}
