diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/CAVEWALL.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/CAVEWALL.msg
new file mode 100644
index 000000000..d68dc17df
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/CAVEWALL.msg
@@ -0,0 +1,2 @@
+{100}{}{This portion of the cave seems a bit weaker.}
+{101}{}{The rock here looks very fragmented, and will most likely crumble under extreme conditions.}
