diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/CHARBODY.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/CHARBODY.msg
new file mode 100644
index 000000000..78940fd97
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/CHARBODY.msg
@@ -0,0 +1,5 @@
+{100}{}{This person must have been very close to the blast.}
+{101}{}{This body is so charred, you cannot even guess what it once was.}
+{102}{}{The shifting winds have caused some of the ashes to drift away.}
+{103}{}{The nuclear blast must have done this damage.}
+{104}{}{Ashes fall through your fingers.}
