diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/CHILDRN2.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/CHILDRN2.msg
new file mode 100644
index 000000000..4efbbb62f
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/CHILDRN2.msg
@@ -0,0 +1,9 @@
+{100}{}{Time left: }
+{101}{}{ seconds.}
+{102}{}{This is where you make your mark on the world. (A small pile of ash.)}
+{104}{}{Time left: }
+{105}{}{ seconds.}
+{106}{}{This is where you make your mark on the world. (A small pile of ash.)}
+{108}{}{Time left: }
+{109}{}{ seconds.}
+{110}{}{This is where you make your mark on the world. (A small pile of ash.)}
