diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/COMPUTER.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/COMPUTER.msg
new file mode 100644
index 000000000..11b2dcd9d
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/COMPUTER.msg
@@ -0,0 +1,5 @@
+{100}{}{This is a badly worn-out desk with a dusty, cracked computer seated precariously atop it.}
+{101}{}{The computer looks badly damaged and unrepairable.}
+{102}{}{Many circuits are fried from an EMP burst.}
+{103}{}{It doesn't work.}
+{104}{}{You slice your hands on the case.}
