diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/COOK.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/COOK.msg
new file mode 100644
index 000000000..09c0a52fe
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/COOK.msg
@@ -0,0 +1,5 @@
+{100}{}{You see a busy cook preparing a meal.}
+{101}{}{I'm too busy to talk right now. I've got to get this meal ready!}
+{102}{}{Hmmm! That smells really good. I bet it tastes great!}
+{103}{}{I'm too busy to talk right now. I've got to get this meal ready!}
+{104}{}{Why, thank you. I'm sure you will enjoy it. Thanks for the compliment. I'm going to tell all my friends about you.}
