diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/COOKTABL.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/COOKTABL.msg
new file mode 100644
index 000000000..fc037bcf9
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/COOKTABL.msg
@@ -0,0 +1,2 @@
+{100}{}{The table appears to be full of items the cooks are using to make a meal.}
+{101}{}{The food is not cooked yet. You'll get sick if you eat it raw.}
