diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/COOKSHEL.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/COOKSHEL.msg
new file mode 100644
index 000000000..0153dd4cd
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/COOKSHEL.msg
@@ -0,0 +1,2 @@
+{100}{}{The shelf contains bottles of various items.}
+{101}{}{The cooks are extremely watchful of their store of food and prevent you from taking anything.}
