diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/CRASHRM.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/CRASHRM.msg
new file mode 100644
index 000000000..44bb2544d
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/CRASHRM.msg
@@ -0,0 +1,4 @@
+{100}{}{You are in your room. Do you want to rest here until tomorrow morning?}
+{101}{}{Yes}
+{102}{}{No}
+{103}{}{You awake refreshed.}
