diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/CRVNTEAM.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/CRVNTEAM.msg
index d33997ece..c017be325 100644
--- a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/CRVNTEAM.msg
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/CRVNTEAM.msg
@@ -1,16 +1,16 @@
-{100}{}{We're heading back to the Hub.}
-{101}{}{Are you goin' to return with us?}
-{102}{}{Ready to go.}
-{103}{}{Hope I have enough bullets left...}
-{104}{}{I love this job!}
-{105}{}{Let's go!}
-{106}{}{Get movin'!}
-{107}{}{We killed 'em all!}
-{108}{}{That was easy!}
-{109}{}{Don't talk, move yer ass!}
-{110}{}{Are you ready to go?}
-{111}{}{I'm not paid to talk.}
-{112}{}{Nasty freakin' place, man!}
-{113}{}{I'm not paid to talk.}
-{114}{}{Mind your own business, would ya?}
-{115}{}{Mind your own business, would ya?}
+{100}{}{We're heading back to the Hub.}
+{101}{}{Are you goin' to return with us?}
+{102}{}{Ready to go.}
+{103}{}{Hope I have enough bullets left...}
+{104}{}{I love this job!}
+{105}{}{Let's go!}
+{106}{}{Get movin'!}
+{107}{}{We killed 'em all!}
+{108}{}{That was easy!}
+{109}{}{Don't talk, move yer ass!}
+{110}{}{Are you ready to go?}
+{111}{}{I'm not paid to talk.}
+{112}{}{Nasty freakin' place, man!}
+{113}{}{I'm not paid to talk.}
+{114}{}{Mind your own business, would ya?}
+{115}{}{Mind your own business, would ya?}
