diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/DANWIFE.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/DANWIFE.msg
new file mode 100644
index 000000000..c02de3854
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/DANWIFE.msg
@@ -0,0 +1,6 @@
+{100}{}{You see: Dan's wife.}
+{101}{}{I'm not supposed to talk to anyone. Please see my husband.}
+{102}{}{I'm sorry, you need to speak with my husband.}
+{103}{}{You horrible man, why did you do this? You killed my Danny!}
+{104}{}{You horrible woman, why did you do this? You killed my Danny!}
+{105}{}{You see: Dan's crying wife.}
