diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/DEAD.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/DEAD.msg
new file mode 100644
index 000000000..83ca4d5f4
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/DEAD.msg
@@ -0,0 +1,5 @@
+{100}{}{Dead bodies...}
+{101}{}{They were recently killed by some sort of animal.}
+{102}{}{They were recently killed.}
+{103}{}{They might be sleeping.}
+{104}{}{They're dead.}
