diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/DEMOLEX.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/DEMOLEX.msg
index 74f050681..cd800c00e 100644
--- a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/DEMOLEX.msg
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/DEMOLEX.msg
@@ -1,4 +1,4 @@
-{100}{}{You see Lex.}
+{100}{}{You see: Lex.}
 {101}{}{You see a loser.}
 {102}{}{[More]}
 {103}{}{[Done]}
