diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/DEMOCOMP.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/DEMOCOMP.msg
new file mode 100644
index 000000000..e5af00964
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/DEMOCOMP.msg
@@ -0,0 +1,27 @@
+{100}{}{This is a badly worn-out desk with a dusty, cracked computer
+ seated precariously atop it.}
+{101}{}{You plug a loose chip back in and surprisingly the monitor lights up.}
+{102}{}{Many circuits are fried from an EMP burst.}
+{103}{}{It doesn't work.}
+{104}{}{You slice your hands on the case.}
+{105}{}{Wyndoze 99 booting up.
+                                          U:\ > }
+{106}{}{dir}
+{107}{}{exit}
+{108}{}{FALLOUT    <DIR>          10-20-99   1:04p
+                            0 bytes in 1 file     0 bytes allocated
+                            4,379,255 bytes free}
+{109}{}{cd fallout}
+{110}{}{exit}
+{111}{}{U:\ > }
+{112}{}{dir}
+{113}{}{fallout.exe}
+{114}{}{exit}
+{115}{}{.          <DIR>               10-20-99   1:04p
+                            ..         <DIR>               10-20-99   1:04p
+                            fallout.exe     1384432   10-20-99   1:04p
+                            1,384,432 bytes in 3 files 1,384,482 bytes allocated}
+{116}{}{[more]}
+{117}{}{ }
+{118}{}{Hmm. This is a great game!}
+{119}{}{You gain 100 experience points for playing Fallout.}
