diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/DEMOPHIL.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/DEMOPHIL.msg
index fa581e3fb..d3ebb24cd 100644
--- a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/DEMOPHIL.msg
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/DEMOPHIL.msg
@@ -1,5 +1,5 @@
 {100}{}{You see a peasant.}
-{101}{}{You see Phil.}
+{101}{}{You see: Phil.}
 {102}{}{It ain't my dog!}
 {103}{}{Damn right, shoot that beast!}
 {104}{}{I wouldn't trust that beast, if I was you.
