diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/DEMOGEN.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/DEMOGEN.msg
new file mode 100644
index 000000000..150e8eb69
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/DEMOGEN.msg
@@ -0,0 +1,25 @@
+{100}{}{The power generator has been sabotaged. It no longer functions.}
+{101}{}{You see a power generator. It is the only source of power in Scrapheap.}
+{102}{}{You clip some wires on the transformer. The power generator no longer functions.}
+{103}{}{The cutters are of no further use.}
+{104}{}{That item is useless on the generator.}
+{105}{}{You notice nothing in particular about the generator.}
+{106}{}{The generator itself looks fairly tamper-proof; however you notice a set of wires between the transformer
+ and the generator.}
+{107}{}{You cannot see anything special about this large generator.}
+{108}{}{You can determine nothing further about the generator.}
+{109}{}{The wires could be repaired, but it may take a while, and there is no guarantee of success.}
+{110}{}{You cannot see anything special about the damaged generator.}
+{111}{}{You are unable to do any further repair.}
+{112}{}{There is no need to repair the power generator. It is functioning perfectly.}
+{113}{}{You are unable to repair the functioning generator.}
+{114}{}{It takes several hours, but you fix the power generator. With the power restored, and the gangs gone,
+ the townspeople are grateful. They give you some valuable information. Your task in Scrapheap is done. The
+ mission to find the water chip still awaits.}
+{115}{}{You cannot fix the generator. The people of Scrapheap will lead a poorer life without their power;
+ but with the gangs gone, at least they will have a life. Your task in Scrapheap is done. The mission to
+ find the water chip still awaits.}
+{116}{}{Further tampering with the generator yields nothing.}
+{117}{}{That skill has no effect on the generator.}
+{118}{}{The generator is already running.}
+{119}{}{You are unable to start the generator, because it is broken.}
