diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/DEMODOOR.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/DEMODOOR.msg
new file mode 100644
index 000000000..5d673128c
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/DEMODOOR.msg
@@ -0,0 +1,9 @@
+{100}{}{You successfully unlock the refrigerator.}
+{101}{}{You are having trouble with this lock.}
+{102}{}{You see a refrigerator. It is closed and locked.}
+{103}{}{You see a refrigerator. It is open.}
+{104}{}{You see a refrigerator. It is closed.}
+{105}{}{The door is locked.}
+{106}{}{You see a refrigerator.}
+{107}{}{It is already unlocked.}
+{108}{}{That does nothing.}
