diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/DIRTNAP.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/DIRTNAP.msg
new file mode 100644
index 000000000..dd77a1a5b
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/DIRTNAP.msg
@@ -0,0 +1,4 @@
+{100}{}{You see: Glowing Ghoul.}
+{101}{}{A dirt nap will be given if alterations are made. Last one here tried.}
+{102}{}{The flow ends. So does your life.}
+{103}{}{Thank you for fixing the water pump.}
