diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/DOCLOCKR.MSG b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/DOCLOCKR.MSG
index aee53750c..53b556e4b 100644
--- a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/DOCLOCKR.MSG
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/DOCLOCKR.MSG
@@ -9,4 +9,7 @@
 {108}{}{You fail to bypass the lock on the locker.}
 {109}{}{The locker is already unlocked.}
 {110}{}{While tampering with the lock, you trip up the tumblers. There's no bypassing this lock now.}
-{111}{}{You notice that there is a crude explosive device on the locker.}
\ No newline at end of file
+{111}{}{You notice that there is a crude explosive device on the locker.}
+
+# Fo1in2
+{200}{}{ unlocks the locker.}
