diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/DOORCODE.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/DOORCODE.msg
new file mode 100644
index 000000000..b38029e93
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/DOORCODE.msg
@@ -0,0 +1,2 @@
+{100}{}{Although this tape is damaged beyond use, there is some writing scratched on the plastic: KODE 010597}
+{101}{}{You see a holodisk in very poor condition.}
