diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/DOOR.MSG b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/DOOR.MSG
index ca8f63697..251672ead 100644
--- a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/DOOR.MSG
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/DOOR.MSG
@@ -10,6 +10,10 @@
 {109}{}{The door is already unlocked.}
 {110}{}{The lock is beyond your ability to pick.}
 
+# Fo1in2
+{111}{}{The lock is beyond }
+{112}{}{'s ability to pick.}
+
 # look_at_proc
 {120}{}{You see a metal door.}
 
