diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/ECHUNTRB.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/ECHUNTRB.msg
new file mode 100644
index 000000000..bde908080
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/ECHUNTRB.msg
@@ -0,0 +1,10 @@
+# Encounter - Gecko hunter, the Son
+
+{100}{}{You see: Young Hunter.}
+{101}{}{You see: Cairook, the Hunter.}
+{102}{}{A young tribesman.}
+
+# Float
+{200}{}{FATHER!}
+
+{1000}{}{He doesn't talk to you.}
