diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/ELECTRAP.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/ELECTRAP.msg
new file mode 100644
index 000000000..3437dde19
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/ELECTRAP.msg
@@ -0,0 +1,2 @@
+{100}{}{The section of the floor adjacent to you is electrified.}
+{101}{}{A section of the electrified floor has been disabled.}
