diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/ENCQUACK.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/ENCQUACK.msg
index 0dea6d4aa..3f309389b 100644
--- a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/ENCQUACK.msg
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/ENCQUACK.msg
@@ -1,42 +1,41 @@
-# Random Encounter - Old Quack
-
-{100}{}{You see a quack.}
-{101}{}{You see: Dr. Cleave O. Buker.}
-{102}{}{You see a snake oil merchant.}
-
-# dumb dude
-{103}{}{You see a great medicine man.} 
-
-# map enter
-{110}{}{You see a lone traveler with a colorfully painted wagon in the distance.}
-
-# floats
-{150}{}{I'll show you, you inbred codhopper!}
-
-# Talk
-{1000}{}{Look there, a traveler! Welcome, welcome! Do YOU suffer from syphilis? Bullet wounds? Scalpings? Swellings? Scurvy? Maybe rabies or... 
-pesky snake bites? Then step closer, Dr. Cleave O. Buker has the right medication for one such as yours!}
-{1001}{}{Ah! Welcome, welcome! Step closer, Dr. Cleave O. Buker has the right medication for one such as yours!}
-	{1010}{}{This sounds like exactly what I need right now! Show me what you have to sell.}
-	{1011}{}{Uh, WOW!}
-	{1020}{}{OK, but you better not be wasting my time.}
-	{1030}{}{No thanks and goodbye.}
-	{1031}{}{Uh, na.}
-
-{1100}{}{A man who knows what's good for him! Take a look at this one-hundred percent original and two-hundred percent effective miracle cure, 
-and I guarante you won't be able to get this any cheaper elsewhere around!}
-{1101}{}{A woman who knows what's good for her! Take a look at this one-hundred percent original and two-hundred percent effective miracle cure, 
-and I guarante you won't be able to get this any cheaper elsewhere around!}
-{1102}{}{Take a look and be amazed!}
-	{1110}{}{[MORE]}
-
-{1200}{}{This top quality product, used ALL AROUND THE WORLD, against rheumatism! Liver complaints! Indicestion and especially ALL blood diseases! ...}
-	{1210}{}{You know what? Show me your product again.}
-	{1211}{}{Uhm...}
-	{1220}{}{Alright, I get it. Goodbye.}
-	{1221}{}{Bye!}
-	
-{1300}{}{Wasting your time?! Dear good sir! Taking this medicine will give you plenty more time!}
-{1301}{}{Wasting your time?! Dear good madam! Taking this medicine will give you plenty more time!}
-	{1310}{}{[MORE]}
-	
\ No newline at end of file
+# Random Encounter - Old Quack
+
+{100}{}{You see a quack.}
+{101}{}{You see: Dr. Cleave O. Buker.}
+{102}{}{You see a snake oil merchant.}
+
+# dumb dude
+{103}{}{You see a great medicine man.}
+
+# map enter
+{110}{}{You see a lone traveler with a colorfully painted wagon in the distance.}
+
+# floats
+{150}{}{I'll show you, you inbred codhopper!}
+
+# Talk
+{1000}{}{Look there, a traveler! Welcome, welcome! Do YOU suffer from syphilis? Bullet wounds? Scalpings? Swellings? Scurvy? Maybe rabies or... 
+pesky snake bites? Then step closer, Dr. Cleave O. Buker has the right medication for one such as yours!}
+{1001}{}{Ah! Welcome, welcome! Step closer, Dr. Cleave O. Buker has the right medication for one such as yours!}
+	{1010}{}{This sounds like exactly what I need right now! Show me what you have to sell.}
+	{1011}{}{Uh, WOW!}
+	{1020}{}{OK, but you better not be wasting my time.}
+	{1030}{}{No thanks and goodbye.}
+	{1031}{}{Uh, na.}
+
+{1100}{}{A man who knows what's good for him! Take a look at this one-hundred percent original and two-hundred percent effective miracle cure, 
+and I guarante you won't be able to get this any cheaper elsewhere around!}
+{1101}{}{A woman who knows what's good for her! Take a look at this one-hundred percent original and two-hundred percent effective miracle cure, 
+and I guarante you won't be able to get this any cheaper elsewhere around!}
+{1102}{}{Take a look and be amazed!}
+	{1110}{}{[MORE]}
+
+{1200}{}{This top quality product, used ALL AROUND THE WORLD, against rheumatism! Liver complaints! Indicestion and especially ALL blood diseases! ...}
+	{1210}{}{You know what? Show me your product again.}
+	{1211}{}{Uhm...}
+	{1220}{}{Alright, I get it. Goodbye.}
+	{1221}{}{Bye!}
+
+{1300}{}{Wasting your time?! Dear good sir! Taking this medicine will give you plenty more time!}
+{1301}{}{Wasting your time?! Dear good madam! Taking this medicine will give you plenty more time!}
+	{1310}{}{[MORE]}
