diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/FLARE.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/FLARE.msg
new file mode 100644
index 000000000..b6b4f9483
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/FLARE.msg
@@ -0,0 +1,6 @@
+{100}{}{You light the flare.}
+{101}{}{This flare has been expended.}
+{102}{}{The flare runs out of fuel and is extinguished.}
+{103}{}{You see a lit flare.}
+{104}{}{You see a spent flare.}
+{105}{}{You see a flare.}
