diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/FOOTPRNT.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/FOOTPRNT.msg
new file mode 100644
index 000000000..95c228205
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/FOOTPRNT.msg
@@ -0,0 +1 @@
+{100}{}{There seems to be a large amount of foot traffic in the area, possibly leading to the manhole cover.}
