diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/FORCCOMP.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/FORCCOMP.msg
new file mode 100644
index 000000000..d2c8b9df5
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/FORCCOMP.msg
@@ -0,0 +1,2 @@
+{100}{}{This computer is labeled: "Force Field Controls".}
+{101}{}{You manipulate the computer and a moment later it sparks once and then shuts down.}
