diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/FREEZER.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/FREEZER.msg
new file mode 100644
index 000000000..c660b7189
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/FREEZER.msg
@@ -0,0 +1,2 @@
+{100}{}{A heavy military refrigerator.}
+{101}{}{A faded paper on the freezer reads: "Federal Bureau of Investigation, Special Projects, Classified: Security Terminus."}
