diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/FRUIT.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/FRUIT.msg
new file mode 100644
index 000000000..24b857a38
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/FRUIT.msg
@@ -0,0 +1 @@
+{100}{}{Although the fruit here appears abnormal, you recognize it and recall that it has healing properties.}
