diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/GAMBLER.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/GAMBLER.msg
new file mode 100644
index 000000000..404db7873
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/GAMBLER.msg
@@ -0,0 +1,46 @@
+{100}{}{You see a seedy-looking Mexican man with slick hair.}
+{101}{}{You see: Hernandez, the rascally gambler.}
+{102}{}{Oye, muchacho, allow me to introduce myself: I am Hernandez, a man of mucho gusto! You look like you are new here. Please, allow me to be your informant about the casino.}
+{103}{}{Can you tell me about the casino?}
+{104}{}{No, thanks.}
+{105}{}{Bloo?}
+{106}{}{But of course! This fine establishment is run by the famous Gizmo. A true visionary and a very wealthy man.}
+{107}{}{Thanks.}
+{108}{}{You say Gizmo is a visionary. What's his vision?}
+{109}{}{What are the odds like on the games here?}
+{110}{}{My friend, I hate to impose, but perhaps you could assist me. I have developed a sure-fire system because I have discovered how Gizmo runs his tables, but I do not have the caps to play.}
+{111}{}{ (You get the feeling that he's trying to scam you.)}
+{112}{}{Perhaps you can spare a few caps? I guarantee success.}
+{113}{}{Sure, sounds good.}
+{114}{}{No, thanks.}
+{115}{}{I think you're lying. You're trying to scam me.}
+{116}{}{Sorry, mi amigo. Hey, let me show you a neat game... }
+{117}{}{Thank you very much, mi amigo! (He accepts 25 caps from you.) In a few days, I shall be able to pay you back, with interest. My system is slow, you see, it requires much patience, but it will most certainly reap great profits!}
+{118}{}{Very well, but I guarantee success! If you want to get in on the riches just waiting, come back soon, or I will find another partner!}
+{119}{}{My friend, I see you are a man of much taste, as well. Perhaps a simple game of dice?}
+{120}{}{All right!}
+{121}{}{No, thanks, I'd rather try something else.}
+{122}{}{Gizmo had a dream of setting up a place where people could meet and enjoy themselves. He had the ambition to make it happen. The result is the casino you see here.}
+{123}{}{He gives people a chance to shrug off their cares in the broken world outside. And he brings money into Junktown!}
+{124}{}{Sounds like a good idea.}
+{125}{}{Life's hard enough. I don't trust the motives of someone who would encourage people to lose their money.}
+{126}{}{You obviously don't share Gizmo's brilliance. This place is a respite from the harshness of everyday life.}
+{127}{}{Well, Gizmo runs a pretty good house, although the games are, how shall we say, rather weighted toward the house. It's not like his dealers cheat, though. Unless you have a system, like me, you will need to be very clever.}
+{128}{}{Thanks.}
+{129}{}{}
+{130}{}{How is your system coming?}
+{131}{}{Nothing, thanks.}
+{132}{}{My system is developing slowly. I have living expenses and such, you know, but it is working!}
+{133}{}{That's great!}
+{134}{}{I think that you are a liar and a cheat. Where's my money?}
+{135}{}{So, you think you have stumbled onto me, eh? Well, you won't get anything from me!}
+
+#
+# TRANSLATION NOTE: The following two lines are identical in English, but they
+# differ in foreign translations. The first line is for the case of the
+# player-character masculine ("my friend" will be masculine); the second line is
+# for the case of the player-character feminine ("my friend" will be feminine).
+#
+
+{136}{}{Buenos dias, my friend! What can I do for you today?}
+{137}{}{Buenos dias, my friend! What can I do for you today?}
