diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/GASTRAP.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/GASTRAP.msg
new file mode 100644
index 000000000..736bbd464
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/GASTRAP.msg
@@ -0,0 +1,7 @@
+{100}{}{Uh-oh.}
+{101}{}{You set off a gas trap and take a point of damage.}
+{102}{}{You set off a gas trap and take }
+{103}{}{ points of damage.}
+{104}{}{You set off a gas trap!}
+
+{105}{}{ sets off a gas trap!}
