diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/GDOOR.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/GDOOR.msg
new file mode 100644
index 000000000..954b2abf7
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/GDOOR.msg
@@ -0,0 +1,6 @@
+{100}{}{You unlock the door.}
+{101}{}{You accidentally jam the lock on the door.}
+{102}{}{You cannot open this lock.}
+{103}{}{You were not able to pick the lock.}
+{104}{}{The door is locked.}
+{105}{}{The door is unlocked.}
