diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/GENCHAT.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/GENCHAT.msg
index 4202812bd..5601ae4e6 100644
--- a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/GENCHAT.msg
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/GENCHAT.msg
@@ -109,6 +109,14 @@
 {300}{}{Your Melee Damage is increased by 1 point.}
 {301}{}{% Damage Resistance.}
 
+# Fixt
+{194}{}{you move? in my WAY! *frown*}# Call "Step Aside" code
+{195}{}{go go, no need frend.}# Telling NPC to leave party
+{196}{}{we be frend?}# Request NPC to rejoin party
+
+# Message trying to steal from things like brahmin, rats, scorpions
+{211}{}{You cannot steal from this critter.}
+
 # Dehydration encounter
 {305}{}{Dehydrated}
 {306}{}{The dehydration effects have worn off.}
