diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/GENCHAT.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/GENCHAT.msg
new file mode 100644
index 000000000..4202812bd
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/GENCHAT.msg
@@ -0,0 +1,153 @@
+{100}{}{Thank you.}
+{101}{}{Goodbye.}
+{102}{}{Hello.}
+
+{103}{}{You earn }
+{104}{}{ experience.}
+
+{105}{}{Uh oh!}
+{106}{}{Click!}
+{107}{}{Oops!}
+{108}{}{Ouch!}
+{109}{}{See ya 'round.}
+{110}{}{Take it easy.}
+{111}{}{It's been fun.}
+{112}{}{Here you go.}
+{113}{}{Anything else?}
+{114}{}{Yes.}
+{115}{}{No.}
+
+{116}{}{You gain }
+
+{117}{}{ skill points in Small Guns.}
+{118}{}{ skill points in Big Guns.}
+{119}{}{ skill points in Energy Weapons.}
+{120}{}{ skill points in Unarmed Combat.}
+{121}{}{ skill points in Melee Weapons.}
+{122}{}{ skill points in Throwing.}
+{123}{}{ skill points in First Aid.}
+{124}{}{ skill points in Doctor.}
+{125}{}{ skill points in Sneak.}
+{126}{}{ skill points in Steal.}
+{127}{}{ skill points in Lockpick.}
+{128}{}{ skill points in Traps.}
+{129}{}{ skill points in Science.}
+{130}{}{ skill points in Repair.}
+{131}{}{ skill points in Speech.}
+{132}{}{ skill points in Barter.}
+{133}{}{ skill points in Gambling.}
+{134}{}{ skill points in Outdoorsman.}
+
+{135}{}{You gain 1 skill point in Small Guns.}
+{136}{}{You gain 1 skill point in Big Guns.}
+{137}{}{You gain 1 skill point in Energy Weapons.}
+{138}{}{You gain 1 skill point in Unarmed Combat.}
+{139}{}{You gain 1 skill point in Melee Weapons.}
+{140}{}{You gain 1 skill point in Throwing.}
+{141}{}{You gain 1 skill point in First Aid.}
+{142}{}{You gain 1 skill point in Doctor.}
+{143}{}{You gain 1 skill point in Sneak.}
+{144}{}{You gain 1 skill point in Steal.}
+{145}{}{You gain 1 skill point in Lockpick.}
+{146}{}{You gain 1 skill point in Traps.}
+{147}{}{You gain 1 skill point in Science.}
+{148}{}{You gain 1 skill point in Repair.}
+{149}{}{You gain 1 skill point in Speech.}
+{150}{}{You gain 1 skill point in Barter.}
+{151}{}{You gain 1 skill point in Gambling.}
+{152}{}{You gain 1 skill point in Outdoorsman.}
+
+{153}{}{Your Strength has increased by }
+{154}{}{Your Perception has increased by }
+{155}{}{Your Endurance has increased by }
+{156}{}{Your Charisma has increased by }
+{157}{}{Your Intelligence has increased by }
+{158}{}{Your Agility has increased by }
+{159}{}{Your Luck has increased by }
+{160}{}{points.}
+
+{161}{}{Your Strength has increased by 1 point.}
+{162}{}{Your Perception has increased by 1 point.}
+{163}{}{Your Endurance has increased by 1 point.}
+{164}{}{Your Charisma has increased by 1 point.}
+{165}{}{Your Intelligence has increased by 1 point.}
+{166}{}{Your Agility has increased by 1 point.}
+{167}{}{Your Luck has increased by 1 point.}
+
+{168}{}{Thanks for your help.}
+{169}{}{Thanks for nothing.}
+
+{170}{}{Aradesh is furious and refuses to talk to you any more.}
+
+{171}{}{I believe the Master wishes to speak with you.}
+{172}{}{You shall pay for your BLASPHEMY!}
+
+{173}{}{This robot seems to be inoperable. This is possibly due to the lack of power.}
+{174}{}{You see nothing special about this robot.}
+
+{175}{}{This Blade refuses to talk to you.}
+
+{176}{}{The suitcase is sealed and unopenable.}
+
+{177}{}{Please, we must hurry. We don't have time to speak.}
+{178}{}{Only Morpheus has the key to this door. I must leave now to let Nicole know what is going on here.}
+{179}{}{Please, I have to get away from here before they find out who I am.}
+
+{180}{}{You fail to steal anything and wonder if you got caught.}
+{181}{}{The Brotherhood members have caught you stealing.}
+
+{186}{}{You hear over the loudspeaker: "Attention! Intruder Alert."}
+{187}{}{The goo has slowed down the transmission of your keystrokes.}
+
+{200}{}{You see a horribly-damaged field emitter.}
+{201}{}{The emitter is too damaged to use.}
+{202}{}{The emitter is too damaged to repair.}
+{203}{}{You notice that the field is damaged enough to walk through.}
+
+# Fo1in2
+{190}{}{ activates Stealth Boy.}
+{300}{}{Your Melee Damage is increased by 1 point.}
+{301}{}{% Damage Resistance.}
+
+# Dehydration encounter
+{305}{}{Dehydrated}
+{306}{}{The dehydration effects have worn off.}
+
+##################################################
+##################################################
+## Additions by Sduibek for Fallout Fixt:
+##################################################
+##################################################
+
+{214}{}{The poison has left your system.}
+
+{400}{}{The blast greatly weakens this energy field. It should now be harmless to pass through.}
+
+#
+# For door to Brotherhood armory (weapons/armor stash on Lvl1)
+#
+{500}{}{The door mocks you defiantly by withstanding the blast, receiving only dents and scorch marks.}
+{501}{}{The door is concerned with its ability to take more punishment, as it creaks and buckles from the blast.}
+{502}{}{The door groans in agony as it is blown from its hinges, falling to the floor with a mighty crash.}
+#TRANSLATION NOTE: the spaces at end of 503 and beginning of 504 are required, do not remove them.
+{503}{}{You earn }
+{504}{}{ experience for successful use of explosives.}
+{505}{}{The blast doesn't quite destroy the door. You'll need one more.}
+
+#
+# For guards within Brotherhood armory (weapons/armor stash on Lvl1)
+#
+{601}{}{You shouldn't be in here, Initiate.}
+{602}{}{What are you doing in here?}
+{603}{}{Please leave. You shouldn't be here.}
+{604}{}{Hey, no Initiates.}
+{605}{}{Sorry, I shouldn't be talking to anyone. You know how it is.}
+{606}{}{This is not the library, my boy. If you want to browse, go to the library.}
+{607}{}{This is not the library, my girl. If you want to browse, please go to the library.}
+
+#
+# For endgame (when blowing up Vats, and Master's Vault. Both give the XP but didn't notify user previously.)
+#
+{700}{}{ experience!}
+{701}{}{ experience for destroying the Vats.}
+{702}{}{ experience for destroying The Master.}
