diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/GENERIC.MSG b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/GENERIC.MSG
index 07ffa076c..90b6b80ec 100644
--- a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/GENERIC.MSG
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/GENERIC.MSG
@@ -78,7 +78,7 @@
 
 {930}{}{Hoooooowl!}
 
-{950}{}{You take some time out to practice some moves. Your unarmed combat skills increase by 5%.}
+{950}{}{You take some time out to practice some moves. You gain 5 skill points in Unarmed Combat.}
 
 {951}{}{The fighting mole rat goes in for the death blow.}
 {952}{}{The match is over.}
