diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/GENGRDA.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/GENGRDA.msg
new file mode 100644
index 000000000..963a7df84
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/GENGRDA.msg
@@ -0,0 +1,6 @@
+{100}{}{Boy, it sure is hot.}
+{101}{}{I could use a drink.}
+{102}{}{These trips sure are long.}
+{103}{}{I hope we don't run into any raiders.}
+{104}{}{I really need the money from this job.}
+{105}{}{I hate this job.}
