diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/GENMERCB.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/GENMERCB.msg
new file mode 100644
index 000000000..97f68970b
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/GENMERCB.msg
@@ -0,0 +1,18 @@
+{101}{}{I do not need your services right now. Perhaps later.}
+{102}{}{Help! Save me!}
+{103}{}{You must die for what you've done!}
+{104}{}{Spare me! Please!}
+{105}{}{Are they dead yet? Maybe there's more of them!}
+
+#
+# TRANSLATION NOTE: The following three sets of lines are identical in English,
+# but they differ in foreign translations. The first line is for the case of the
+# player-character masculine ("my friend" will be masculine); the second line is
+# for the case of the player-character feminine ("my friend" will be feminine).
+#
+
+{106}{}{Your arrival was timely, my friend. Allow me to present you with a small token of my gratitude.}
+{107}{}{Your arrival was timely, my friend. Allow me to present you with a small token of my gratitude.}
+
+# Fo1in2
+{110}{}{The merchant thanks you for your help.}
