diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/GENMERCB.MSG b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/GENMERCB.MSG
index b8dc01711..1253c2bea 100644
--- a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/GENMERCB.MSG
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/GENMERCB.MSG
@@ -13,3 +13,6 @@
 
 {106}{}{Your arrival was timely, my friend. Allow me to present you with a small token of my gratitude.}
 {107}{}{Your arrival was timely, my friend. Allow me to present you with a small token of my gratitude.}
+
+# Fo1in2
+{110}{}{The merchant thanks you for your help.}
