diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/GHGENDED.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/GHGENDED.msg
new file mode 100644
index 000000000..238a2f20b
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/GHGENDED.msg
@@ -0,0 +1,5 @@
+{100}{}{You see a lifeless corpse laying on the ground.}
+{101}{}{An unfortunate soul lays here.}
+{102}{}{The remains of a mutated human rest upon the floor.}
+{103}{}{The feeding grounds for carrion.}
+{104}{}{It appears this person challenged Thanatos and lost.}
