diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/GHKICK.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/GHKICK.msg
new file mode 100644
index 000000000..4aeed44cd
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/GHKICK.msg
@@ -0,0 +1,2 @@
+{100}{}{You see a ghoul gang member.}
+{101}{}{You not supposed to be here! Man, I'm gonna have to kick your ass.}
