diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/GHRNDGRD.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/GHRNDGRD.msg
new file mode 100644
index 000000000..5a3e63aac
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/GHRNDGRD.msg
@@ -0,0 +1,4 @@
+{100}{}{You see a ghoul gang guard.}
+{101}{}{Go away.}
+{102}{}{I don't like your kind.}
+{103}{}{Leave now!}
