diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/GHZERO.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/GHZERO.msg
new file mode 100644
index 000000000..650f82127
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/GHZERO.msg
@@ -0,0 +1,2 @@
+{100}{}{You see a ghoul guard.}
+{101}{}{I'm on duty. If you want to see Set, you gotta do it inside.}
