diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/GLO1WEAP.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/GLO1WEAP.msg
new file mode 100644
index 000000000..4b76d558b
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/GLO1WEAP.msg
@@ -0,0 +1,7 @@
+{100}{}{These doors have been blasted by the nuclear warhead.}
+{101}{}{The door is welded too tight for you to open it.}
+
+# Fo1in2:
+{102}{}{With the help of the crowbar, you force open the door.}
+{103}{}{Even with the crowbar, you still aren't strong enough to force the door.}
+{104}{}{*Hrnnnnnn*}
