diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/GLO3SUPL.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/GLO3SUPL.msg
new file mode 100644
index 000000000..c04a7f702
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/GLO3SUPL.msg
@@ -0,0 +1,2 @@
+{100}{}{These doors have been blasted by the nuclear warhead.}
+{101}{}{The door is welded too tight for you to open it.}
