diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/GLO4SUPL.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/GLO4SUPL.msg
new file mode 100644
index 000000000..4d63c08f1
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/GLO4SUPL.msg
@@ -0,0 +1,8 @@
+{100}{}{The door is already unlocked.}
+{101}{}{You manage to pick the lock.}
+{102}{}{You can't seem to pick the lock.}
+{103}{}{The electronic security system shocks you!}
+{104}{}{You have tripped an alarm on the supply doors.}
+{105}{}{The door has a physical lock that will need to be picked.}
+{106}{}{The electronic security is already disarmed.}
+{107}{}{You disarm the electronic security.}
