diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/GLO4WEAP.MSG b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/GLO4WEAP.MSG
index cf7b969a3..75672d435 100644
--- a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/GLO4WEAP.MSG
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/GLO4WEAP.MSG
@@ -23,4 +23,9 @@
 {122}{}{ points of damage.}
 {123}{}{The door seems to be protected by an energy field. Perhaps you can de-activate it somehow.}
 {124}{}{You successfully unlock the door.}
-{125}{}{You failed to pick the lock on the door.}
\ No newline at end of file
+{125}{}{You failed to pick the lock on the door.}
+
+#Fo1in2
+{200}{}{ gets shocked by the security system for }
+{201}{}{ successfully unlocks the door.}
+{202}{}{ failed to pick the lock on the door.}
\ No newline at end of file
