diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/GLOBLDOR.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/GLOBLDOR.msg
new file mode 100644
index 000000000..feb3f15b8
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/GLOBLDOR.msg
@@ -0,0 +1,115 @@
+{100}{}{The frame of this security door has remnants of blue paint.}
+{101}{}{You get a mild shock from the electric field on the door and take a point of damage.}
+{102}{}{You get a strong shock from the electric field on the door. You sustain }
+{103}{}{ points of damage.}
+{104}{}{You force the door open.}
+{105}{}{The door is locked.}
+{106}{}{Authorization granted. Now disarming electric field.}
+{107}{}{You are not authorized to enter this room.}
+{108}{}{This item does not effect the security systems.}
+{109}{}{You pick the lock on the security door.}
+{110}{}{There seems to be some type of security measure on this door.}
+{111}{}{You failed to pick the lock on the door.}
+{112}{}{This skill has no effect on the door.}
+{113}{}{You disarm the electric field security system on the door.}
+{114}{}{You intensified the power of the electric field.}
+{115}{}{You fail to disarm the electric field.}
+{116}{}{You pick the lock on the door without setting off the security system.}
+{117}{}{You fail to pick the lock and take serious damage from the security system.}
+{118}{}{You fail to pick the lock and take damage from the security system.}
+{119}{}{You are shocked by the security system and lose concentration to pick the lock.}
+{120}{}{You take and intense shock from the security system and lose concentration to pick the lock.}
+{121}{}{You utterly fail in trying to pick this lock and destroy your picks.}
+{122}{}{You fail to pick this lock.}
+{123}{}{You see a sturdy, rusted door.}
+
+{124}{}{You see an unlocked door with red paint on it.}
+{125}{}{You see an unlocked door with red paint on it.}
+{126}{}{You see an unlocked door with a moving red field in front of it.}
+{127}{}{You see an unlocked door with a high voltage security system armed upon it.}
+{128}{}{You see a door with red paint on it.}
+{129}{}{You see a door with red paint on it.}
+{130}{}{You see a door with a moving red field in front of it.}
+{131}{}{You see a door with a high voltage security system armed upon it.}
+{132}{}{You see a locked door with red paint on it.}
+{133}{}{You see a locked door with red paint on it.}
+{134}{}{You see a locked door with a moving red field in front of it.}
+{135}{}{You see a locked door with a high voltage security system armed upon it.}
+{136}{}{The lock on this red door seems to need a special passkey.}
+{137}{}{The lock on this red door seems to need a special passkey.}
+{138}{}{The lock on this door seems to need a special passkey. Additionally, you notice a shimmering red field before the door.}
+{139}{}{The lock on this door seems to need a special red passkey. Additionally, you notice a high voltage security system armed upon it.}
+
+{140}{}{You see an unlocked door.}
+{141}{}{You see an unlocked door.}
+{142}{}{You see an unlocked door with a moving field in front of it.}
+{143}{}{You see an unlocked door with a high voltage security system armed upon it.}
+{144}{}{You see a door.}
+{145}{}{You see a door.}
+{146}{}{You see a door with a moving field in front of it.}
+{147}{}{You see a door with a high voltage security system armed upon it.}
+{148}{}{You see a locked door.}
+{149}{}{You see a locked door.}
+{150}{}{You see a locked door with a moving field in front of it.}
+{151}{}{You see a locked door with a high voltage security system armed upon it.}
+{152}{}{The lock on this door seems to need a special passkey.}
+{153}{}{The lock on this door seems to need a special passkey.}
+{154}{}{The lock on this door seems to need a special passkey. Additionally, you notice a shimmering field before the door.}
+{155}{}{The lock on this door seems to need a special passkey. Additionally, you notice a high voltage security system armed upon it.}
+
+{156}{}{You see an unlocked door with blue paint on it.}
+{157}{}{You see an unlocked door with blue paint on it.}
+{158}{}{You see an unlocked door with a moving blue field in front of it.}
+{159}{}{You see an unlocked door with a high voltage security system armed upon it.}
+{160}{}{You see a door with blue paint on it.}
+{161}{}{You see a door with blue paint on it.}
+{162}{}{You see a door with a moving blue field in front of it.}
+{163}{}{You see a door with a high voltage security system armed upon it.}
+{164}{}{You see a locked door with blue paint on it.}
+{165}{}{You see a locked door with blue paint on it.}
+{166}{}{You see a locked door with a moving blue field in front of it.}
+{167}{}{You see a locked door with a high voltage security system armed upon it.}
+{168}{}{The lock on this blue door seems to need a special passkey.}
+{169}{}{The lock on this blue door seems to need a special passkey.}
+{170}{}{The lock on this door seems to need a special passkey. Additionally, you notice a shimmering blue field before the door.}
+{171}{}{The lock on this door seems to need a special passkey. Additionally, you notice a high voltage security system armed upon it.}
+
+{172}{}{You see an unlocked door with blue paint on it.}
+{173}{}{You see an unlocked door with blue paint on it.}
+{174}{}{You see an unlocked door with a moving blue field in front of it.}
+{175}{}{You see an unlocked door with a high voltage security system armed upon it.}
+{176}{}{You see a door with blue paint on it.}
+{177}{}{You see a door with blue paint on it.}
+{178}{}{You see a door with a moving blue field in front of it.}
+{179}{}{You see a door with a high voltage security system armed upon it.}
+{180}{}{You see a locked door with blue paint on it.}
+{181}{}{You see a locked door with blue paint on it.}
+{182}{}{You see a locked door with a moving blue field in front of it.}
+{183}{}{You see a locked door with a high voltage security system armed upon it.}
+{184}{}{The lock on this blue door seems to need a special passkey.}
+{185}{}{The lock on this blue door seems to need a special passkey.}
+{186}{}{The lock on this door seems to need a special passkey. Additionally, you notice a shimmering blue field before the door.}
+{187}{}{The lock on this door seems to need a special blue passkey. Additionally, you notice a high voltage security system armed upon it.}
+
+{188}{}{You see a door with red paint on it.}
+{189}{}{You see a door with a moving red field in front of it.}
+{190}{}{You see a high voltage security system on this red door.}
+{191}{}{You see a high voltage security system on this red door. You wonder if there is a passkey to bypass it.}
+
+{192}{}{You see a door.}
+{193}{}{You see a door with a moving field in front of it.}
+{194}{}{You see a high voltage security system on this door.}
+{195}{}{You see a high voltage security system on this door. You wonder if there is a passkey to bypass it.}
+
+{196}{}{You see a door with blue paint on it.}
+{197}{}{You see a door with a moving blue field in front of it.}
+{198}{}{You see a high voltage security system on this blue door.}
+{199}{}{You see a high voltage security system on this blue door. You wonder if there is a passkey to bypass it.}
+
+{200}{}{You see a door with blue paint on it.}
+{201}{}{You see a door with a moving blue field in front of it.}
+{202}{}{You see a high voltage security system on this blue door.}
+{203}{}{You see a high voltage security system on this blue door. You wonder if there is a passkey to bypass it.}
+
+{204}{}{You notice there is a trap on this door.}
+{205}{}{The elevator is inoperable. It seems to need more power than emergency power can handle.}
