diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/GLOBLELV.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/GLOBLELV.msg
new file mode 100644
index 000000000..d91a93fc4
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/GLOBLELV.msg
@@ -0,0 +1,7 @@
+{100}{}{You see an elevator door with blue markings on it.}
+{101}{}{You are shocked by the security system, but sustain no damage. Perhaps you should try using something to bypass the security system.}
+{102}{}{You are shocked by the security system, and sustain a point of damage. Perhaps you should try using something to bypass the security system.}
+{103}{}{You are shocked by the security system, and sustain }
+{104}{}{ points of damage. Perhaps you should try using something to bypass the security system.}
+{105}{}{You disarm the security system on the door.}
+{106}{}{That item has no effect on the door.}
