diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/GLOCABN.MSG b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/GLOCABN.MSG
index 1a5915630..22473a89a 100644
--- a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/GLOCABN.MSG
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/GLOCABN.MSG
@@ -5,3 +5,6 @@
 {104}{}{You easily bypass the lock on the cabinet.}
 {105}{}{This lock seems more troublesome than you first believed.}
 {106}{}{That skill does not effect the cabinet.}
+
+# Fo1in2
+{200}{}{ easily unlocks the cabinet.}
\ No newline at end of file
