diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/GLOCABN.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/GLOCABN.msg
new file mode 100644
index 000000000..703b902a5
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/GLOCABN.msg
@@ -0,0 +1,10 @@
+{100}{}{You see a cabinet.}
+{101}{}{There seems to be an electronic lock on this cabinet.}
+{102}{}{You easily unlock the cabinet.}
+{103}{}{This lock seems more troublesome than you first believed.}
+{104}{}{You easily bypass the lock on the cabinet.}
+{105}{}{This lock seems more troublesome than you first believed.}
+{106}{}{That skill does not effect the cabinet.}
+
+# Fo1in2
+{200}{}{ easily unlocks the cabinet.}
