diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/GLOWCAGE.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/GLOWCAGE.msg
new file mode 100644
index 000000000..7dfff5790
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/GLOWCAGE.msg
@@ -0,0 +1 @@
+{100}{}{It appears as though this cage contained lab animals for testing. Somehow, the bars have been torn off.}
