diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/GLOWPUMP.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/GLOWPUMP.msg
new file mode 100644
index 000000000..46bab790f
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/GLOWPUMP.msg
@@ -0,0 +1,3 @@
+{100}{}{You see a water pump.}
+{101}{}{The design of this pump looks very different from the one in your Vault.}
+{102}{}{You realize that the pump's parts will be incompatible with those of your Vault.}
