diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/GLOWDESK.MSG b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/GLOWDESK.MSG
index 09cd39d90..6482e4c58 100644
--- a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/GLOWDESK.MSG
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/GLOWDESK.MSG
@@ -2,3 +2,6 @@
 {101}{}{You pry the drawer open.}
 {102}{}{The drawer is still jammed tight.}
 {103}{}{The desk is unaffected by the skill.}
+
+# Fo1in2
+{200}{}{ prys the drawer open.}
