diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/GLOWONE.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/GLOWONE.msg
new file mode 100644
index 000000000..c599491f8
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/GLOWONE.msg
@@ -0,0 +1,6 @@
+{100}{}{You see a glowing creature.}
+{101}{}{Mmmm...}
+{102}{}{Ahhh...}
+{103}{}{The glowing ghoul mumbles something and is then silent. It is drooling uncontrollably.}
+{104}{}{You receive no response.}
+{105}{}{You see: Glowing One.}
