diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/GLOYLDOR.MSG b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/GLOYLDOR.MSG
index a0d464b7e..b0e453480 100644
--- a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/GLOYLDOR.MSG
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/GLOYLDOR.MSG
@@ -112,4 +112,9 @@
 {203}{}{You see a high voltage security system on this yellow door. You wonder if there is a passkey to bypass it.}
 
 {204}{}{You notice there is a trap on this door.}
-{205}{}{The elevator is inoperable. It seems to need power.}
\ No newline at end of file
+{205}{}{The elevator is inoperable. It seems to need power.}
+
+# Fo1in2
+{400}{}{ picks the lock on the door without setting off the security system.}
+{401}{}{ failed to pick the lock on the door.}
+{402}{}{ fails to pick this lock.}
