diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/GLWALIEN.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/GLWALIEN.msg
new file mode 100644
index 000000000..11fdc4f45
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/GLWALIEN.msg
@@ -0,0 +1,5 @@
+{100}{}{A frozen corpse.}
+{101}{}{This is definitely not of terrestrial origin.}
+{102}{}{This body might be some kind of genetic experiment.}
+{103}{}{What? It's a dead body.}
+{104}{}{This frozen body looks odd.}
