diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/GLWBOX.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/GLWBOX.msg
new file mode 100644
index 000000000..7eac3ae49
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/GLWBOX.msg
@@ -0,0 +1,8 @@
+{100}{}{This looks to be the footlocker of some guards.}
+{101}{}{You pick the lock on the box.}
+{102}{}{You failed to pick the lock on the lockbox.}
+{103}{}{This item is useless on the lockbox.}
+{104}{}{The box is locked.}
+{105}{}{You pick the lock on the box.}
+{106}{}{You failed to pick the lock on the box.}
+{107}{}{This skill is useless on the box.}
