diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/GRAFFITI.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/GRAFFITI.msg
new file mode 100644
index 000000000..90f49bd9c
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/GRAFFITI.msg
@@ -0,0 +1 @@
+{100}{}{Scrawled on a scrap of paper is a barely legible message: 'Hea  d  or t   lower-level exit. No time  o  vac   e test p  je     T     s t e big    ! Dan  Sc  ly.}
