diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/GUNCACHE.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/GUNCACHE.msg
new file mode 100644
index 000000000..85525d3bc
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/GUNCACHE.msg
@@ -0,0 +1,6 @@
+{100}{}{This locker is locked.}
+{101}{}{You unlock the locker.}
+{102}{}{You pick the lock on the locker.}
+{103}{}{You fail to pick the lock on the locker.}
+{104}{}{You break your lockpicks while trying to unlock the door.}
+{105}{}{This lock seems too complicated for you.}
