diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/HDTVAULT.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/HDTVAULT.msg
new file mode 100644
index 000000000..b771d816e
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/HDTVAULT.msg
@@ -0,0 +1,8 @@
+# this is a door. Neutral/male. no tone of voice.
+
+{100}{}{There seems to be a lock on this door.}
+{101}{}{The door is already unlocked.}
+{102}{}{Those are inadequate for the job.}
+{103}{}{You bypass the electronic lock.}
+{104}{}{You failed to bypass the electronic lock.}
+{105}{}{The door is locked.}
