diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/HGENCVAN.MSG b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/HGENCVAN.MSG
index 62a6fc569..98d5396a7 100644
--- a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/HGENCVAN.MSG
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/HGENCVAN.MSG
@@ -1,4 +1,4 @@
-{100}{}{ERROR}
+{100}{}{You see a caravan driver.}
 {101}{}{Today's going to be a long day!}
 {102}{}{I really don't want to work today.}
 {103}{}{I hope we don't get raided... or worse.}
