diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/HOUSCHAR.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/HOUSCHAR.msg
new file mode 100644
index 000000000..14c29d2d3
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/HOUSCHAR.msg
@@ -0,0 +1,3 @@
+{100}{}{You see a worn chair.}
+{101}{}{You see a worn chair.}
+{102}{}{The chair seems inviting; however, you must continue on your mission.}
