diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/HTWRBOX.MSG b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/HTWRBOX.MSG
index 54be26685..bf2de2637 100644
--- a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/HTWRBOX.MSG
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/HTWRBOX.MSG
@@ -17,3 +17,15 @@
 {115}{}{You successfully disarmed the trap.}
 {116}{}{You accidentally set off the trap.}
 {117}{}{You failed to disarm the trap.}
+
+# Fo1in2
+# ex: Katja's multiple attempts ...
+{200}{}{'s multiple attempts to unlock the strongbox have set off a trap.}
+{201}{}{ was unable to unlock the strongbox, and the lock has become jammed.}
+{202}{}{ was unable to unlock the strongbox.}
+{203}{}{ finds a trap on the strongbox and successfully disarms it.}
+{204}{}{ finds a trap on the strongbox.}
+{205}{}{ successfully unlocked the strongbox.}
+
+
+
