diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/HTWRGRD.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/HTWRGRD.msg
new file mode 100644
index 000000000..1daa6647e
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/HTWRGRD.msg
@@ -0,0 +1,36 @@
+# Guards are all Male. They are akin to mercenaries. brief of words and very aburpt.
+# they have a general attitude of don't bother me.
+
+{100}{}{You see one of Hightower's guards.}
+{101}{}{You better leave. Daren Hightower doesn't take visitors.}
+{102}{}{Your company is not desired.}
+{103}{}{Leave!}
+{104}{}{You're not allowed here.}
+{105}{}{Go away, or you'll regret it.}
+{106}{}{This house is off limits to the likes of you.}
+# "You are not permitted to enter the house."
+
+{107}{}{What the... Get em!}
+{108}{}{You made a big mistake coming in here!}
+{109}{}{What are you doing here? You're dead meat.}
+# "You're dead meat" == "i'm going to kill you"
+
+{110}{}{Mmmm, looks good.}
+# refering to food
+
+{111}{}{Is that brahmin I smell?}
+
+{112}{}{Hook it to my veins!}
+# referring to getting more alcohol
+
+{113}{}{Anything in the fridge?}
+# fridge == refridgerator
+
+{114}{}{How much longer?}
+# referring to dinner
+
+{115}{}{I'm hungry.}
+{116}{}{Is it almost ready?}
+# referring to dinner.
+
+{117}{}{I'm going to pass out!}
