diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/HTWRWIFE.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/HTWRWIFE.msg
new file mode 100644
index 000000000..be5e55bed
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/HTWRWIFE.msg
@@ -0,0 +1,7 @@
+{100}{}{You see: Hightower's Wife.}
+{101}{}{Oh, hello. You must be one of my husband's associates.}
+{102}{}{Hello again.}
+{103}{}{My God!!! Someone's in the house!!!}
+{104}{}{Take what you want! Just leave!}
+{105}{}{How did you get in?!?}
+{106}{}{Guards!!! Help me!!!}
