diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/HUBJAKE.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/HUBJAKE.msg
new file mode 100644
index 000000000..a5b8db858
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/HUBJAKE.msg
@@ -0,0 +1,124 @@
+{100}{}{You see: Jacob, the arms dealer.}
+{101}{}{Hi! Welcome to Jake's. I've got some great merchandise in stock. Feel free to look around.}
+{102}{}{I would like to make a deal with you.}
+{103}{}{What are you selling here?}
+{104}{}{What can you tell me about a Water Chip?}
+{105}{}{Tell me about yourself.}
+{106}{}{Never mind. Bye.}
+{107}{}{wahnt.}
+{108}{}{nO wahnt.}
+{109}{}{I deal with high-end arms. The best stuff you can find this side of the Brotherhood, and since the Brotherhood doesn't deal with outsiders, I've gotta' say I'm the best. I have a Sniper Rifle, and you would be hard pressed to find flamethrower fuel anywhere else in the wastes. Oh yeah... I've got a Geiger counter, too. Very rare.}
+{110}{}{Show me what ya' got.}
+{111}{}{Tell me more about the Sniper Rifle.}
+{112}{}{What would I need a Geiger counter for?}
+{113}{}{Maybe I'll check with you later. Bye.}
+{114}{}{Man, you do like the rare stuff! Where were you when I was selling some depleted uranium rounds a few years back? I don't do chips, though. Ain't a market in it. Anything else?}
+{115}{}{Show me what ya' got.}
+{116}{}{Then tell me what you DO have.}
+{117}{}{Tell me about yourself.}
+{118}{}{Well, I've been around. Used to be with the Union of Atomic Workers. Sort of like the Brotherhood, but less friendly. They wanted to keep to themselves, and I... well, I wanted to see the world and be my own boss, so I left.}
+{119}{}{Let's see what you have.}
+{120}{}{Tell me what you've got.}
+{121}{}{What can you tell me about a Water Chip?}
+{122}{}{Good meeting you, friend. I may stop by later. Bye.}
+{123}{}{Ahh, the DKS is a fine long range weapon. It was originally .308, but we rechambered it for the more common .223 round. We tuned the barrel, too. Most importantly, check out this trigger. Smooth, huh? The trigger breaks perfectly now. Overall, this is a good choice for hunting any type of animal...}
+{124}{}{Sounds good. Let's see if we can make a deal.}
+{125}{}{What would I need a Geiger counter for?}
+{126}{}{Maybe I'll check with you later. Bye.}
+{127}{}{What do you need a Geiger counter for!? [Laughs] Though it's been 80 years or so since the bombs fell, there's still radiation around, you goof. You can't see it, never could, but it's there. Heck I'm willin' to bet you got some counts on you now. Everyone that lives in the wastes has a few rad counts.}
+{128}{}{Show me what you have to sell.}
+{129}{}{Tell me more about radiation.}
+{130}{}{How does a Geiger counter work?}
+{131}{}{Goof!?! Do you always insult your customers? If you think I'm going to buy something from you now you're sadly mistaken.}
+{132}{}{Well, some call it the Phantom Death, 'cause that's what it is. You can't see it, you can't hear it, you can't even smell it. It basically builds up in your system. You never feel it until it's too late. The more trips you make to Hot Spots, the more it builds up and it will never leave your system. Ever. The rads just keep building up. Well... unless you can get a hold of some Rad-Away.}
+{133}{}{Let's see if we can make a deal.}
+{134}{}{How does a Geiger counter work?}
+{135}{}{Where can I find Rad-Away? I haven't seen it being sold many places.}
+{136}{}{Wow. You know a lot. It sounds like you've been around.}
+{137}{}{Thanks for the info. Bye.}
+{138}{}{It's pretty simple, actually. You use it and it tells you how many rads you have. As long as you keep your Rad count under 100, you should be fine. Go above 100 and you're in some trouble. The worst part is you won't even know you've been sucking up rads until a few days later. Get over 1000 and you're cooked - unless you get a hold of a whole lot of Rad-Away.}
+{139}{}{Let's see if we can make a deal.}
+{140}{}{What can you tell me about radiation?}
+{141}{}{I don't go for this hard sell crap. See you later.}
+{142}{}{Never mind.}
+{143}{}{}
+{144}{}{gEt.}
+{145}{}{Me go.}
+{146}{}{Yes.}
+{147}{}{No. Just looking around.}
+{148}{}{I don't want any trouble. If you want to do business then let's do it. If not, get out.}
+{149}{}{gEt.}
+{150}{}{Me go.}
+{151}{}{Let's see what you've got.}
+{152}{}{You can go to Hell. I go where I want, when I want!}
+{153}{}{I'm leaving.}
+{154}{}{What the hell are you doing in here?!?!?}
+{155}{}{I don't carry it. Any I find I keep for myself. Right up the street here, there's a guy that may be able to help you. His name is Vance. Don't ask too many questions of him. Let's just say this is his area of expertise. Just tell him Jacob sent you and he should do you up just fine.}
+{156}{}{How does a Geiger counter work?}
+{157}{}{Thanks for the info. Bye.}
+{158}{}{Here's what I've got.}
+{159}{}{Bye.}
+{160}{}{Thanks.}
+{161}{}{Ook.}
+{162}{}{The mutants wiped 'em out about ten years ago, so I guess I made the right decision. I eventually settled down here. The cops don't like my trade and I've been rousted more than once, but I'm a helluva lot more honest than most of the merchants here.}
+
+#
+# TRANSLATION NOTE: The following three sets of lines are identical in English,
+# but they differ in foreign translations. The first line is for the case of the
+# player-character masculine ("my friend" will be masculine); the second line is
+# for the case of the player-character feminine ("my friend" will be feminine).
+#
+
+{163}{}{Hey, how's it going my friend? Lookin' to buy somethin'?}
+{164}{}{Hey, how's it going my friend? Lookin' to buy somethin'?}
+{165}{}{Your selection is good, but I'm looking for something with a bit more punch. Any idea where I could go?}
+{166}{}{Well, you might try the Gun Runners over in L.A. Boneyard. They'll probably have something to suit your needs. I've heard the guys at the Brotherhood of Steel pack some pretty powerful hardware too, so you might check them out.}
+
+# Fo1in2
+# Upgrade Weapons
+	{1999}{}{Can you modify or upgrade weapons?}
+
+{2000}{}{Sure. What do you want me to look at?}
+{2001}{}{Sure, but you don't look like you have anything I can help you with. Bring me pistols and rifles, and I might be able to help you.}
+	{2010}{}{Can you do anything with any of these pistols I have?}
+	{2020}{}{One of my rifles could use some attachments.}
+	{2025}{}{Can you beef up my flamethrower?}
+	{2030}{}{Can you do anything with this cattleprod here?}
+	{2040}{}{Let's see if we can make a deal instead.}
+	{2050}{}{Never mind. See you later.}
+
+{2100}{}{Let me see what you have.} # unused
+	{2110}{}{Actually, could you upgrade one of my other weapons instead?}
+	{2120}{}{I wanted something else...}
+	{2130}{}{Actually, none of them. I have to go.}
+	{2140}{}{Too rich for my blood. I'll be back when I have more cash.}
+
+# Pistols
+{2200}{}{I can rig a speed loader on that magnum for }
+{2201}{}{Hmm, that Desert Eagle would sure be a sweet rig with an expanded magazine. I can do that for }
+{2202}{}{ caps.}
+{2203}{}{Oh, wow!  I see you have a Mauser M/96 there! Not easy to get ammo for that thing nowadays. If you want, I can convert it to 10mm for }
+	{2210}{}{Put the speed loader on the magnum.}
+	{2220}{}{Expand the magazine in this Desert Eagle here.}
+	{2230}{}{Convert the Mauser to 10mm.}
+
+# Rifles
+{2300}{}{I can put a nice all-weather scope on that hunting rifle for }
+{2301}{}{Nothing like having an assault rifle with an expanded magazine. I can fix yours up with one for }
+{2302}{}{That advanced assault rifle is a good gun. Hey, I got a low-light scope for that. It's yours for }
+{2303}{}{Your combat shotgun... I can give it a drum magazine for }
+	{2310}{}{All right then...put a scope on the Colt hunting rifle.}
+	{2320}{}{Expand the magazine of the AK-112 assault rifle.}
+	{2330}{}{Attach that low-light scope on the AK-198.}
+	{2340}{}{Attach a round magazine adapter to the Winchester City-Killer.}
+
+# Melee
+{2400}{}{You want that cattleprod juiced up? We can SUPERCHARGE it for }
+{2410}{}{Sure.}
+
+# Big Guns
+{2500}{}{Let me take a look... Yeah, we can do something with it for }
+{2510}{}{Cool. Let's do this.}
+
+{2600}{}{Wait here, it'll take just a bit.}
+{2601}{}{There you go. Anything else?}
