diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/HUBJAKE.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/HUBJAKE.msg
index a5b8db858..cd35d1964 100644
--- a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/HUBJAKE.msg
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/HUBJAKE.msg
@@ -80,6 +80,7 @@
 
 {2000}{}{Sure. What do you want me to look at?}
 {2001}{}{Sure, but you don't look like you have anything I can help you with. Bring me pistols and rifles, and I might be able to help you.}
+{2002}{}{Bring me pistols and rifles, and I might be able to help you.}
 	{2010}{}{Can you do anything with any of these pistols I have?}
 	{2020}{}{One of my rifles could use some attachments.}
 	{2025}{}{Can you beef up my flamethrower?}
