diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/HUBPRIS.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/HUBPRIS.msg
new file mode 100644
index 000000000..e566da10f
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/HUBPRIS.msg
@@ -0,0 +1,8 @@
+{100}{}{You see a prisoner.}
+{101}{}{I didn't kill anyone!}
+{102}{}{I'm innocent!}
+{103}{}{What are you lookin' at?}
+{104}{}{I was just borrowing it. Honest!}
+{105}{}{Where's my bread and water?}
+{106}{}{I've been framed!}
+{107}{}{Nobody knows 'da trouble I seen...}
