diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/IAN.MSG b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/IAN.MSG
index 3e2fca7e5..fb6bcf946 100644
--- a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/IAN.MSG
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/IAN.MSG
@@ -128,7 +128,7 @@
 # Ian Tell-Me-Abouts
 #
 
-{1000}{}{Water|Waterchip|Water Chip}
+{1000}{}{Waterchip|Water Chip}
 {1001}{}{War}
 {1002}{}{Shady|Shady Sands|Sands}
 {1003}{}{Junktown|Junk Town}
@@ -148,12 +148,12 @@
 {1017}{}{Hall|HALL OF THE DEAD|DEADHALL|HALL OF DEAD}
 {1018}{}{Library}
 {1019}{}{Overseer}
-{1020}{}{Doctor|Medic|Doc}
+{1020}{}{Razlo}
 {1021}{}{Healing}
 {1022}{}{Water Merchants|Water Merchant}
 {1023}{}{Crimson Caravans|Crimson|Crimson Caravan}
 {1024}{}{Far Go Traders|Fargo|Fargo Trader|Fargo Traders|FAR-GO-TRADERS|Far Go|Far-Go}
-{1025}{}{Killian|Kilian|Killian Darkwaters|Kilian Darkwaters|Killian Darkwater|Kilian Darkwater}
+{1025}{}{Killian|Kilian|Killian Darkwaters|Kilian Darkwaters|Killian Darkwater|Kilian Darkwater|Darkwater|Darkwaters}
 {1026}{}{Gizmo}
 {1027}{}{Boneyard|Angels Boneyard}
 {1028}{}{Casino}
@@ -168,6 +168,7 @@
 {1037}{}{Vats}
 {1038}{}{Military|Base|Military Base}
 {1039}{}{Scorpions|Scorpion|Rad Scorpion|Radscorpions|Radscorpion}
+{1040}{}{Buy|Buying|Sell|Selling|Barter|Trade|Trading|Business|Bartering}
 {1041}{}{Apocalypse}
 {1042}{}{Mutants|Mutant}
 {1043}{}{Morpheus}
@@ -182,7 +183,7 @@
 {1052}{}{Drugs|Drug}
 {1053}{}{Seth}
 {1054}{}{Aradesh}
-{1055}{}{Razlo}
+{1055}{}{Doctor|Medic|Doc}
 {1056}{}{Zimmerman|Jon Zimmerman|Jon|Zimermann|Jon Zimmermann|Zimmermann}
 {1057}{}{Regulators|Regulator}
 {1058}{}{Scavs|Scav|Scavenger}
@@ -193,7 +194,6 @@
 {1063}{}{Experiments|Experiment}
 {1064}{}{Ian}
 {1065}{}{Merchants|Merchant}
-{1066}{}{Buy|Buying|Sell|Selling|Barter|Trade|Trading|Business|Bartering}
 
 {1100}{}{No idea what a Water Chip is. Sorry, friend.}
 {1101}{}{Happened about a hundred years ago. Basically, a bunch of countries got kind of antsy, and everything went BOOM!}
@@ -235,6 +235,7 @@
 {1137}{}{Sorry, bud. Don't know much about that.}
 {1138}{}{Sorry, bud. Don't know much about that.}
 {1139}{}{Radscorpions are very dangerous. They have a poisonous stinger in the tail, and they are very quick.}
+{1140}{}{Could always check in with one of the caravan companies in the Hub if you're low on caps.}
 {1141}{}{You know, we'll always have war. Just the next one's gonna be fought with sticks.}
 {1142}{}{I saw some weird things when I was guarding caravans for the Hub. Mutated coyotes and manti and scorpions, that sort of thing.}
 {1143}{}{Sorry, bud. Don't know much about that.}
@@ -260,7 +261,6 @@
 {1163}{}{Sorry, bud. Don't know much about that.}
 {1164}{}{That's me.}
 {1165}{}{Some merchants come from the Hub to trade with Junktown. That's all I know.}
-{1166}{}{Could always check in with one of the caravan companies in the Hub if you're low on caps.}
 
 {1999}{}{I've learned some new tricks!}
 {2000}{}{Let's discuss our combat tactics.}
