diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/IIMINE.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/IIMINE.msg
new file mode 100644
index 000000000..63ead2ced
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/IIMINE.msg
@@ -0,0 +1,13 @@
+
+#############################################################
+#  The following lines are for what the player received     #
+#  experience for (ie, picking a lock, disarming traps).    #
+#############################################################
+{102}{}{You successfully disarm the trap.}
+{103}{}{You fail to disarm the trap.}
+{104}{}{You accidentally set off the trap.}
+
+# Prepend Lines with NPC name
+{105}{}{ successfully disarms the trap.}
+{106}{}{ fail to disarms the trap.}
+{107}{}{ accidentally sets off the trap.}
