diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/INVADER.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/INVADER.msg
new file mode 100644
index 000000000..5aaf930ec
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/INVADER.msg
@@ -0,0 +1,8 @@
+{100}{}{You see an invading Super Mutant.}
+{101}{}{Come here, Human!}
+{102}{}{Stop!}
+{103}{}{Die, human!}
+{104}{}{Death to you!}
+{105}{}{Hu-maaaan!}
+{106}{}{Me squish you now!}
+{107}{}{All normies will die!}
