diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/IZO.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/IZO.msg
new file mode 100644
index 000000000..149b1dd06
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/IZO.msg
@@ -0,0 +1,7 @@
+{100}{}{He has the stance and musculature of a master martial artist.}
+{101}{}{Izo says nothing. He is watching you carefully with a gaze that makes you uncomfortable.}
+{102}{}{ [Izo gives you your money]}
+{103}{}{Izo motions to the door, makes a throat-cutting gesture, and shakes his head.}
+{104}{}{Thanks.}
+{105}{}{Duhh...}
+{106}{}{Izo motions to the door.}
