diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/JTPEASNT.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/JTPEASNT.msg
new file mode 100644
index 000000000..02cb518d9
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/JTPEASNT.msg
@@ -0,0 +1,6 @@
+{100}{}{You see a Junktown peasant.}
+{101}{}{Get out of here!}
+{102}{}{I've heard bad things about you. Are they true?}
+{103}{}{Hi.}
+{104}{}{This is our place. Wanna hang out for a while?}
+{105}{}{Hmmm, you look juicy...}
