diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/JUNKJAIL.MSG b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/JUNKJAIL.MSG
index 25c59c360..0e7076e70 100644
--- a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/JUNKJAIL.MSG
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/JUNKJAIL.MSG
@@ -7,3 +7,10 @@
 {106}{}{You fail to lock the door.}
 {107}{}{You earn  }
 {108}{}{ experience for busting out of jail!}
+
+# Fo1in2
+{200}{}{ unlocks the door.}
+{201}{}{ locks the door.}
+{202}{}{ fails to unlock the door.}
+{203}{}{ fails to lock the door.}
+
