diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/JUNKIE.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/JUNKIE.msg
new file mode 100644
index 000000000..b1675296f
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/JUNKIE.msg
@@ -0,0 +1,4 @@
+{100}{}{You see a junkie.}
+{101}{}{Look at the colors!}
+{102}{}{Ohhhhh wowwwwwww! I'm eating my own face...}
+{103}{}{Mother?}
