diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/JUNKDEMO.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/JUNKDEMO.msg
new file mode 100644
index 000000000..c4b0e3a3b
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/JUNKDEMO.msg
@@ -0,0 +1,52 @@
+{100}{}{You think that puny leather will save you from Baka?}
+{101}{}{You're no match for us!}
+{102}{}{If I were that ugly, I'd kill myself.}
+{103}{}{You Crypts don't have a chance.}
+{104}{}{Do you like pain?}
+{105}{}{I'm going to enjoy this.}
+{106}{}{You shall die scum!}
+{107}{}{It's pay back time.}
+{108}{}{Are you ready for some pain?}
+{109}{}{The only thing stupider than you is Rayze.}
+{110}{}{I guess afterbirth can live a little while.}
+{111}{}{The Crypts shall be buried once more!}
+{112}{}{Perish scum!}
+{113}{}{Who said you could come out of your graves? Return!}
+{114}{}{Thy lowly ways shall be eradicated from the earth!}
+{115}{}{Only Fools have true knowledge to rule Scrapheap.}
+{116}{}{Begone gut slime of a three legged chicken!}
+{117}{}{What happened to you? You momma drop you on your head at birth?}
+{118}{}{}
+{119}{}{}
+{120}{}{}
+{200}{}{Run, Fools, while you still have a chance to live.}
+{201}{}{Hey, Rayze, hunting season just openned, and Fools are on the list of dumb animals.}
+{202}{}{Hey guys, leave a Fool for me to kill.}
+{203}{}{The Crypts will bury you!}
+{204}{}{Fools got no reason to live.}
+{205}{}{The Foolish shall perish.}
+{206}{}{Look who got hit with the ugly stick.}
+{207}{}{You going to get buried, Fool!}
+{208}{}{I am going to kick some serious ass! I am ready to ruuummmblllle!}
+{209}{}{Crypts rule! And Fools drool!}
+{210}{}{We are the bringers of death!}
+{211}{}{Don't worry, it'll be quick and painful!}
+{212}{}{PURPLE BABOONS!}
+{213}{}{The Brahmin have more sense than a Fool.}
+{214}{}{The Fools are so weak, even Lex can beat them all up.}
+{215}{}{Haha! More fools to kill!!}
+{216}{}{The only thing you Fools will rule is a tombstone.}
+{217}{}{It's about time.  I've been waiting for this.}
+{218}{}{Even Lenny has better aim than you.}
+{219}{}{Eat lead fools.}
+{220}{}{I'll mount your head on a spike.}
+{300}{}{Hey! The generator is broken. Let's leave this dump.}
+{400}{}{With the generator broken, there is little reason for the gangs to fight over Scrapheap. Your job here is done, but your mission to find the water purification chip continues.}
+{402}{}{With the gangs defeated, Scrapheap is a safe haven once again.}
+{403}{}{Thanks, bye.}
+{404}{}{Okay, thanks.}
+{405}{}{Hmm. Bye.}
+{406}{}{See ya.}
+{407}{}{Thanks.}
+{408}{}{Okay, bye.}
+{409}{}{[more]}
