diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/JUSTIN.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/JUSTIN.msg
new file mode 100644
index 000000000..6481e04bd
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/JUSTIN.msg
@@ -0,0 +1,108 @@
+{100}{}{You see: Sheriff Justin Greene.}
+{101}{}{Aw, it's the slayer of the innocents. I've heard of you, and you're lucky I have a badge stopping me from blowing you away on the spot. I suggest you leave our town - Right now. And stay away from our children.}
+{102}{}{Hello sir! The name's Justin Greene. I'm the sheriff of this fine city. Is there something I can do for you this fine day?}
+{103}{}{Hello ma'am! The name's Justin Greene. I'm the sheriff of this fine city. Is there something I can do for you this fine day?}
+{104}{}{Well, spit it out!}
+{105}{}{Decker tried to hire me to kill some merchant and his wife.}
+{106}{}{Decker tried to hire me to kill someone named Jain.}
+{107}{}{Nothing, never mind.}
+{108}{}{Sorry, I can't tell you. I'd be dead.}
+{109}{}{Well, thank you. We try our best. It's really tough in a city this size, but we do the best we can with what we've got. It's hard when the thugs have better firepower than you do.}
+{110}{}{Thanks again.}
+{111}{}{It's time for you to leave, child killer.}
+{112}{}{We couldn't have that. But, if you ever change your mind, please let me know. It's a dangerous place out there and someone has to stand up for what's right.}
+{113}{}{Excellent! I had already heard from other sources about his plan. You're the final confirmation I needed to take action against him. This is going to be rough, and I can use any help I can get. I can pay you 1300 caps. 300 now, and the rest when we're all done. What do you say?}
+{114}{}{You're too late... Damn! I knew he was responsible, I only wish I knew in time to stop him. In any case, you're the final witness I needed to act. We'll have to 'take him out'. This is going to be rough, and I can use any help I can get. I can pay you 1300 caps. 300 now, and the rest when we're all done. What do you say?}
+{115}{}{You're too late... How do I know you're not responsible? Decker's not likely to let someone know about his plans and live.}
+{116}{}{I just got word that she's already dead... wait a minute. How do I know that you're not the one responsible? Decker's not likely to let someone know about his plans and live.}
+{117}{}{It'll be a pleasure.}
+{118}{}{Sorry, I want it all up front.}
+{119}{}{No way! You're on your own.}
+{120}{}{No way. You seem trustworthy, but I can't be sure that you won't leave us when it gets hot. This way, one of us has to survive to be sure you get paid. This way, it might add a 'little' bit of insurance.}
+{121}{}{Let's do it.}
+{122}{}{You're on your own, Sheriff.}
+{123}{}{Great! Here's the first part of your payment. [The Sheriff hands you 300 caps]}
+{124}{}{I'm ready.}
+{125}{}{Okay, Bye.}
+{126}{}{I'm really sorry to hear that. If you change your mind, let me know, but I can't wait very long.}
+{127}{}{Okay, I'll do it. Just wait for me to get some supplies.}
+{128}{}{You're mad.}
+{129}{}{Bye.}
+{130}{}{Great! Here's the first part of your payment. [The Sheriff gives you 300 caps]}
+{131}{}{What? Do you expect me to stand around and let him kill innocent people? I've a responsibility to this city, which I don't expect you to understand.}
+{132}{}{You're right. Now's it's your turn.}
+{133}{}{How dare you accuse me!}
+{134}{}{I don't care what you think.}
+{135}{}{Well, he didn't try anything.}
+{136}{}{We were in the Falcon, and there were a lot of people around.}
+{137}{}{I'm sorry. Decker's been playing us all against each other for years. I don't know who to trust any more. I really could use your help, though. It'll be rough, but I'll pay you 1300 caps. 300 now and 1000 when we're done.}
+{138}{}{I know a rat when I smell one! You're lucky I don't have any proof, and I suggest you leave town before I find some.}
+{139}{}{That's very strange. I've never heard of Decker coming out of his office. He must be getting braver. That damn, murdering... argh! I'll get him yet! I'll tell you what, you help us out, and I'll pay you 1300 caps - 300 now and 1000 when we're all done. What do you say?}
+{140}{}{That's the biggest pile of brahma shit I've ever heard. Decker never leaves his office. I knew there was something not right about you. You're lucky I don't have any proof, and I suggest you leave town before I find some!}
+{141}{}{Okay, let's go...}
+{142}{}{So, you came to turn yourself in? I'm impressed.}
+{143}{}{Yes, I'm sorry.}
+{144}{}{Turn myself in? Don't make me laugh.}
+{145}{}{Get him!}
+{146}{}{Get her!}
+{147}{}{Alright, into your cell you go.}
+{148}{}{Just sit tight. You're not going anywhere for a while.}
+{149}{}{Fugitive! Get him!}
+{150}{}{Fugitive! Get her!}
+{151}{}{Ready to go take out Decker?}
+{152}{}{Yes.}
+{153}{}{No.}
+{154}{}{Ug.}
+{155}{}{I can't believe Decker's finally dead. I never thought I'd see the day.}
+{156}{}{Have you changed your mind? We could still use you to get rid of that disease we talked about.}
+{157}{}{I'll go.}
+{158}{}{No, sorry.}
+{159}{}{Fah?}
+{160}{}{We're watching you.}
+{161}{}{I have a crime to report!}
+{162}{}{You're doing a fine job.}
+{163}{}{Just stopping by. See ya.}
+{164}{}{Oula.}
+{165}{}{Hello again, sir. What can I do for you today?}
+{166}{}{Hello again, ma'am. What can I do for you today?}
+{167}{}{Hello. How can I help you?}
+{168}{}{Yes?}
+{169}{}{Just sleep if off, okay?}
+{170}{}{You're kidding! Well... uh... did... did he ask you personally?}
+{171}{}{Yes, he did.}
+{172}{}{Man, that was one hell of a battle! You did well in there. Now that Decker and Kane are gone the rest of the Underground will fall very shortly. Without leadership, organizations like that can't last. Nobody around here is smart enough to take up the reigns.}
+{173}{}{Well, here's the rest of your reward. Thanks again for your help. This city owes you a debt of gratitude. [The Sheriff hands you 1000 caps]}
+{174}{}{We can leave as soon as you're ready.}
+{175}{}{Let me know when you're ready. We'll wait for you, but don't take too long.}
+
+##--by Sduibek, for Fallout Fixt:
+
+{180}{}{Iguana Bob is using meat from human corpses for his Prime Choice Select cuts.}
+{185}{}{Prime Choice Select is made of people!!!! It's made of people!!!}
+
+{181}{}{You're kidding! Are you sure? Hmmm... Let me look into this and i'll get back to you.}
+{182}{}{Okay.}
+{183}{}{Well, your story checks out. Damn. Bob always seemed a little "off" but I'd never have suspected he was selling human flesh to us. I was having one or two of those for lunch every day!}
+{184}{}{I know. It's a shame, considering how delicious they are.}
+
+{186}{}{Right. Okay, we'll take care of this. Thanks for your report.}
+{187}{}{Great, i'm just happy to know he'll receive due justice. Bye!}
+{188}{}{What, no finder's fee?}
+{189}{}{Sure, any time.}
+{190}{}{Damn fool, I knew you were gonna say that! Here, take your reward and get out. [The Sheriff hands you 250 caps]}
+{191}{}{Whatever, bye.}
+{192}{}{Thank you. Have a nice day!}
+
+# Fo1in2:
+{200}{}{ experience for reporting Iguana Bob to the Sheriff.}
+{201}{}{I've talked to Bob. He said you're just trying to pull his good name through the mud and honestly, I think his word has more weight than yours.}
+{202}{}{You better get out of here now before I'll put YOU in one of our cells.}
+
+# TMA
+{1000}{}{Greene|Justin|Justin Greene|Sheriff|Sherif|Mayor|Sheriff Greene}
+{1001}{}{Police|Hub Police}
+{1002}{}{Stealing|Thief|Crime|Criminals|Thieves|Thiev}
+
+{1100}{}{You're talking to me.}
+{1101}{}{We are keeping law and order here in the Hub.}
+{1102}{}{Do you want to report any crimes?}
