diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/JUSTIN.MSG b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/JUSTIN.MSG
index 95bcc7cd3..af7e4936d 100644
--- a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/JUSTIN.MSG
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/JUSTIN.MSG
@@ -97,3 +97,12 @@
 {200}{}{ experience for reporting Iguana Bob to the Sheriff.}
 {201}{}{I've talked to Bob. He said you're just trying to pull his good name through the mud and honestly, I think his word has more weight than yours.}
 {202}{}{You better get out of here now before I'll put YOU in one of our cells.}
+
+# TMA
+{1000}{}{Greene|Justin|Justin Greene|Sheriff|Sherif|Mayor|Sheriff Greene}
+{1001}{}{Police|Hub Police}
+{1002}{}{Stealing|Thief|Crime|Criminals|Thieves|Thiev}
+
+{1100}{}{You're talking to me.}
+{1101}{}{We are keeping law and order here in the Hub.}
+{1102}{}{Do you want to report any crimes?}
\ No newline at end of file
