diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/KERI.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/KERI.msg
new file mode 100644
index 000000000..01cf86334
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/KERI.msg
@@ -0,0 +1,137 @@
+{100}{}{You see: Keri.}
+{101}{}{Who the hell are you? If you want a job you'll need to talk to Demetre.}
+{102}{}{I said to talk to my dad if you want a job, not me.}
+{103}{}{So, that old bastard gave you the okay to work for us, eh? Well you must be one tough bitch. I'm impressed, but we'll see how you do. We leave only twice a month, on the 3rd and 17th. Can you handle that? We go to all the local hot spots: the Brotherhood of Steel, Junktown, Necropolis, and L.A. Boneyard.}
+{104}{}{So, that old bastard gave you the okay to work for us, eh? Well you must have balls of steel. I'm impressed, but we'll see how you do. We leave only twice a month, on the 3rd and 17th. Can you handle that? We go to all the local hot spots: the Brotherhood of Steel, Junktown, Necropolis, and L.A. Boneyard.}
+{105}{}{And the pay?}
+{106}{}{Uh.}
+{107}{}{Our pay's the best; 600 scripts - and yes, that is for each way... but you don't have to do both. This is a pretty tough caravan ride, and you're almost guaranteed to run into some raiders... or worse. We'll have no sissies on my caravans and you better have plenty of ammo.}
+{108}{}{ We're leaving today. Are you interested?}
+{109}{}{ We leave on }
+{110}{}{Empty}
+{111}{}{Empty}
+{112}{}{. Are you interested?}
+{113}{}{We're all heading out today.}
+{114}{}{We're going out on }
+{115}{}{}
+{116}{}{}
+{117}{}{ Are you interested?}
+{118}{}{I'll still give your sorry ass a job, if you're interested?}
+{119}{}{Sure.}
+{120}{}{No.}
+{121}{}{I think I'm interested in you.}
+{122}{}{Maybe.}
+{123}{}{Ug.}
+{124}{}{Ha, I thought you were a coward! Well... if you brave'n up, stop by and I might still give you a job. We leave on the 3rd and 17th, you can just show up that day if you like. We can always use more guns.}
+{125}{}{You just love to waste my time, don't you.}
+{126}{}{Whatever. If you get the urge for some action come back and I'll hook you up on the next caravan out.}
+{127}{}{Hey! I know you can get some good drugs here, but I'm not supposed to hire you unless you're sober.}
+{128}{}{I said to lay off the drugs, man!}
+{129}{}{I said to lay off the drugs, bitch!}
+{130}{}{Well now... I hate to say it but... I've heard of you. Are you as tough in private as in public? Let's see... [She leads you around a corner]}
+{131}{}{[On the way back she slips you a few things.]}
+{132}{}{Well, you can last for QUITE a while.}
+{133}{}{Well, you ARE interesting and VERY different from most people... show me what you've got tough guy. [She leads you around a corner]}
+{134}{}{[On the way back she slips you a few things.]}
+{135}{}{Well, you can last for QUITE a while.}
+{136}{}{Hey Pops, looks like we've got a thief here! What do you have to say for yourself, you scum?}
+{137}{}{I'm sorry. I'll never do it again.}
+{138}{}{Catch me if you can...}
+{139}{}{I don't like quiet! Come and get me!}
+{140}{}{[Nod yes]}
+{141}{}{OOOGABOOOGA!!!}
+{142}{}{Hey Pops, this dumb shit didn't learn a lesson the first time. She tried to steal from me again. Let's pummel her.}
+{143}{}{Hey Pops, this dumb shit didn't learn a lesson the first time. He tried to steal from me again. Let's pummel him.}
+{144}{}{Well... I know how easy it is to be tempted to steal, but... Hell, I'll give you a chance. Don't show your face around here again and I won't call the cops. Now beat it.}
+{145}{}{Oh we'll make sure of that, won't we Pops?}
+{146}{}{Go to hell!}
+{147}{}{I guess so.}
+{148}{}{You're dead meat.}
+{149}{}{I said to get out!}
+{150}{}{Police!}
+{151}{}{Sorry, we won't deal with you anymore.}
+{152}{}{No jobs here, beat it.}
+{153}{}{Don't get on my bad side.}
+{154}{}{Moron.}
+{155}{}{Intriguing, but you're not my type.}
+{156}{}{Good. Like I said before we pay 600 scripts. Which route did you want to take?}
+{157}{}{Cool. The next one's going out on }
+{158}{}{. Come back then and I'll put you on a caravan. As always, it pays 600 scripts.}
+{159}{}{Okay. }
+{160}{}{Sorry, I can't wait that long.}
+{161}{}{I should string you up by your eyelids! Because of you we lost a whole damn caravan! Get the hell out of my face! I'm going to catch some serious shit for this one.}
+{162}{}{I should string you up by your balls! Because of you we lost a whole damn caravan! Get the hell out of my face! I'm going to catch some serious shit for this one.}
+{163}{}{Brotherhood of Steel}
+{164}{}{Junktown}
+{165}{}{Necropolis}
+{166}{}{Boneyard}
+{167}{}{Never mind. I have something I forgot to do. }
+{168}{}{Lock and load! We're out of this joint!}
+{169}{}{Let me know if anything cool happens, Dad's making me stay around the warehouse.}
+{170}{}{Great! The other psychopaths are ready. We'll leave right away.}
+{171}{}{Cool, it'll just take a few minutes to get the psychos together.}
+{172}{}{Brotherhood of Steel, here we come! Well... I'm not going with you, but you know what I mean.}
+{173}{}{Okay, get out of my face already. Those Brotherhood fanatics are waiting for you.}
+{174}{}{I hope you like mountains! You're going right into them. You should see lots of action there.}
+{175}{}{Ready to go, and you'll be heading straight though the mountains! It should be a blast!}
+{176}{}{Junktown's pretty tame. I prefer the more dangerous routes.}
+{177}{}{I hear there's some crazy doctor there, you should check him out. His name's Doc Morbid.}
+{178}{}{Junktown's a short trip, you should be back soon.}
+{179}{}{Watch your ass, Necropolis isn't a very friendly place.}
+{180}{}{Set's the leader of the town. At least he was the last time I was there.}
+{181}{}{That town's full of skags! Well, I better get you going.}
+{182}{}{LA's pretty cool. You'll like it, but just be sure not to leave the caravan until it arrives. It's not to safe, for the caravans that is.}
+{183}{}{Hope you get a chance to see it, there's also a cool church just a few miles to the south of it. The people are whacked, though.}
+{184}{}{I'm told L.A. Boneyard used to be a place called LA. I don't know what the person's initials stand for, but it's a cool place.}
+{185}{}{What the hell do you want?}
+{186}{}{Shut up and get out of my way, I'm not in the mood!}
+{187}{}{I want another job.}
+{188}{}{Nothing.}
+{189}{}{Zug.}
+{190}{}{Ug.}
+{191}{}{Yeah right! So you can abandon another caravan? You're lucky they made it in one piece.}
+{192}{}{I'm lucky? You're lucky I don't pound your face in!}
+{193}{}{Hey! I had places to go and... stuff!}
+{194}{}{Hey! You didn't say I couldn't leave! It's not like they were in the middle of combat.}
+{195}{}{Get out of here! You make me sick! That's why you left the caravan? You were on drugs I bet!}
+{196}{}{Yeah? Well why don't you go there now! You're off our list for good.}
+{197}{}{What? You've never worked a caravan before? I can't believe my dad hired you in the first place! Get lost! We only hire experienced personnel! That's why we pay the big bucks!}
+{198}{}{You did good on that last job. Are you looking for another one?}
+{199}{}{Sure.}
+{200}{}{No.}
+{201}{}{Maybe.}
+{202}{}{Ug.}
+{203}{}{Lug.}
+{204}{}{Ready to go? We're leaving today.}
+{205}{}{Yep!}
+{206}{}{Nope.}
+{207}{}{Hrurra?}
+{208}{}{You're a day early. Come back tomorrow if you still want the job. If not, don't show. I don't give a crap.}
+{209}{}{Hey, you're early. I'm not sending them out until }
+{210}{}{3rd}
+{211}{}{17th}
+{212}{}{. Come back then if you still want the damn job. I really don't care if you show or not. But if you do still want it come back then.}
+{213}{}{Not yet.}
+{214}{}{Dad, shut the hell up!}
+{215}{}{Hey old man, I'm hungry! I'm going to take a break!}
+{216}{}{I hate working the books.}
+{217}{}{January }
+{218}{}{February }
+{219}{}{March }
+{220}{}{April }
+{221}{}{May }
+{222}{}{June }
+{223}{}{July }
+{224}{}{August }
+{225}{}{September }
+{226}{}{October }
+{227}{}{November }
+{228}{}{December }
+{229}{}{You get 3 drug packs.}
+# invasion - WastelandGhost@RestorationMOD
+{300}{}{So, that old bastard gave you the okay to work for us, eh? Well you must be one tough bitch. I'm impressed, but we'll see how you do. We leave only twice a month, on the 3rd and 17th. Can you handle that? We go to all the local hot spots: the Brotherhood of Steel, and Junktown. Boneyard and Necropolis have been invaded by some very unfriendly mutants, so we don't go there anymore.}
+{301}{}{So, that old bastard gave you the okay to work for us, eh? Well you must have balls of steel. I'm impressed, but we'll see how you do. We leave only twice a month, on the 3rd and 17th. Can you handle that? We go to all the local hot spots: the Brotherhood of Steel, and Junktown. Boneyard and Necropolis have been invaded by some very unfriendly mutants, so we don't go there anymore.}
+{302}{}{So, that old bastard gave you the okay to work for us, eh? Well you must be one tough bitch. I'm impressed, but we'll see how you do. We leave only twice a month, on the 3rd and 17th. Can you handle that? We go to all the local hot spots: the Brotherhood of Steel, Junktown and Necropolis. L.A. Boneyard has been invaded by some very unfriendly mutants, so we don't go there anymore.}
+{303}{}{So, that old bastard gave you the okay to work for us, eh? Well you must have balls of steel. I'm impressed, but we'll see how you do. We leave only twice a month, on the 3rd and 17th. Can you handle that? We go to all the local hot spots: the Brotherhood of Steel, Junktown and Necropolis. L.A. Boneyard has been invaded by some very unfriendly mutants, so we don't go there anymore.}
+{304}{}{So, that old bastard gave you the okay to work for us, eh? Well you must be one tough bitch. I'm impressed, but we'll see how you do. We leave only twice a month, on the 3rd and 17th. Can you handle that? We go to all the local hot spots: the Brotherhood of Steel, Junktown and L.A. Boneyard. Necropolis has been invaded by some very unfriendly mutants, so we don't go there anymore.}
+{305}{}{So, that old bastard gave you the okay to work for us, eh? Well you must have balls of steel. I'm impressed, but we'll see how you do. We leave only twice a month, on the 3rd and 17th. Can you handle that? We go to all the local hot spots: the Brotherhood of Steel, Junktown and L.A. Boneyard. Necropolis has been invaded by some very unfriendly mutants, so we don't go there anymore.}
