diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/KILDOOR1.MSG b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/KILDOOR1.MSG
index 0b8a15da0..0262cffb5 100644
--- a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/KILDOOR1.MSG
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/KILDOOR1.MSG
@@ -2,4 +2,12 @@
 {101}{}{You pick the lock.}
 {102}{}{You attempt to pick the lock, but fail.}
 {103}{}{While picking the lock, you manage to tweak the tumblers and jam it.}
-{104}{}{The door isn't locked.}
\ No newline at end of file
+{104}{}{The door isn't locked.}
+
+# Fo1in2
+{200}{}{ picks the lock.}
+{201}{}{ attempts to pick the lock, but fails.}
+{202}{}{While picking the lock, }
+{203}{}{ manages to tweak the tumblers and jams it.}
+
+
