diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/KILDOOR1.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/KILDOOR1.msg
new file mode 100644
index 000000000..a3451390b
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/KILDOOR1.msg
@@ -0,0 +1,11 @@
+{100}{}{The door is locked.}
+{101}{}{You pick the lock.}
+{102}{}{You attempt to pick the lock, but fail.}
+{103}{}{While picking the lock, you manage to tweak the tumblers and jam it.}
+{104}{}{The door isn't locked.}
+
+# Fo1in2
+{200}{}{ picks the lock.}
+{201}{}{ attempts to pick the lock, but fails.}
+{202}{}{While picking the lock, }
+{203}{}{ manages to tweak the tumblers and jams it.}
