diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/KILLSAFE.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/KILLSAFE.msg
new file mode 100644
index 000000000..c371eddb8
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/KILLSAFE.msg
@@ -0,0 +1,10 @@
+{100}{}{You see a safe.}
+{101}{}{The safe is not locked.}
+{103}{}{Got it!}
+{104}{}{Drat! Almost had it.}
+{105}{}{The safe is locked.}
+{106}{}{ experience for cracking Killian's personal safe, you big jerk.}
+{107}{}{You tweak the tumblers and manage to jam the lock with a pick. Just great.}
+
+# Fo1in2
+{200}{}{ tweaks the tumblers and manages to jam the lock with a pick. Just great.}
