diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/KILLSAFE.MSG b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/KILLSAFE.MSG
index 38a4fb71d..6d3d936c2 100644
--- a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/KILLSAFE.MSG
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/KILLSAFE.MSG
@@ -5,3 +5,6 @@
 {105}{}{The safe is locked.}
 {106}{}{ experience for cracking Killian's personal safe, you big jerk.}
 {107}{}{You tweak the tumblers and manage to jam the lock with a pick. Just great.}
+
+# Fo1in2
+{200}{}{ tweaks the tumblers and manages to jam the lock with a pick. Just great.}
