diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/KLAXON.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/KLAXON.msg
index be3528edf..f06026b22 100644
--- a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/KLAXON.msg
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/KLAXON.msg
@@ -1,5 +1,5 @@
-# Fo1in2
-# from pro_scen.msg:
-# line 90701 - This security light is currently pulsating a bright red.
-
-{100}{}{This security light is currently pulsating a bright red.}
+# Fo1in2
+# from pro_scen.msg:
+# line 90701 - This security light is currently pulsating a bright red.
+
+{100}{}{This security light is currently pulsating a bright red.}
