diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/LARYKNIG.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/LARYKNIG.msg
index 8ecd52509..be7e1c260 100644
--- a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/LARYKNIG.msg
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/LARYKNIG.msg
@@ -1,4 +1,5 @@
 {100}{}{You see a Knight.}
+{1001}{}{You see: Larry.}
 {101}{}{Hello Initiate. I'm Larry, how may I help you?}
 {102}{}{Vree said you could show me a laser pistol?}
 {103}{}{Vree said you could give me a laser pistol.}
