diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/LAURA.MSG b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/LAURA.MSG
index 766c6a965..0e09c1315 100644
--- a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/LAURA.MSG
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/LAURA.MSG
@@ -5,7 +5,7 @@
 {106}{}{Can I ask you a few questions about the Children?}
 {107}{}{What kind of "services" do you offer?}
 {108}{}{Mom?}
-{110}{laura01a}{That's very sweet of you, but no, I'm not. I'm sure you can find her somewhere else.}
+{110}{laura1a}{That's very sweet of you, but no, I'm not. I'm sure you can find her somewhere else.}
 {112}{laura2}{I most certainly am. Did you want something?}
 {113}{}{Nicole sent me to talk to you.}
 {114}{}{Uh... no. Never mind.}
@@ -64,7 +64,7 @@
 {191}{laura34}{Yes and no. I snuck up to the front one day and I found a strange mechanical device with a lens under the altar.}
 {192}{}{Probably a holo-projector of some kind.}
 {193}{}{So?}
-{195}{laura35}{Huh... believe me, it was at first, but then I snuck up and looked at the alter when nobody was looking. And there's a device underneath with wires and a big lens.}
+{195}{laura35}{Huh... believe me, it was at first, but then I snuck up and looked at the altar when nobody was looking. And there's a device underneath with wires and a big lens.}
 {196}{}{Probably a holo-projector of some kind.}
 {197}{}{So?}
 {199}{laura36}{So I'm pretty sure this Master is flesh and blood. And I also think I know where he is.}
@@ -182,7 +182,7 @@
 {341}{}{They have proven themselves to the Church. The rest of us aspire to that.}
 {342}{}{Oh yeah, these are the guys who've survived torture and who knows what else. Totally brainwashed and will do anything they're told.}
 {343}{}{They are the High Priests.}
-{344}{}{Ah, those are the High Priests. So far I think there's only 13, but I'm not really sure.}
+{344}{}{Ah, those are the High Priests. So far I think there's only thirteen, but I'm not really sure.}
 {345}{}{That's not important.}
 {346}{}{That's not important.}
 {347}{}{I'm sorry. I simply don't have time for this.}
