diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/LENNY.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/LENNY.msg
index 1340e17e6..3b0b11602 100644
--- a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/LENNY.msg
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/LENNY.msg
@@ -1,4 +1,4 @@
-{100}{}{You see Lenny.}
+{100}{}{You see: Lenny.}
 {101}{}{You see a loser who smells like a Brahmin.}
 {102}{}{[more]}
 {103}{}{[done]}
