diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/LEON.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/LEON.msg
new file mode 100644
index 000000000..c4f15b531
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/LEON.msg
@@ -0,0 +1,5 @@
+{100}{}{You see: Leone.}
+{101}{}{If you want company, rent a hooker. I'm busy.}
+{102}{}{Not until the boss says so. Just ignore him and he'll go away.}
+{103}{}{Not until the boss says so. Just ignore her and she'll go away.}
+{105}{}{Give me a break. Anyone can bust a knee or break fingers. It just don't have the shock like it used to.}
