diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/LEX.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/LEX.msg
index 4692ce8cb..da3af0574 100644
--- a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/LEX.msg
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/LEX.msg
@@ -1,4 +1,4 @@
-{100}{}{You see Lex.}
+{100}{}{You see: Lex.}
 {101}{}{You see a loser.}
 {102}{}{[more]}
 {103}{}{[done]}
