diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/LIBRARY.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/LIBRARY.msg
new file mode 100644
index 000000000..e2d16b16a
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/LIBRARY.msg
@@ -0,0 +1 @@
+{100}{}{This is the Vault library. Learning machines are connected to the central core. They teach the inhabitants knowledge and skills.}
