diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/LORRAINE.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/LORRAINE.msg
index 8b3b61cad..d9fd8cfa0 100644
--- a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/LORRAINE.msg
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/LORRAINE.msg
@@ -1,5 +1,5 @@
 {100}{}{You see a young woman with dark hair.}
-{101}{}{You see Lorraine, a scav from Adytum.}
+{101}{}{You see: Lorraine, a scav from Adytum.}
 {102}{}{Hi. Are you new here?}
 {103}{}{Yeah. Can you fill me in a little?}
 {104}{}{Yes, I am. I'm called }
