diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/LOSER.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/LOSER.msg
new file mode 100644
index 000000000..c365cf2db
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/LOSER.msg
@@ -0,0 +1,8 @@
+{100}{}{Got any food?}
+{101}{}{Boy, am I hungry. I wish I had some brahma burgers.}
+{102}{}{Please, mister, give me some food.}
+{103}{}{Please, ma'am, give me some food.}
+{104}{}{I really wish I could find some food around here.}
+{105}{}{I am starving!}
+{106}{}{I will work for food.}
+{107}{}{Foooood... give food now!}
