diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/MACRAE.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/MACRAE.msg
new file mode 100644
index 000000000..eb0d77169
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/MACRAE.msg
@@ -0,0 +1,54 @@
+{100}{}{You see a strong, stout man with a bushy beard.}
+{101}{}{You see: MacRae.}
+{102}{}{Hello lad, what might you be looking for today?}
+{103}{}{Hello lass, what might you be looking for today?}
+{104}{}{Ai, What else would you be looking to know?}
+{105}{}{Who are you?}
+{106}{}{What can you tell me about the Blades?}
+{107}{}{Nothing, thanks.}
+{108}{}{Well, I'm MacRae. I help the Blades some against the Deathclaws and that Regulator bunch. Teach'em a bit about protectin 'emselves against those nasties.}
+{109}{}{What kind of stuff do you teach them?}
+{110}{}{Have you seen the Deathclaws?}
+{111}{}{What's the deal with the Regulators}
+{112}{}{Can I ask you something else?}
+{113}{}{Well m' specialty is fightin' with m' fists, and takin' a punch or two more than 
+the guy throwin 'em at me.}
+{114}{}{Sounds like a dumb idea, I'd rather use a gun.}
+{115}{}{Do you think you could teach me anything?}
+{116}{}{Can I ask you something else?}
+{117}{}{Well if that's how you be feeling, you can just turn yourself around and walk away.}
+{118}{}{You seem to be an alright lad, but me times busy with keeping the Blades schooled. If things were to straighten themselves, I might be havin' the time then.}
+{119}{}{Alright, can I ask you something else?}
+{120}{}{Thanks for the info.}
+{121}{}{Seen em I have. Ran like hell, I did. If yer lookin' for 'em, you'll find 'em over there. 
+[MacRae points over to the east]}
+{122}{}{I was thinkin' the Deathclaws were the spawn of an unnatural act, but 'em Regulators are evil. 
+Come once a month or so and they shows their ugly faces and drag a father away from is wee ones.}
+{123}{}{Anything I could do to help?}
+{124}{}{Sounds bad, could I ask you another question?}
+{125}{}{Thanks for the info.}
+{126}{}{You be wantin' to talk to Razor. A good lass if there ever was one.}
+{127}{}{A good group of people is what the Blades are. Stuck 'ere between the damn Regulators and the 
+Deathclaws ain't meant the happiest life for 'em, but they survive.}
+{128}{}{[MacRae claps you on the back] Thanks to you, the Blades are going to live a better life. So, 
+what can an old Scotsman do for ya?}
+{129}{}{You had said earlier that if you had some free time, you could teach me a few things.}
+{130}{}{What have you been doing lately?}
+{131}{}{What do you do around here?}
+{132}{}{Not a problem.}
+{133}{}{Well it's been a wee bit quiet since the Blades helped out Adytum with the Regulators. Me 
+teaching services haven't been in such demand.}
+{134}{}{What do you teach?}
+{135}{}{Sorry to hear that, hope things get more interesting?}
+{136}{}{Could you teach me?}
+{137}{}{Sounds interesting, but I gotta get going.}
+{138}{}{Well I guess I do have the time. Might as well start right now. [MacRae starts showing you some 
+fighting stances, and ways to take punches like a true Scotsman.]}
+{139}{}{You be a quick learner. Hope that helps you in your travels.}
+{140}{}{Well I used to be helpin' the Blades out by teachin' them a few things about protectin' themselves 
+against the Deathclaws and the Regulators. Now with you takin' care of all that, things have been a wee 
+bit on the boring side lately.}
+{141}{}{Good seein' ya.}
+
+# Fo1in2
+{1180}{}{You seem to be an alright lass, but me times busy with keeping the Blades schooled. If things were to straighten themselves, I might be havin' the time then.}
