diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/MANGLED.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/MANGLED.msg
new file mode 100644
index 000000000..0d9921347
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/MANGLED.msg
@@ -0,0 +1,2 @@
+{100}{}{You notice bloody tracks leading to the bodies.}
+{101}{}{It looks like some of the animals dragged the bodies here.}
