diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/MBOUT2IN.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/MBOUT2IN.msg
new file mode 100644
index 000000000..ad9f9a0dc
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/MBOUT2IN.msg
@@ -0,0 +1,7 @@
+{100}{}{The door is locked. A security code is required to unlock it.}
+{101}{}{You enter the code from the tape, and unlock the door.}
+{102}{}{The door is not locked.}
+{103}{}{You bypass the security code and unlock the door.}
+{104}{}{You are unable to unlock the door.}
+{105}{}{You are unable to unlock the door. This might be easier if you used a set of electronic lock-picking tools on it.}
+{106}{}{The explosion is powerful, but did not even dent or mark the security door.}
