diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/MBVATS.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/MBVATS.msg
new file mode 100644
index 000000000..84d9e521d
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/MBVATS.msg
@@ -0,0 +1,13 @@
+{100}{}{off}
+{101}{}{Time left: }
+{102}{}{ seconds.}
+{103}{}{This is where you would be considered dead.}
+{104}{}{off}
+{105}{}{on}
+{106}{}{off}
+{107}{}{Time left: }
+{108}{}{ seconds.}
+{109}{}{This is where you would be considered dead.}
+{110}{}{Time left: }
+{111}{}{ seconds.}
+{112}{}{This is where you would be considered dead.}
