diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/MEDLAB.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/MEDLAB.msg
new file mode 100644
index 000000000..cb6c387df
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/MEDLAB.msg
@@ -0,0 +1 @@
+{100}{}{This area has been previously looted. There is a large amount of damage to the delicate equipment.}
