diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/MEETING.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/MEETING.msg
new file mode 100644
index 000000000..e582b9c8c
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/MEETING.msg
@@ -0,0 +1 @@
+{100}{}{You know this to be the meeting room for the Vault. Where the emergency supply cabinet would be is buried under tons of rock.}
