diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/MIKE.MSG b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/MIKE.MSG
index 6ca5b622f..c70b41967 100644
--- a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/MIKE.MSG
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/MIKE.MSG
@@ -12,3 +12,7 @@
 {111}{}{Don't you be mouthing off to me, punk. Now I'm gonna have to put you in the ground.}
 {112}{}{I'm not from around here. What part of town am I in?}
 {113}{}{Never mind. Bye.}
+
+# Fo1in
+# Name shown in the dialog screen / review interface
+{200}{}{Mike}
\ No newline at end of file
