diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/MOATCHEK.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/MOATCHEK.msg
new file mode 100644
index 000000000..78164c603
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/MOATCHEK.msg
@@ -0,0 +1,26 @@
+#############################################################
+#  Message File: ktgoo.msg                                  #
+#                                                           #
+#  Log:                                                     #
+#                                                           #
+#  Created: March 1, 1998                                   #
+#                                                           #
+#  Updated:                                                 #
+#############################################################
+
+#############################################################
+#  The following lines are for describing the NPC when      #
+#  glanced and and examined.                                #
+#############################################################
+{100}{}{You see Toxic Goo.}
+{101}{}{You see Toxic Goo.}
+
+#############################################################
+#  The following lines are floaters for citizens            #
+#############################################################
+
+{110}{}{Ouch - that goo burns.}
+{120}{}{These old rubber boots sure keep that nasty looking slime out.}
+{130}{}{Your boots just melted.}
+{140}{}{Your feet start to burn and itch.}
+{150}{}{Your feet burn and itch for a little while, but then they feel better.}
