diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/MRHANDYA.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/MRHANDYA.msg
new file mode 100644
index 000000000..38725e0b3
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/MRHANDYA.msg
@@ -0,0 +1,4 @@
+{100}{}{You see a broken robot.}
+{101}{}{You see a functional robot.}
+{102}{}{You fix the broken robot. It immediately powers up and comes to life.}
+{103}{}{The repairs are beyond your skills.}
