diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/MUTANT2.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/MUTANT2.msg
new file mode 100644
index 000000000..93fb470c5
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/MUTANT2.msg
@@ -0,0 +1,3 @@
+{100}{}{It's some kind of huge Super Mutant creature! And it has a very large gun!}
+{101}{}{Take him to Harry. He'll know what to do with him.}
+{102}{}{We should take her to Harry. He has plans for all the women.}
