diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/MUTDISK.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/MUTDISK.msg
new file mode 100644
index 000000000..d3a8248e1
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/MUTDISK.msg
@@ -0,0 +1,9 @@
+{100}{}{You see a disk with some strange markings on it, and stains that you think might be blood.}
+{101}{}{Pip-Boy 2000 displays: "Downloading ... Aborting, data already exists!"}
+{102}{}{Pip-Boy 2000 displays: "Downloading data ... ... ... Done. Have a nice day!"}
+{103}{}{This holodisk looks like it was used to record audio data from a military radio relay.}
+#
+# for Fallout Fixt:
+#
+{104}{}{You see: Super Mutant Radio Transmissions Disk.}
+{105}{}{It has some strange markings on it, and stains that you think might be blood.}
