diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/MUTEJAR.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/MUTEJAR.msg
new file mode 100644
index 000000000..efbe8fdbd
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/MUTEJAR.msg
@@ -0,0 +1,2 @@
+{100}{}{You notice some oddly-shaped fruit inside the jar.}
+{101}{}{You take the odd fruit from the jar.}
