diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/MVBSHELF.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/MVBSHELF.msg
new file mode 100644
index 000000000..baf6e1767
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/MVBSHELF.msg
@@ -0,0 +1,2 @@
+{100}{}{The bookshelf is filled with religious and philosophical books. It is very dusty.}
+{101}{}{You take a book down, examine it, and quickly replace it on the shelf. The shelf itself seems very sturdy.}
