diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/MVNEWS.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/MVNEWS.msg
new file mode 100644
index 000000000..8d5f4d7b7
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/MVNEWS.msg
@@ -0,0 +1,2 @@
+{100}{}{The newspaper headline reads: "Daily Life in the Village".}
+{101}{}{The newspaper machine is inexplicably jammed shut. It is very frustrating.}
