diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/MVRCRDOR.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/MVRCRDOR.msg
new file mode 100644
index 000000000..7a2ffde73
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/MVRCRDOR.msg
@@ -0,0 +1,5 @@
+{100}{}{You see a steel door. A sign on it reads: "Reactor Core Control Room. Authorized Personnel Only."}
+{101}{}{The door is locked electronically.}
+{102}{}{The door is not locked.}
+{103}{}{You manage to bypass the electronic lock.}
+{104}{}{Your attempt to circumvent the electronic lock fails.}
