diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/MYSTSTRN.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/MYSTSTRN.msg
new file mode 100644
index 000000000..6c5154342
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/MYSTSTRN.msg
@@ -0,0 +1,6 @@
+{100}{}{You see a helpful stranger.}
+{101}{}{You see the mysterious stranger.}
+{102}{}{I will help thee.}
+{103}{}{I don't talk much.}
+{104}{}{Looks like you could use some assistance.}
+{105}{}{Let's rock!}
