diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/NEAL.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/NEAL.msg
new file mode 100644
index 000000000..dd41f468d
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/NEAL.msg
@@ -0,0 +1,63 @@
+{100}{}{You see: Neal, the bartender.}
+{101}{}{You see a bartender.}
+{102}{}{He looks overworked.}
+{103}{}{Hey! Don't think you're going to get away with that, thief!}
+{104}{}{We're closed. Come back around four o'clock.}
+{105}{}{Zzzz}
+{106}{}{Welcome to the Skum Pitt. What can I do ya for?}
+{107}{}{Uh, I'll have a drink.}
+{108}{}{I'm looking for information.}
+{109}{}{My name's }
+{110}{}{. What is this place?}
+{111}{}{Gaa hnnngh!}
+{112}{}{Great. Whaddya want?}
+
+{113}{}{Nuka-Cola, 3 caps.}
+{114}{}{Beer, 5 caps.}
+{115}{}{The hard stuff, 20 caps.}
+
+{116}{}{Okay, so whaddya wanna know?}
+{117}{}{What's the scoop on this town?}
+{118}{}{Who's the dangerous-looking stranger in the corner?}
+{119}{}{Who're the jerks?}
+{120}{}{[Neal looks at you quizzically for a moment] Haven't you ever seen a bar before, stranger? This is my place, the Skum Pitt. You buy drinks here. And you tip me.}
+{121}{}{Uh, right. I knew that. So, can I get a drink?}
+{122}{}{Yeah. Um, can I get some other information?}
+{123}{}{Oh. Thanks.}
+{124}{}{Yeah, same to you, buddy.}
+{125}{}{What's that trophy on the shelf?}
+{126}{}{Well, just over there to the west of us is Gizmo's casino. I'd be surprised if you didn't know about it; most of the people here are the ones who just finished losing their money there. Over in the other part of town you got Killian's general store, and Marcelles's Crash House. Killian's a good guy, runs a straight joint, carries everything you need. The Crash House is where you sleep it off when you've been in here for too long. [Neal chuckles]}
+{127}{}{Thanks. Can I get a drink?}
+{128}{}{Thanks for the information. See you around.}
+{129}{}{Fella's called Tycho. Apparently a Ranger from back east or something. Pretty nasty fighter, from what I hear, but a straight-up character.}
+{130}{}{Mind if I ask some other questions?}
+{131}{}{Oh, them. They're the Skulz. They're a local gang. All the riff-raff. Sometimes they start fights or cause a little trouble. Usually Killian keeps them in line, though.}
+{132}{}{Here ya go. Enjoy.}
+{133}{}{Hey! You stole my urn!}
+{134}{}{That's not a trophy, it's an urn. It contains my wife's ashes - besides this bar, it is the most important thing in my life.}
+{135}{}{Anything else?}
+{136}{}{Looking for some more information.}
+{137}{}{No thanks.}
+{138}{}{Oh.}
+{139}{}{Yeah, I'll have a drink.}
+{140}{}{Come back when you've got some money.}
+{141}{}{What'll it be, friend?}
+{142}{}{What happened to that trophy you used to have up there?}
+{143}{}{Thank you. You can't know what this means to me. You'll never pay for a drink in this bar again.}
+{144}{}{I've got your urn back for you.}
+{145}{}{Some slime stole it. It wasn't a trophy, it was an urn. It contained my wife's ashes - besides this bar, it was the most important thing in my life.}
+{146}{}{Get the hell outta here you goddamn punks! If any of ya ever touch her again I'll kill the whole lot o' ya!}
+{147}{}{The day I can't handle little shits like you is the day I die!}
+{148}{}{What the hell ya doin'? We're closed. Get on out of here!}
+{149}{}{Ya damn kids, I'll show you a thing or two!}
+
+# Fo1in2
+{200}{}{What are you doing here? Get out!}
+{201}{}{What?}
+{202}{}{Again?!}
+{203}{}{Must have been my imagination ...}
+
+# Free drinks:
+{210}{}{Nuka-Cola.}
+{211}{}{Beer.}
+{212}{}{The hard stuff.}
