diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/NGARFRDG.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/NGARFRDG.msg
new file mode 100644
index 000000000..cd0411bca
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/NGARFRDG.msg
@@ -0,0 +1,9 @@
+{100}{}{The fridge is locked.}
+{101}{}{The fridge is already unlocked.}
+{102}{}{You unlock the fridge.}
+{103}{}{You haven't quite gotten this lock yet.}
+{104}{}{Someone left the fridge open!}
+
+# Fo1in2
+{200}{}{ unlocks the fridge.}
+{201}{}{ hasn't quite gotten this lock yet.}
