diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/NICOLE.MSG b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/NICOLE.MSG
index ba17976e4..514272a67 100644
--- a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/NICOLE.MSG
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/NICOLE.MSG
@@ -189,7 +189,7 @@
 {285}{}{That was all for now. I gotta go.}
 
 # 286 changed with patch 1.3 from TeamX
-{286}{}{Please, we're busy. Don't you have things to do?}
+{286}{Nic_72}{Please, we're busy. Don't you have things to do?}
 {287}{Nic_74}{What are you doing? Put that down!}
 {288}{Nic_75}{Stop!}
 {289}{Nic_76}{Stop that!}
