diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/NIF2HINT.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/NIF2HINT.msg
new file mode 100644
index 000000000..0890d88f6
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/NIF2HINT.msg
@@ -0,0 +1 @@
+{100}{}{The book contains TONS of inside information on Fallout, more than enough to modify yourself for the better.}
