diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/ORFEO.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/ORFEO.msg
new file mode 100644
index 000000000..cfc51a51d
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/ORFEO.msg
@@ -0,0 +1,8 @@
+{100}{}{You see: Orfeo, a Senior Member of the Children of the Cathedral.}
+{101}{}{Ah. A new convert. Welcome.}
+{102}{}{Me kill!!!}
+{103}{}{I am not a convert!}
+{104}{}{Thank you, Brother.}
+{105}{}{Then we shall take care of that problem shortly.}
+{106}{}{We must first cleanse you from the impurities of the wastelands. Come.}
+{107}{}{We will cleanse you!}
