diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/PENSMELL.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/PENSMELL.msg
new file mode 100644
index 000000000..f1c55c476
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/PENSMELL.msg
@@ -0,0 +1,2 @@
+{100}{}{The stench of the Brahma Pen is unlike any smell you
+ have ever encountered in your Vault.}
