diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/PHRAX.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/PHRAX.msg
index 31bbe2260..d6ddd2d60 100644
--- a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/PHRAX.msg
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/PHRAX.msg
@@ -1,4 +1,4 @@
-{100}{}{You see Phrax of the Crypts.}
+{100}{}{You see: Phrax of the Crypts.}
 {101}{}{That fridge ain’t your property.  I would highly suggest not touching it.}
 {102}{}{Why not?}
 {103}{}{You gonna make me?}
