diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/RADIOCOM.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/RADIOCOM.msg
new file mode 100644
index 000000000..52f0d61c3
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/RADIOCOM.msg
@@ -0,0 +1,6 @@
+{100}{}{The computer says: RADIO CONTROL.}
+{101}{}{ It looks like the radio channels have been shut down.}
+{102}{}{ Radio channel control seems to be on.}
+{103}{}{You switch the radio channels on.}
+{104}{}{You switch the radio channels off.}
+{105}{}{You flip some switches but can't get the radio channel selectors to change.}
