diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/RAE.MSG b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/RAE.MSG
index 72a82a3f8..75f0a4006 100644
--- a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/RAE.MSG
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/RAE.MSG
@@ -83,4 +83,5 @@
 # You gain x experience ...
 {200}{}{ for convincnig Rae to destroy the Master.}
 {201}{}{ for convincing Rae to fight with you against the Master.}
+{202}{}{It is done. Leave now.}
 
