diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/RAZOR.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/RAZOR.msg
index 40933d753..69b70a0e3 100644
--- a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/RAZOR.msg
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/RAZOR.msg
@@ -154,3 +154,4 @@
 	{401}{}{I've changed my mind. You have to deal with this on your own.}
 	{402}{}{It's not worth it. We should attack the Regulators now.}
 	{403}{}{Not yet. I'm on my way now.}
+	{404}{}{Give me the holodisk. He won't believe me otherwise.}
