diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/REGDISK.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/REGDISK.msg
new file mode 100644
index 000000000..2f819e47d
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/REGDISK.msg
@@ -0,0 +1,9 @@
+# This file is new for Fallout Fixt.
+#
+{100}{}{You see the holodisk that Razor gave you.}
+#
+{101}{}{Pip-Boy 2000 displays: "Downloading ... Aborting, data already exists!"}
+{102}{}{Pip-Boy 2000 displays: "Downloading data ... ... ... Done. Have a nice day!"}
+#
+{103}{}{This disk contains information about the real fate of Jon Zimmerman's son.}
+{104}{}{You see: Regulator Transmissions Disk.}
