diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/RIPPPRIS.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/RIPPPRIS.msg
new file mode 100644
index 000000000..7a68baa1d
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/RIPPPRIS.msg
@@ -0,0 +1,7 @@
+{100}{}{You see a hapless prisoner.}
+{101}{}{These people are crazy here. Help me get out of here!}
+{102}{}{No way!}
+{103}{}{Die!}
+{104}{}{Okay.}
+{105}{}{Oh, thank you. Thank you. Goodbye!}
+{106}{}{But... }
