diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/RNDMTN.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/RNDMTN.msg
new file mode 100644
index 000000000..a8b8efbf5
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/RNDMTN.msg
@@ -0,0 +1,107 @@
+# NORTH ENCOUNTERS
+
+{105}{}{Radscorpions!!}
+{121}{}{You see a lone traveler in the distance.}
+{106}{}{A particularly sneaky radscorpion seems to be trying to ambush you.}
+{107}{}{Out of nowhere, a radscorpion suddenly strikes!}
+{108}{}{You have stumbled upon a family of mole rats.}
+{100}{}{You are nearly crushed by a large boulder in an avalanche.}
+{102}{}{The displaced rock brings with it the remains of a long-dead traveler.}
+
+
+# SOUTH ENCOUNTERS
+
+#first
+#114 You are traveling through sheer cliffs and rugged terrain.
+#100 You are nearly crushed by a large boulder in an avalanche.
+
+{136}{}{You stumble onto a pack of hostile radscorpions.}
+
+{120}{}{You find several rats scavenging about here.}
+
+#CHANGE THIS ONE		#patrick
+
+{122}{}{The ground ahead is littered with debris and bodies. It looks like an old battle zone.}
+
+
+
+# VAULT ENCOUNTERS
+
+{128}{}{You see a lone radscorpion, skittering about.}
+
+{135}{}{You come across the bones of a long-dead traveler.}
+
+{129}{}{Tremendous mole rats are nosing about here.}
+
+{130}{}{You find several rats scavenging about here.}
+
+#fall or slip
+
+{134}{}{You see several rats running toward you.}
+
+
+#
+# TERRAIN ENCOUNTERS (NO MONSTERS)
+#
+{114}{}{You are traveling through sheer cliffs and rugged terrain.}
+{115}{}{Fortunately, your climbing skills are up to the task.}
+{116}{}{You find a cave with a small spring and manage to get your first good night's sleep in a long time.}
+{117}{}{You fall from a ledge, landing on several sharp stones.}
+{118}{}{You fall from a cliffside to the rocky ground below.}
+{109}{}{You are overtaken by a landslide.}
+{123}{}{You lose your footing on the rocky terrain.}
+{132}{}{You slip on the uneven terrain and twist an ankle, painfully. It makes several unhealthy sounds.}
+{133}{}{You fall over the rough terrain and scrape some flesh.}
+#
+# No Damage
+#
+{101}{}{You escape with no damage.}
+{110}{}{Amazingly, you escape with no damage.}
+{124}{}{You skillfully escape with no damage.}
+{131}{}{You nearly fall over some rough terrain. They could use some environment management out here.}
+#
+# Small Damage
+#
+{104}{}{Some rock shards slice your face and hands, but you escape with minimal damage.}
+{113}{}{Some rocks shards slice your face and hands, but you escape with minimal damage.}
+{127}{}{Some rock shards slice your face and hands, but you escape with minimal damage.}
+#
+# Big Damage
+#
+{147}{}{Your pitiful attempt to escape the rocks is a colossal failure. All of your limbs are crushed and battered in a spectacular spray of blood. Amazingly, you survive.}
+{148}{}{You fail to find any handholds during the fall, and land with a mighty crash, crushing both your legs like radroaches.}
+#
+# Damage Numbers
+#
+{138}{}{You sustain }
+{139}{}{ points of damage and a crippled left leg.}
+{140}{}{ points of damage and a crippled right leg.}
+{141}{}{ points of damage.}
+{144}{}{ points of damage and a crippled left arm.}
+{145}{}{ points of damage and a crippled right arm.}
+{146}{}{ points of damage and two crippled limbs.}
+#
+# Cripples
+#
+{103}{}{Your leg is critically damaged by falling rocks.}
+{112}{}{Your leg is critically damaged by falling rocks.}
+{126}{}{Your leg is critically damaged by falling rocks.}
+#
+# Critical Success
+#
+{125}{}{You escape with no damage. The displaced rock reveals a gold deposit!}
+{111}{}{The displaced rock reveals a gold deposit!}
+{142}{}{You refill your empty flask in the spring.}
+{143}{}{You refill your empty flasks in the spring.}
+#
+#
+# MONSTERS - not used
+#
+{119}{}{You stumble onto a large pack of radscorpions.}
+
+# Fo1in2:
+{200}{}{You stumble onto a pack of geckos.}
+{201}{}{You stumble onto a pack of hostile geckos.}
+{202}{}{You find several geckos scavenging about here.}
+{203}{}{You find yourself among spore plants!}
+{204}{}{Because you used a rope while climbing, you manage to avoid worse injuries.}
