diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/ROBOT.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/ROBOT.msg
new file mode 100644
index 000000000..e320a6b74
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/ROBOT.msg
@@ -0,0 +1,6 @@
+{100}{}{[Click] Peace to you. [Beep!]}
+{101}{}{Exterminate. Exterminate.}
+{102}{}{Kill the human!}
+{103}{}{Kill the intruder!}
+{104}{}{[Brrzk!] Death to flesh!}
+{105}{}{[Breep!] No peace. No peace.}
