diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/ROPE.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/ROPE.msg
new file mode 100644
index 000000000..5aae2d765
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/ROPE.msg
@@ -0,0 +1,7 @@
+{100}{}{This elevator shaft cannot be descended without a rope.}
+{101}{}{You successfully sling the rope onto and down the elevator shaft.}
+{102}{}{You cannot climb down the pit using just the metal beam.}
+{103}{}{You successfully attach the rope to the metal beam.}
+{104}{}{This elevator shaft cannot be descended without a rope.}
+{105}{}{You successfully sling the rope onto and down the elevator shaft.}
+{106}{}{There is no need to secure another rope to the elevator shaft.}
