diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/RUBCHIP.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/RUBCHIP.msg
new file mode 100644
index 000000000..9c75a996c
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/RUBCHIP.msg
@@ -0,0 +1,4 @@
+{100}{}{It seems that the Command and Control Center is buried under even more rock. You will have to look for the Water Chip elsewhere.}
+{101}{}{The large amount of rubble blocking this corridor looks impressive. You don't think that it could be moved, even with high-yield explosives.}
+{102}{}{You earn }
+{103}{}{ experience.}
