diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/SECDOOR.MSG b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/SECDOOR.MSG
index 81241dee1..f25a89668 100644
--- a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/SECDOOR.MSG
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/SECDOOR.MSG
@@ -5,3 +5,10 @@
 {104}{}{You manage to trigger the passive alert on the door, clamping down the electronic lock. Good job, chromo.}
 {105}{}{You earn }
 {106}{}{ experience for picking the lock.}
+
+# Fo1in2
+{200}{}{ manages to unlock the security door.}
+{201}{}{ fails to unlock the security door.}
+{202}{}{ manages to trigger the passive alert on the door, clamping down the electronic lock.}
+
+
