diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/SET.MSG b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/SET.MSG
index fb902f89c..d64d73591 100644
--- a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/SET.MSG
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/SET.MSG
@@ -199,7 +199,7 @@
 {298}{Set203}{Natter to Garret. My time flies.}
 {299}{Set204}{Is your head empty?}
 {300}{Set205}{Yours is the only benefit. This exchange lacks.}
-{301}{Set206}{Thoughts not in my possession}
+{301}{Set206}{Thoughts not in my possession.}
 {302}{Set207}{The time is wrong. Evening is proper for your shadow.}
 {303}{Set208}{Guards! Seize the norm!}
 {304}{Set209}{Guards, end this annoyance!}
@@ -223,7 +223,7 @@
 #
 {970}{Set01}{Simple time. I'm Set. I'm in charge. Get it?}
 
-{980}{Set206}{Thoughts not in my possession}
+{980}{Set206}{Thoughts not in my possession.}
 {981}{Set310}{Beneath my thoughts.}
 {982}{Set311}{Not in my thoughts.}
 {983}{Set204}{Is your head empty?}
