diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/SETDESK.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/SETDESK.msg
new file mode 100644
index 000000000..450662c19
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/SETDESK.msg
@@ -0,0 +1,2 @@
+{100}{}{You find nothing of importance.}
+{101}{}{You find a note from someone called the Lieutenant to Set. It orders him to watch for pure-strain humans.}
