diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/SETH.MSG b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/SETH.MSG
index 7be20d0e4..f27c5e4df 100644
--- a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/SETH.MSG
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/SETH.MSG
@@ -74,7 +74,7 @@
 {1005}{}{Water}
 {1006}{}{Tandi}
 {1007}{}{Aradesh}
-{1008}{}{Seth}
+{1008}{}{Seth|Set}
 {1009}{}{Razlo}
 {1010}{}{Sands}
 {1011}{}{Scorpion}
