diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/SETH.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/SETH.msg
index 994f98d0b..3ab36a13a 100644
--- a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/SETH.msg
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/SETH.msg
@@ -103,7 +103,7 @@
 {2000}{}{ I can't go, but I can show you the way.}
 {2010}{}{[More]}
 {2020}{}{Great. Show me the way.}
-{2100}{}{[With the help of Seth's directions you mark the cavern on the map in your PipBoy]}
+{2100}{}{[With Seth's directions, you mark the cavern on the map in your Pip-Boy]}
 
 {2200}{}{I wish you'd save Tandi first, stranger, but, if you're willing to help us with the radscorpions ...}
 {2210}{}{I'll be back soon. Goodbye.}
