diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/SETSCTDR.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/SETSCTDR.msg
new file mode 100644
index 000000000..6fdcc26bf
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/SETSCTDR.msg
@@ -0,0 +1,10 @@
+{100}{}{You find a crude explosive device in the doorframe.}
+{101}{}{You set off an explosive trap!}
+{102}{}{You disarm the trap.}
+{103}{}{You did not disarm the trap.}
+{104}{}{You accidentally set off the explosive!}
+{105}{}{You accidentally set off the explosive, because you were smart enough to find it, but not smart enough to leave it alone.}
+{106}{}{Ooof!}
+{107}{}{Aargh!}
+{200}{}{You earn }
+{201}{}{ experience for finding the secret door.}
