diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/SHADDESK.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/SHADDESK.msg
new file mode 100644
index 000000000..88419b2af
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/SHADDESK.msg
@@ -0,0 +1,2 @@
+{100}{}{You notice a desk cluttered with papers.}
+{101}{}{The papers detail reports on raider and radscorpion attacks, a list of Shady Sands' citizens, and farming information.}
