diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/SHARK.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/SHARK.msg
new file mode 100644
index 000000000..32b8a7a4c
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/SHARK.msg
@@ -0,0 +1,36 @@
+{100}{}{You see: Shark.}
+{101}{}{You see a man with filed teeth.}
+{102}{}{You bitch!}
+{103}{}{Hey, fresh bait! So, what brings you in to our territory, chum? Here for a little fun? At your expense, of course.}
+{104}{}{Actually, I'm here to kick butt and chew gum... you can guess the rest.}
+{105}{}{What do you do for fun around here?}
+{106}{}{Uh, I was just leaving. Bye.}
+{107}{}{Just stopped by for a drink and some... entertainment.}
+{108}{}{Glopf!}
+{109}{}{We beat up posers like you!}
+{110}{}{Not so fast! Nobody just walks into our place without payin' the pain tax. }
+{111}{}{I think you'll do. Boys, grab her.}
+{112}{}{Let me introduce you to the tax man. He's called Mr. Sledgehammer!}
+{113}{}{I don't think so!}
+{114}{}{Well, for this evening's entertainment, we've decided to thrash this place! Heh. Grab what you can carry and let's see some blood run, eh?}
+{115}{}{Think I'll just grab a brew.}
+{116}{}{Smash and grab? Count me in!}
+{117}{}{Not tonight, I have a headache.}
+{118}{}{Nah, I'd rather have something to eat... perhaps grilled shark!}
+{119}{}{Ah, you loser. Go run back to your momma.}
+{120}{}{You again? Geez, some people just don't learn!}
+{121}{}{Just stopped in for a quick drink.}
+{122}{}{Just leaving, never mind.}
+{123}{}{Glorpf!}
+{124}{}{Not a bad idea. Think I'll have one myself. Of course, you always have to wash a beer down with some bloodshed!}
+{125}{}{I thought you'd get the picture by now to beat it. Guess I was wrong. Well, at least it'll be fun to beat you!}
+{126}{}{We'll be back, old man. This isn't over.}
+
+# TMA
+{970}{}{That's me.}
+{971}{}{Who's asking?}
+{972}{}{You lookin' for trouble?}
+
+{980}{}{Are you trying to annoy me? Beat it!}
+{981}{}{Get out of my sight.}
+{982}{}{Piss off.}
