diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/SKIPPY.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/SKIPPY.msg
index bfb0f2021..d0f5b9c71 100644
--- a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/SKIPPY.msg
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/SKIPPY.msg
@@ -1,4 +1,4 @@
-{100}{}{You see Skippy.}
+{100}{}{You see: Skippy.}
 {101}{}{You see a ganger.}
 {102}{}{He is tossing what looks like an exotic knife up in the air.}
 {103}{}{[more]}
