diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/SKIZZER.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/SKIZZER.msg
index 6e6794af7..7846256f4 100644
--- a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/SKIZZER.msg
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/SKIZZER.msg
@@ -1,4 +1,4 @@
-{100}{}{You see Skizzer.}
+{100}{}{You see: Skizzer.}
 {101}{}{You see a man.}
 {102}{}{Fellow Crypter, you doing fine.}
 {103}{}{You some dumb Fool.}
