diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/SKUMDOOR.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/SKUMDOOR.msg
new file mode 100644
index 000000000..3de437888
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/SKUMDOOR.msg
@@ -0,0 +1,10 @@
+{100}{}{The door to the Skum Pitt is locked.}
+{101}{}{You manage to unlock the door.}
+{102}{}{You fiddle with the door, but fail to unlock it.}
+{103}{}{You hear something snap. Looks like the lock is jammed.}
+{104}{}{The door isn't locked.}
+
+# Fo1in2
+{200}{}{ manages to unlock the door.}
+{201}{}{ fiddles with the door, but fails to unlock it.}
+{202}{}{ hears something snap. Looks like the lock is jammed.}
