diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/SLAPPY.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/SLAPPY.msg
index f2eb891e6..f0e137ad3 100644
--- a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/SLAPPY.msg
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/SLAPPY.msg
@@ -39,9 +39,9 @@
 # Fo1in2:
 {150}{}{You earn }
 {151}{}{ experience for learning about the Deathclaw.}
-{2000}{}{Ohhh! No. No... No... No... Not going to see the Deathclaw. No.}
+{2000}{}{Ohhh! No. No no no no. Not *going* to the Deathclaw.}
 {2010}{}{Just shut up and show me on my PipBoy where it is.}
-{2100}{}{Ahh. Show! Da... Da... Da...}
+{2100}{}{Ahh. Show! Here... Here... Here...}
 {2110}{}{[More]}
 {2200}{}{[It takes a while to figure out Slappy's directions, but eventually you are able to nail it down to the mountain range west of the Hub]}
 
