diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/SSRADSCO.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/SSRADSCO.msg
new file mode 100644
index 000000000..92f73d6c5
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/SSRADSCO.msg
@@ -0,0 +1,8 @@
+# Now split to allow different exp reward amount:  (Sduibek)
+{100}{}{You earn }
+{101}{}{ experience for protecting Shady Sands by killing all the radscorpions.}
+{102}{}{It looks to have originated from the North American Emperor scorpion.}
+# From GENCHAT.MSG:  (Sduibek)
+{188}{}{The radscorpions are now sealed in their cave, no longer posing a threat to Shady Sands.}
+{189}{}{}
+{200}{}{The cave mouth beckons you deeper into the lair of the radscorpions...}
