diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/SSWELL.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/SSWELL.msg
new file mode 100644
index 000000000..e783f65d8
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/SSWELL.msg
@@ -0,0 +1,4 @@
+{100}{}{You see the water supply to Shady Sands. This is too little water to supply your Vault.}
+{101}{}{You have no reason to climb down the well, but perhaps the rope will be useful for climbing down holes.}
+{102}{}{You learn nothing more from the well.}
+{103}{}{You make a wish as you hear the bottle cap splash into the water.}
