diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/SUPER.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/SUPER.msg
new file mode 100644
index 000000000..970bd3b35
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/SUPER.msg
@@ -0,0 +1,5 @@
+{100}{}{It's some kind of huge Super Mutant creature! And it has a very large gun!}
+{101}{}{You're on Unity soil. I'd better take you in.}
+{102}{}{Okay.}
+{103}{}{I don't think so.}
+{104}{}{Wahh?}
