diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/TMPLELDR.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/TMPLELDR.msg
new file mode 100644
index 000000000..023d461bc
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/TMPLELDR.msg
@@ -0,0 +1,10 @@
+# Special Encounter
+
+{100}{}{You see: A robed person.}
+
+{150}{}{Father! The Chosen One is on his way. Does he get your blessing? Is that why you are here now?}
+{151}{}{Mother! The Chosen One is on his way. Does he get your blessing? Is that why you are here now?}
+{152}{}{Ugh!}
+
+{200}{}{You earn }
+{201}{}{ experience points for finishing the Temple of Trials.}
