diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/VALTOP.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/VALTOP.msg
new file mode 100644
index 000000000..8b104cb27
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/VALTOP.msg
@@ -0,0 +1 @@
+{100}{}{This is the Operations room. All command and control operations are performed here. These people are the leaders of the Vault.}
