diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/VALTMED.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/VALTMED.msg
new file mode 100644
index 000000000..048c28724
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/VALTMED.msg
@@ -0,0 +1 @@
+{100}{}{This is the Vault Emergency Medical Lab - EML. It is here so that all serious injuries and ailments might be treated. There should be a medic on duty 24 hours a day.}
