diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/VALTMEET.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/VALTMEET.msg
new file mode 100644
index 000000000..c09ba18db
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/VALTMEET.msg
@@ -0,0 +1 @@
+{100}{}{This is the Vault meeting room. This is a good gathering place for the vault-dwellers. Oddly, it does not hold your attention like it did in the past.}
