diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/VALTMON.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/VALTMON.msg
new file mode 100644
index 000000000..c9d57a0bd
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/VALTMON.msg
@@ -0,0 +1,2 @@
+{100}{}{You see one of those classic Science Fiction movies from the early 21st Century playing.}
+{101}{}{You are too pressed for time to be watching movies.}
