diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/VALTCOMP.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/VALTCOMP.msg
new file mode 100644
index 000000000..2bec4a2d9
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/VALTCOMP.msg
@@ -0,0 +1,3 @@
+{100}{}{You could glean more information from this terminal with a more scientific approach.}
+{101}{}{This terminal is regulating Vault supplies and functions, and performing routine administrative work.}
+{102}{}{The terminals on this level perform atmosphere and temperature control and backup maintenance for the medical lab.}
