diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/VALTGLO.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/VALTGLO.msg
new file mode 100644
index 000000000..711eaf73c
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/VALTGLO.msg
@@ -0,0 +1,9 @@
+{100}{}{You see: Glowing Ghoul.}
+{101}{}{The ghoul ignores you.}
+{102}{}{The liquid clear no longer flows. Without the flow we all take final rest.}
+#
+# Sduibek, Fallout Fixt:
+#
+{103}{}{You have doomed us all.}
+{104}{}{You will pay for your actions with blood...}
+{105}{}{The flow ends. So does your life.}
