diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/VALTCTZN.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/VALTCTZN.msg
new file mode 100644
index 000000000..035520ed1
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/VALTCTZN.msg
@@ -0,0 +1,7 @@
+{100}{}{You see a citizen of Vault 13.}
+{101}{}{What is the outside world like?}
+{102}{}{You must be brave to venture outside the Vault.}
+{103}{}{The Overseer says that the outside world is dangerous.}
+{104}{}{Be careful while traveling outside the Vault.}
+{105}{}{Don't forget to see the medic if you are hurt.}
+{106}{}{The Overseer makes sure everyone is well cared for in the Vault.}
