diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/VALTSTOR.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/VALTSTOR.msg
new file mode 100644
index 000000000..9843c342f
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/VALTSTOR.msg
@@ -0,0 +1 @@
+{100}{}{This is the storage room. It looks emptier than you remember it. The Vault must be running out of supplies at a greater rate than you could have imagined.}
