diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/VALTSEC.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/VALTSEC.msg
new file mode 100644
index 000000000..03d95d388
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/VALTSEC.msg
@@ -0,0 +1 @@
+{100}{}{This is the Security office. An officer is on duty here at all times. For what, you are not sure.}
