diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/VANCE.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/VANCE.msg
index 60f4fc323..b5a15f931 100644
--- a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/VANCE.msg
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/VANCE.msg
@@ -64,3 +64,8 @@
 {160}{}{Justin, Chad, let's show this asshole how we deal with their kind...}
 {161}{}{me Want some.}
 {162}{}{Nuuhhh... buh-bye.}
+
+#
+# Fo1in2
+#
+{164}{}{You see: Vance.}
