diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/VATWARN.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/VATWARN.msg
new file mode 100644
index 000000000..72a6a7991
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/VATWARN.msg
@@ -0,0 +1 @@
+{100}{}{You hear a female voice over a loudspeaker: "Attention! You have entered a secured area. Lethal force authorized."}
