diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/VBROKELV.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/VBROKELV.msg
new file mode 100644
index 000000000..b4345d291
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/VBROKELV.msg
@@ -0,0 +1,3 @@
+{100}{}{You notice a sign on the door saying that the elevator is out of order.}
+{101}{}{You see a sign on the door stating that it is out of order.}
+{102}{}{A sign says: Warning: Elevator Shaft under construction. Please use other elevator.}
