diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/VCONCOMP.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/VCONCOMP.msg
new file mode 100644
index 000000000..24ff26fb4
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/VCONCOMP.msg
@@ -0,0 +1,122 @@
+{100}{}{You see the Vats Control computer.}
+{101}{}{Access denied.}
+{102}{}{[the machine whirrs and clicks, followed by a loud "beep!"]  Access granted.}
+{103}{}{Display security codes.}
+{104}{}{Search logs.}
+{105}{}{Uplink to medical computers.}
+{106}{}{Set base to alert status.}
+{107}{}{17652-1028-1285362.}
+{108}{}{67209-5574-7805564.}
+{109}{}{55658-8678-1251533.}
+{110}{}{31914-1041-1251514.}
+{111}{}{53478-2565-7763929.}
+{112}{}{44309-3674-1254730.}
+{113}{}{[attempt to hack through the firewall to get more data]  Display security code encryption table.}
+{114}{}{\> Executing 17652-1028-1285362 ...}
+{115}{}{\> Executing 67209-5574-7805564 ...}
+{116}{}{\> Executing 55658-8678-1251533 ...}
+{117}{}{\> Executing 31914-1041-1251514 ...}
+{118}{}{\> Executing 53478-2565-7763929 ...}
+{119}{}{\> Executing 44309-3674-1254730 ...}
+#
+#TRANSLATION NOTE: Different spaces between the code and description is intentional for self-destruct execute lines
+#This is so the descriptions line up in-game so that it looks organized. 
+#If possible, spacing should stay the same when translated.
+#
+{120}{}{[you have successfully hacked into the encryption table]       > Command?}
+{121}{}{[17652-1028-1285362]    Silent self-destruct in 1 minute.}
+{122}{}{[67209-5574-7805564]  Self-destruct in 30 seconds.}
+#
+#123 is used if player selected Bug Fixes Only, otherwise 183 is used
+#
+{123}{}{[55658-8678-1251533]   Set base to alert status.}
+#
+#124 is used if player selected Bug Fixes Only, otherwise 184 is used
+#
+{124}{}{[31914-1041-1251514]     Silent self-destruct in 3 minutes.}
+{125}{}{[53478-2565-7763929]   Self-destruct in 3 minutes.}
+{126}{}{[44309-3674-1254730]   Immediate self-destruct sequence.}
+{127}{}{Invalid command. End input session.}
+{128}{}{Specify log you would like to review.}
+{129}{}{Grey}
+{130}{}{Boyarsky}
+{131}{}{Maxson}
+{132}{}{Anderson}
+{133}{}{Download log of Richard Grey?}
+{134}{}{Yes.}
+{135}{}{[return to previous menu]  No.}
+{136}{}{Download log of Capt. Maxson?}
+{137}{}{Yes.}
+{138}{}{[return to main menu]}
+{139}{}{ERROR: Unexpected end of line. Bad data encountered.}
+{140}{}{> Command?}
+{141}{}{Uplink successful. What is your command?}
+{142}{}{What is the current status of the patients?}
+{143}{}{Current status on patients as follows: FEV batch #23872-A consists of four male units and two female units. All individuals suffering from FEV exposure. Vital signs are within normal parameters. FEV process on all subjects is approximately 50% complete. Expect full FEV transformation within 12 hours.}
+{144}{}{I think something is happening to one of the patients. Is there anything I can do?}
+{145}{}{Patient suffering arrhythmia. Display patient status logs.}
+{146}{}{Available options are:}
+{147}{}{Status report on individual patients.}
+{148}{}{Shut down life support systems.}
+{149}{}{Increase dosage of original DNA strands of subjects to 100%.}
+{150}{}{[Log off]}
+{151}{}{Please enter authorization code.}
+{152}{}{Authorization 3578X287A.}
+{153}{}{Z876342BCD}
+{154}{}{Alpha Delta 287 Foxtrot}
+{155}{}{[Attempt to hack the computer]}
+{156}{}{Invalid authorization code. Command?}
+{157}{}{Authorization code accepted. Please stand by. Accessing data. Ready to process request.}
+{158}{}{Terminate life support systems on FEV batch #23872-A.}
+{159}{}{Increase original DNA strand dosage to 100% on all subjects.}
+{160}{}{Access granted. Which patient file would you like to examine?}
+{161}{}{Patient file #23869.}
+{162}{}{Patient file #23870.}
+{163}{}{Patient file #23871.}
+{164}{}{Patient file #23872.}
+{165}{}{Patient file #23869. Subject: Male. Age 25. Radiation count: 40. FEV exposure: 50%. Original DNA strands remaining: 50%. Vital signs are high but within normal parameters.}
+{166}{}{I would like to examine another patient file.}
+{167}{}{Patient file #23870. Subject: Male. Age 23. Radiation count: 15. FEV exposure: 60%. Original DNA strands remaining: 40%. Vital signs are within normal parameters.}
+{168}{}{Patient file #23781. Subject: Female. Age 20. Radiation count: 10. FEV exposure: 55%. Original DNA strands remaining: 45%. Vital signs are within normal parameters.}
+{169}{}{Patient file #23782. Subject: Male. Age 26. Radiation count: 35. FEV exposure: 60%. Original DNA strands remaining: 40%. Vital signs are above normal but within acceptable parameters.}
+{170}{}{Are you sure you wish to terminate the life support systems on FEV batch #23872-A?}
+{171}{}{FEV batch #23872-A terminated. Command?}
+{172}{}{Yes.}
+{173}{}{No.}
+{174}{}{Are you sure you wish to increase the dosage? Auto-correction will be disabled if you do this.}
+{175}{}{Increasing dosage of original DNA to 100%. WARNING: Auto-correction has been disabled.}
+{176}{}{You manage to get an interface screen out of the Vats Control computer.}
+{177}{}{You try to use the computer, but are unable to get any useful information out of it.}
+{178}{}{You try to use the computer, but the system is too complex for you to understand easily. You'll have to keep trying.}
+{179}{}{The explosion destroys the Vats Control system. Without regulation, it will certainly cycle out of control in minutes.}
+{180}{}{You slide the security card across the reader. The terminal clicks and whirrs. An unencrypted screen comes up.}
+{181}{}{[exit system]}
+{182}{}{Set base to alert status and disable console.}
+{183}{}{[10710-8971-2014510]     Activate alarms and disable console.}
+{184}{}{[31914-1041-1251514]      Silent self-destruct in 5 minutes.}
+{185}{}{10710-8971-2014510.}
+{186}{}{76223-4533-2540092.}
+{187}{}{[76223-4533-2540092]   Self-destruct in 1 minute.}
+{188}{}{\> Executing 76223-4533-2540092 ...}
+{189}{}{The explosion destroys the Vats Control computer.}
+#
+{200}{}{You notice that the maintenance panel is removed, exposing the fragile computer components.}
+{201}{}{Deactivate the base's alarms.}
+{202}{}{Command completed successfully.}
+{203}{}{[Continue]}
+{204}{}{[return to main menu]   Never mind.}
+{205}{}{\> Executing 536475-69-62656B ... ... ... Command completed successfully.}
+{206}{}{\> Executing 10710-8971-2014510 ...}
+{207}{}{ERROR: Records already found on Pip-Boy 2000. Aborting.}
+{208}{}{[return to main menu]   Never mind.}
+#
+{300}{}{10810-1103-9799121.}
+{301}{}{[10810-1103-9799121]     Self-destruct in 5 minutes.}
+{302}{}{\> Executing 10810-1103-9799121 ...}
+{303}{}{83100-1171-0539115.}
+{304}{}{[83100-1171-0539115]     Silent self-destruct in 10 minutes.}
+{305}{}{\> Executing 83100-1171-0539115 ...}
+
+# Fo1in2
+{400}{}{ manages to get an interface screen out of the Vats Control computer.}
+{401}{}{ tries to use the computer, but the system is too complex to understand easily.}
