diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/VCONDOOR.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/VCONDOOR.msg
new file mode 100644
index 000000000..bfa3a1f82
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/VCONDOOR.msg
@@ -0,0 +1,7 @@
+{100}{}{This looks like the door to the Vats Control Room.}
+{101}{}{The door is electronically locked.}
+{102}{}{You unlock the door.}
+{103}{}{You short out the electronic lock, fusing it shut. So much for that.}
+{104}{}{The door is already unlocked.}
+{105}{}{No luck. You'll need to use electronic lockpicks here.}
+{106}{}{You fiddle with the lock but can't yet bypass the code. Gotta keep trying!}
