diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/VCONCOMP.MSG b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/VCONCOMP.MSG
index 7d827efdc..0b265d653 100644
--- a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/VCONCOMP.MSG
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/VCONCOMP.MSG
@@ -116,3 +116,9 @@
 {303}{}{83100-1171-0539115.}
 {304}{}{[83100-1171-0539115]     Silent self-destruct in 10 minutes.}
 {305}{}{\> Executing 83100-1171-0539115 ...}
+
+# Fo1in2
+{400}{}{ manages to get an interface screen out of the Vats Control computer.}
+{401}{}{ tries to use the computer, but the system is too complex to understand easily.}
+
+
