diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/VLASER.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/VLASER.msg
new file mode 100644
index 000000000..e033b4044
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/VLASER.msg
@@ -0,0 +1,5 @@
+{100}{}{Laser fire strikes from the south end of the corridor!}
+{101}{}{You notice a small trip plate at your feet.}
+{102}{}{ Your knowledge of traps leads you to believe that this is a trigger of some kind.}
+{103}{}{You carefully sever the pressure springs on the trip plate.}
+{104}{}{You're not quite sure how to prevent the plate from triggering.}
