diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/VREE.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/VREE.msg
index 86a0ee9a8..da55ac1a4 100644
--- a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/VREE.msg
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/VREE.msg
@@ -92,7 +92,7 @@
 {191}{Vree203}{Guards! Guards!}
 {192}{Vree204}{You've destroyed my room!}
 {193}{Vree205}{What do you think you're doing with my things?}
-{194}{Vree300}{Who? Wait, that name sounds a familiar. Perhaps he is a leader of some of the mutants.}
+{194}{Vree300}{Who? Wait, that name sounds familiar. Perhaps he is a leader of some of the mutants.}
 {195}{Vree301}{I assume you're referring to the last nuclear war. The Brotherhood is doing everything it can to restore that which was lost.}
 {196}{Vree302}{They are high-density, laser-readable, manufactured-crystal storage devices. Each one holds over 4000 gigabytes of information.}
 {197}{Vree303}{The only salvation this tortured planet and its people have. Without us, humanity is sure to perish.}
