diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/VREEDISK.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/VREEDISK.msg
new file mode 100644
index 000000000..424640801
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/VREEDISK.msg
@@ -0,0 +1,8 @@
+{100}{}{This is the disk containing Vree's autopsy reports.}
+{101}{}{Pip-Boy 2000 displays: "Downloading ... Aborting, data already exists!"}
+{102}{}{Pip-Boy 2000 displays: "Downloading data ... ... ... Done. Have a nice day!"}
+{103}{}{This holodisk contains medical evidence from Vree of the Brotherhood of Steel showing that Super Mutants are sterile.}
+#
+# for Fallout Fixt:
+#
+{104}{}{You see: Vree's Autopsy Reports.}
