diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/WOMEN.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/WOMEN.msg
new file mode 100644
index 000000000..8cf588a6f
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/WOMEN.msg
@@ -0,0 +1,5 @@
+{100}{}{You see a bruised and battered woman.}
+{101}{}{No please don't!}
+{102}{}{[sobs uncontrollably]}
+{103}{}{[whispers] Please help me!}
+{104}{}{Thank you for saving me.}
