diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/YOURROOM.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/YOURROOM.msg
new file mode 100644
index 000000000..98f7f4050
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/YOURROOM.msg
@@ -0,0 +1 @@
+{100}{}{This door leads to your room. You have lived your entire life in this room, until now. Your fellow Vault citizens have told you that they will leave this room empty in your honor.}
