diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/ZAX.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/ZAX.msg
index 8b83627ea..7c8e748c3 100644
--- a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/ZAX.msg
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/ZAX.msg
@@ -55,10 +55,15 @@
 {145}{}{In this fashion, I am effectively capable of 'faith'. Barring evidence to the contrary, I therefore have 'faith' that I possess the equivalent of a 'soul'.}
 	{146}{}{May I ask some other questions?}
 	{147}{}{I see. Interesting.}
+
+{148}{}{I would be delighted.}
+
+{300}{}{This system is too advanced for you.}
+
 #
-# 148 continues at 228, listing wins/draws/losses. Space is intentional.
+# 1481 continues at 228, listing wins/draws/losses. Space is intentional. FIXT addition.
 #
-{148}{}{I would be delighted. Your current record is }
+{1481}{}{I would be delighted. Your current record is }
 {149}{}{It would seem that you have defeated me. Extraordinary.}
 {150}{}{Excellent form. However, you will need additional practice to defeat me.}
 	{151}{}{Play again?}
@@ -131,7 +136,7 @@
 	{207}{}{Thanks.}
 
 {208}{}{Several doors require a specific passkey to access. You must physically locate and use the correct passkey.}
-{209}{}{I do, however, have control of the secondary security locks to the weapon locker. I will release those locks. You will still need the Red Passkey to open the locker, however.}
+{209}{}{I do, however, have control of the secondary security locks to the weapon locker. I will release those locks. You will still need a red key card to open the locker.}
 {210}{}{You do not register as an individual of sufficient clearance to requisition that information.}
 {211}{}{However, sensors indicate that this facility is no longer operational in its original capacity. Probability favors destruction through nuclear force. Therefore, security concerns are now irrelevant.}
 
