diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/ZSWSIGN.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/ZSWSIGN.msg
new file mode 100644
index 000000000..bb427a94a
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/ZSWSIGN.msg
@@ -0,0 +1,14 @@
+{100}{}{A wanted poster with a crude charcoal sketch of a wanted person. The person on the poster looks familiar.}
+{101}{}{The caption reads:}
+
+# Orig. Fo2 - unused
+{102}{}{Wanted Dead or Alive for Child Killing and being a Rat Bastard. Reward $10,000 Dead or $5,000 Alive.}
+{103}{}{Wanted Dead or Alive for Child Killing. Reward $10,000 Dead or $5,000 Alive.}
+{104}{}{Wanted Dead or Alive for Being a Rat Bastard. Reward $10,000 Dead or $5,000 Alive.}
+
+# Fo1in2
+{110}{}{Wanted Dead or Alive for Child Killing and being a Rat Bastard. Reward }
+{111}{}{Wanted Dead or Alive for Child Killing. Reward }
+{112}{}{Wanted Dead or Alive for Being a Rat Bastard. Reward }
+{113}{}{ caps Dead or }
+{114}{}{ caps Alive.}
