diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/game/EDITOR.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/game/EDITOR.msg
new file mode 100644
index 000000000..078370de0
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/game/EDITOR.msg
@@ -0,0 +1,350 @@
+# Text labels and messages used by EDITOR.C
+#
+# (These MUST match "text message" enumeration on the top of EDITOR.C)
+#
+#
+{100}{}{DONE}
+{101}{}{OPTIONS}
+{102}{}{CANCEL}
+{103}{}{PRINT}
+{104}{}{AGE}
+{105}{}{SEX}
+{106}{}{NAME}
+{107}{}{MALE}
+{108}{}{FEMALE}
+{109}{}{PERKS}
+{110}{}{KARMA}
+{111}{}{KILLS}
+{112}{}{SKILL POINTS}
+{113}{}{Level:}
+{114}{}{Exp:}
+{115}{}{Next Level:}
+{116}{}{CHAR POINTS}
+{117}{}{SKILLS}
+{118}{}{You must use all character points} # 1
+{119}{}{before starting the game!}         # 2
+#
+{120}{}{Character Points}
+{121}{}{Amount of free character points that can be added to one of the basic stats.}
+{122}{}{Next Level}
+{123}{}{The amount of experience points needed to move up to the next level.}
+{124}{}{Perks}
+{125}{}{Reputation (General)}
+{126}{}{Kills}
+{127}{}{Perks add additional abilities. Every third experience level, you can choose one perk.}
+{128}{}{These are the effects of the choices you make during the game.}
+{129}{}{A counter that keeps track of the type and numbers of creatures you have slaughtered, or have seen killed in your presence.}
+{130}{}{Skill Points}
+{131}{}{Use free skill points to increase the level of a skill.}
+#
+{132}{}{At maximum level.}        # 1
+{133}{}{Unable to increment it.}  # 2
+{134}{}{At minimum level.}        # 1
+{135}{}{Unable to decrement it.}  # 2
+{136}{}{Not enough skill points available.}
+{137}{}{Base:}
+{138}{}{TAG SKILLS}
+{139}{}{OPTIONAL TRAITS}
+{140}{}{You already have the maximum of} # 1
+{141}{}{three tag skills!}               # 2
+{142}{}{You must select all tag skills}  # 1
+{143}{}{before starting the game!}       # 2
+{144}{}{Tag Skills}
+{145}{}{Tag skills are skills your character specializes in.  Each tag skill gains +20%, and increases twice as fast.  You must pick three tag skills.}
+{146}{}{Optional Traits}
+{147}{}{Optional traits describe your character in more detail.  All traits will have positive and negative effects.  You may choose up to two traits during creation.}
+{148}{}{You already have the maximum of} # 1
+{149}{}{two traits!}                     # 2
+{150}{}{Skills}
+{151}{}{Skills are learned knowledge.  Skills increase by experience, or during the course of the game by special events.  The higher the skill level, the better you are at that skill.}
+#
+{152}{}{PICK A PERK}
+{153}{}{PICK A NEW TRAIT}
+{154}{}{LOSE A TRAIT}
+{155}{}{PICK A NEW TAG SKILL}
+{156}{}{TRAITS}
+#
+{157}{}{All stats must be between 1 and 10} # 1
+{158}{}{before starting the game!}          # 2
+# Fallout 2:
+{160}{}{Warning: You haven't changed your player} # 1
+{161}{}{name.  Use this character anyway?}        # 2
+
+{175}{}{Next Lvl:}  # Short for Level:
+
+
+#
+# Abbreviated basic stat levels
+#
+{200}{}{V.Bad}  # Terrible
+{201}{}{Bad}    # Bad
+{202}{}{Poor}   # Poor
+{203}{}{Fair}   # Fair
+{204}{}{Avrg}   # Average
+{205}{}{Good}   # Good
+{206}{}{V.Good} # Very Good
+{207}{}{Great}  # Great
+{208}{}{Exclnt} # Excellent
+{209}{}{Heroic} # Heroic
+
+#
+# Abbreviated Derived/Non Derived stat names
+#
+{300}{}{Hit Points}
+{301}{}{Action Points }
+{302}{}{Armor Class}
+{303}{}{Unarmed Damage}
+{304}{}{Melee Damage}
+{305}{}{Damage Res.  }
+{306}{}{Poison Res.   }
+{307}{}{Radiation Res.}
+{308}{}{Sequence}
+{309}{}{Healing Rate}
+{310}{}{Critical Chance}
+{311}{}{Carry Weight}
+{312}{}{Poisoned}
+{313}{}{Radiated}
+{314}{}{Eye Damage}
+{315}{}{Crippled Right Arm}
+{316}{}{Crippled Left Arm}
+{317}{}{Crippled Right Leg}
+{318}{}{Crippled Left Leg}
+
+#
+# Damage info texts
+#
+# Poisoned
+{400}{}{Your character has been poisoned.  Poison will do damage over a period of time, until cured or it passes from your system.}
+# Radiation
+{401}{}{Your character is suffering from a significant amount of Radiation poisoning.  The more radiation damage, the more deadly the effect.}
+# Eye Damage
+{402}{}{This means your character has been seriously hit in one or both of your eyes.  This affects your Perception.}
+# Crippled right arm
+{403}{}{The right arm has been severely hurt, and cannot function well.  If one arm has been crippled, you cannot use two-handed weapons.  If both arms have been crippled, you cannot attack with weapons.}
+# Crippled left arm
+{404}{}{Your character's left arm has been severely hurt, and cannot function well.  If one arm has been crippled, you cannot use two-handed weapons.  If both arms have been crippled, you cannot attack with weapons.}
+# Crippled right leg
+{405}{}{Your character has a crippled right leg.}
+# Crippled left leg
+{406}{}{Your character has a crippled left leg.}
+
+# Month strings
+{500}{}{January}
+{501}{}{February}
+{502}{}{March}
+{503}{}{April}
+{504}{}{May}
+{505}{}{June}
+{506}{}{July}
+{507}{}{August}
+{508}{}{September}
+{509}{}{October}
+{510}{}{November}
+{511}{}{December}
+
+#
+# Option menu text
+#
+{600}{}{SAVE}
+{601}{}{LOAD}
+{602}{}{PRINT TO FILE}
+{603}{}{ERASE}
+{604}{}{DONE}
+{605}{}{Are you sure you want to erase}      # line 1
+{606}{}{the current character?}                 # line 2
+{607}{}{ was successfully saved.}
+{608}{}{ was successfully loaded.}
+{609}{}{already exists.}
+{610}{}{Do you wish to overwrite?}
+{611}{}{Error saving }
+{612}{}{Error loading }
+{613}{}{Character file invalid.}
+{614}{}{Character data invalid.}
+{615}{}{Error reading file list!}
+{616}{}{PRINT}
+{617}{}{PLAYER}
+{618}{}{BLANK.GCD}
+
+#
+# ASCII text file printing text
+#
+{620}{}{FALLOUT}                   # Title line 1
+{621}{}{VAULT-13 PERSONNEL RECORD} # Title line 2
+{622}{}{hours} # Title line 3
+# -
+# -
+#      Strength: 00        Hit Points: 000/000          Sequence: 00
+#    Perception: 00       Armor Class: 000          Healing Rate: 00
+#     Endurance: 00     Action Points: 000       Critical Chance: 000%
+#      Charisma: 00      Melee Damage: 000          Carry Weight: 000 lbs.
+#  Intelligence: 00       Damage Res.: 000%
+#       Agility: 00    Radiation Res.: 000%
+#          Luck: 00       Poison Res.: 000%
+#
+{623}{}{ ::: Statistics :::}
+{624}{}{       Strength:}
+{625}{}{        Hit Points:}
+{626}{}{        Sequence:}
+# -
+{627}{}{     Perception:}
+{628}{}{       Armor Class:}
+{629}{}{        Healing Rate:}
+# -
+{630}{}{      Endurance:}
+{631}{}{     Action Points:}
+{632}{}{      Critical Chance:}
+# -
+{633}{}{       Charisma:}
+{634}{}{      Melee Damage:}
+{635}{}{         Carry Weight:}
+# -
+{636}{}{   Intelligence:}
+{637}{}{       Damage Res.:}
+# -
+{638}{}{        Agility:}
+{639}{}{    Radiation Res.:}
+# -
+{640}{}{           Luck:}
+{641}{}{       Poison Res.:}
+# -
+# -
+# Name: 01234567890        Age: 00         Gender: Female
+{642}{}{  Name:}
+{643}{}{Age:}
+{644}{}{               Gender:}
+{645}{}{Male}
+{646}{}{Female}
+#-
+# Level: 00                 Exp: 000,000    Next Level: 000,000
+{647}{}{ Level:}
+{648}{}{                Exp:}
+{649}{}{Next Level:}
+#-
+{650}{}{ ::: Traits :::}
+{651}{}{ ::: Perks :::}
+{652}{}{ ::: Karma :::}
+
+#-
+{653}{}{ ::: Skills :::                ::: Kills :::}
+#-
+{654}{}{ ::: Inventory :::}
+#-
+{655}{}{                                      Total Weight:}
+{656}{}{ ::: Addictions :::}
+{657}{}{ ::: Reputation :::}
+
+#
+# Karma name strings
+#
+{1000}{}{Reputation (General) }
+{1001}{}{Berserker}
+{1002}{}{Champion}
+{1003}{}{Childkiller}
+{1004}{}{Nuka Cola addiction}
+{1005}{}{Buffout addiction}
+{1006}{}{Mentats addiction}
+{1007}{}{Psycho addiction}
+{1008}{}{RadAway addiction}
+{1009}{}{Alcohol addiction}
+{1010}{}{Jet addiction}
+{1011}{}{Tragic addiction}
+{1012}{}{THIS GOES WITH THE TEMPORARY ONE FOR DESCRIPTIONS #1112: DON'T TRANSLATE THIS}
+{1013}{}{Sexpert}
+{1014}{}{Prizefighter}
+{1015}{}{Gigolo}
+{1016}{}{Grave Digger}
+{1017}{}{Made Man: Salvatore}
+{1018}{}{Married}
+{1019}{}{Porn Star}
+{1020}{}{Slaver}
+{1021}{}{Virgin of the Wastes}
+{1022}{}{Made Man: Bishop}
+{1023}{}{Made Man: Mordino}
+{1024}{}{Made Man: Wright}
+{1025}{}{Separated}
+
+#
+# Karma description strings
+#
+{1100}{}{This is a ranged stat that represents your general reputation. The higher the number, the better known and respected you are. If your rep is negative, you are hated by some for your evil actions.}
+{1101}{}{You have killed a large number of people. This is generally not a good thing to get known for. People from the wrong side of the track will like you a little more, however.}
+{1102}{}{Your actions have revealed you to be a champion of the people. Your war against evil and villainy is widely known. Honorable people will respect you better.}
+{1103}{}{You have killed children, the youth of the wasteland. This is considered to be a really bad thing. You evil, evil person.}
+{1104}{}{You thoroughly enjoy Nuka-Cola, the pop of the post-nuclear world.}
+{1105}{}{You are addicted to the chem Buffout. You must take this chem on an ongoing basis or suffer the effects of withdrawal.}
+{1106}{}{You are suffering from Mentat addiction. You must take this particular chem or suffer withdrawal.}
+{1107}{}{You have a chem addiction. If you do not take Psycho on a regular schedule, you will suffer. If you do, others will suffer.}
+{1108}{}{Your body has become addicted to Radaway. While not life threatening, you must take it regularly.}
+{1109}{}{You have a problem with alcohol. Actually, you have many problems with your drinking habits.}
+{1110}{}{You have a chem addiction. You must take Jet on a regular basis.}
+{1111}{}{Tragically, you are addicted to the Garnering. You are constantly in search of new expansions.}
+{1112}{}{TEMPORARY - What is this?}
+{1113}{}{You know sex. You know foreplay. You know how to please a member of the opposite sex and leave them hungry for more.}
+{1114}{}{You are the heavyweight champion of Northern California. You have gained fame, respect, the love of thousands... and a bonus to your toughness and unarmed skill.}
+{1115}{}{Let's be honest: You sleep with anything that walks on two legs. Sometimes, you're not even that discriminating.}
+{1116}{}{"They're dead, they don't care," has become your motto. Digging up the remains of others is more than a hobby for you.}
+{1117}{}{You are a "Made Man" of the Salvatore Family. You are well-known, well-respected, and look great in a fedora.}
+{1118}{}{You got hitched. Hey, it's your problem, not ours.}
+{1119}{}{You are a big, bright, shining star. Your sexual exploits are well-known throughout New Reno.}
+{1120}{}{You have been marked as a member of the Slaver's Guild. The tattoo on your forehead makes your profession evident to everyone you meet.}
+{1121}{}{You really need to get out more. Your sexual exploits have been...well, two dimensional.}
+{1122}{}{You are a "Made Man" of the Bishop Family. You are well-known, well-respected, and look great in a fedora.}
+{1123}{}{You are a "Made Man" of the Mordino Family. You are well-known, well-respected, and look great in a fedora.}
+{1124}{}{You are a "Made Man" of the Wright Family. You are well-known, well-respected, and look great in a fedora.}
+{1125}{}{Your spouse is no longer with you. I hope you're happy.}
+
+#
+# Town reputations
+#
+{2000}{}{Idolized}
+{2001}{}{Liked}
+{2002}{}{Accepted}
+{2003}{}{Neutral}
+{2004}{}{Antipathy}
+{2005}{}{Hated}
+{2006}{}{Vilified}
+
+{2100}{}{Everyone in this area would like to be you. You're the toast-of-the-town and are held up as a shining example of all that a person can be.}
+{2101}{}{People wave, and greet you, wherever you go in this area. Everyone has heard talk of you, and they like what they've heard.}
+{2102}{}{Although, you're not greeted with open arms, most people in this area will at least nod in acknowledgment when you meet. You fit in as well as most.}
+{2103}{}{Though normally suspicious of strangers, the people of this area haven't made up their minds about you yet.}
+{2104}{}{You're just another out-of-towner in this area. Probably up to no good.}
+{2105}{}{You've proven to the people in this area that their initial distrust was well founded. People in this area look at you and mutter to one another when you come near. They know you're up to no good.}
+{2106}{}{Staring and muttering, some cross the street to avoid walking near you, while others, spit and turn away when you come near. Everyone knows that you're one of the worst things that's ever slithered into this area.}
+
+#
+# Karma status strings
+#
+{3000}{}{Savior of the Damned}
+{3001}{}{Guardian of the Wastes}
+{3002}{}{Shield of Hope}
+{3003}{}{Defender}
+{3004}{}{Wanderer}
+{3005}{}{Betrayer}
+{3006}{}{Sword of Despair}
+{3007}{}{Scourge of the Wastes}
+{3008}{}{Demon Spawn}
+
+#
+# Karma section titles.
+#
+{4000}{}{Reputation}
+{4001}{}{Addictions}
+
+#
+# Karma section descriptions.
+#
+{4100}{}{Represents your reputation for a particular location.}
+{4101}{}{Things you have become addicted to.}
+
+#
+# Special Demo version only text
+#
+{5000}{}{Sorry, you can only print your}   # 1
+{5001}{}{character in the full version.}   # 2
+{5002}{}{Sorry, you can only save your}    # 1
+{5003}{}{character in the full version.}   # 2
+{5004}{}{Sorry, you can only load a saved} # 1
+{5005}{}{character in the full version.}   # 2
+{5006}{}{CHEAT MODE}
+{5007}{}{Not available in demo.}
