diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/game/MISC.MSG b/Fallout2/Fallout1in2/mods/fo1_base/text/english/game/MISC.MSG
index 4fce11009..4f3096920 100644
--- a/Fallout2/Fallout1in2/mods/fo1_base/text/english/game/MISC.MSG
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/game/MISC.MSG
@@ -108,7 +108,7 @@
 {9032}{}{Looks like all this travelling has paid off.}
 {9033}{}{I've learned some new tricks.}
 
-{9040}{}{Oh dear...I've actually learned something from this.} # pMKatja_PID
+{9040}{}{Oh man...I've actually learned something from this.} # pMKatja_PID
 {9041}{}{There is so much we can still learn.}
 {9042}{}{I think I'm finally getting the hang of this.}
 {9043}{}{I can't believe I survived this far.}
