diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/game/PIPBOY.MSG b/Fallout2/Fallout1in2/mods/fo1_base/text/english/game/PIPBOY.MSG
index 083a495c8..f625c9c24 100644
--- a/Fallout2/Fallout1in2/mods/fo1_base/text/english/game/PIPBOY.MSG
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/game/PIPBOY.MSG
@@ -50,13 +50,29 @@
 {307}{}{Rest for four hours}
 {308}{}{Rest for five hours}
 {309}{}{Rest for six hours}
-{310}{}{Rest until morning (0600)}
+{310}{}{Rest until morning (0800)}
 {311}{}{Rest until noon (1200)}
 {312}{}{Rest until evening (1800)}
 {313}{}{Rest until midnight (0000)}
 {314}{}{Rest until healed}
 {315}{}{Rest until party is healed}
 
+# Fo1 resting times
+{320}{}{Rest for ten minutes}
+{321}{}{Rest for thirty minutes}
+{322}{}{Rest for an hour}
+{323}{}{Rest for two hours}
+{324}{}{Rest for three hours}
+{325}{}{Rest for four hours}
+{326}{}{Rest for five hours}
+{327}{}{Rest for six hours}
+{328}{}{Rest until morning (0600)}
+{329}{}{Rest until noon (1200)}
+{330}{}{Rest until evening (1800)}
+{331}{}{Rest until midnight (0000)}
+{332}{}{Rest until healed}
+{333}{}{Rest until party is healed}
+
 #
 # Holodisk titles
 #
