diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/game/PIPBOY.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/game/PIPBOY.msg
new file mode 100644
index 000000000..19d9c2c28
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/game/PIPBOY.msg
@@ -0,0 +1,1312 @@
+#
+#
+# Text labels and messages used in PIPBOY.C
+#
+# (These MUST match "text message" enumeration on the top of PIPBOY.C)
+#
+
+#
+# Special holiday dates
+#
+{100}{}{Happy New Year's Day!}      # New years
+{101}{}{Happy Valentine's Day!}     # April fools
+{102}{}{Happy All Fool's Day!}      # April fools
+{103}{}{Happy Ship Day!}            # Shipping day!
+{104}{}{Happy Independence Day!}    # Independence day
+{105}{}{Happy Halloween!}           # Halloween
+{106}{}{Happy Thanksgiving Day!}    # Thanksgiving
+{107}{}{Merry Christmas!}           # X-mas
+
+#
+# System messages
+#
+{200}{}{MORE}
+{201}{}{BACK}
+{202}{}{STATUS}
+{203}{}{No current tasks.}
+{204}{}{CLUES}  # clues gone
+{205}{}{AUTOMAPS}
+{206}{}{VIDEO ARCHIVES}
+{207}{}{DATA}
+{208}{}{}
+{209}{}{}
+{210}{}{Status}
+{211}{}{DATA}
+{212}{}{of}
+{213}{}{Page}
+{214}{}{DONE}
+{215}{}{You cannot rest at this location!}
+
+#
+# Alarm clock
+#
+{300}{}{ALARM CLOCK}
+{301}{}{Hit Points}
+{302}{}{Rest for ten minutes}
+{303}{}{Rest for thirty minutes}
+{304}{}{Rest for an hour}
+{305}{}{Rest for two hours}
+{306}{}{Rest for three hours}
+{307}{}{Rest for four hours}
+{308}{}{Rest for five hours}
+{309}{}{Rest for six hours}
+{310}{}{Rest until morning (0800)}
+{311}{}{Rest until noon (1200)}
+{312}{}{Rest until evening (1800)}
+{313}{}{Rest until midnight (0000)}
+{314}{}{Rest until healed}
+{315}{}{Rest until party is healed}
+
+# Fo1 resting times
+{320}{}{Rest for ten minutes}
+{321}{}{Rest for thirty minutes}
+{322}{}{Rest for an hour}
+{323}{}{Rest for two hours}
+{324}{}{Rest for three hours}
+{325}{}{Rest for four hours}
+{326}{}{Rest for five hours}
+{327}{}{Rest for six hours}
+{328}{}{Rest until morning (0600)}
+{329}{}{Rest until noon (1200)}
+{330}{}{Rest until evening (1800)}
+{331}{}{Rest until midnight (0000)}
+{332}{}{Rest until healed}
+{333}{}{Rest until party is healed}
+
+#
+# Holodisk titles
+#
+{400}{}{FEV Experiment Tape}
+{401}{}{Security Tape}
+{402}{}{Ancient Brotherhood Tape}
+{403}{}{Alpha Experiment Tape}
+{404}{}{Delta Experiment Tape}
+{405}{}{Vree's Experiment Tape}
+{406}{}{Brotherhood Code}
+{407}{}{Mutant Transmissions}
+{408}{}{Brotherhood Tape}
+{409}{}{Sophia Tape}
+{410}{}{Maxson Log}
+{411}{}{Richard Grey Audio Diary}
+{412}{}{Captain Maxson's Diary}
+{413}{}{FEV Research}
+{414}{}{Power Armor Specs}
+{415}{}{West Tech Records}
+{416}{}{Vault Locations v34.129}
+{417}{}{Regulator Transmission}
+{418}{}{RESERVED (available)}
+{419}{}{RESERVED (available)}
+{420}{}{RESERVED (available)}
+
+#
+# Movie archive titles
+# Order must match the list (see gmovie.h)
+#
+{500}{}{InterPlay Logo}               	# Not show in Pip-Boy 2000
+{501}{}{MacPlay Logo}                 	# MacPlay version ,Not show in Pip-Boy 2000
+{502}{}{Overseer Briefing}
+{503}{}{Vat Destruction}
+{504}{}{Cathedral Destruction}
+{505}{}{Dipped in the Vats}
+{506}{}{Vault 13 Invasion}
+{507}{}{ }
+{508}{}{ }
+{509}{}{ }
+{510}{}{Leaving Vault} 					# Male
+{511}{}{Leaving Vault} 					# Female
+{512}{}{Vault 13 Water}
+{513}{}{Vault 13 Water Running Low}
+{514}{}{Vault 13 Water Deprivation}
+{515}{}{ }
+{516}{}{ }
+
+#
+# Things for the Mac version to speak
+#
+#
+{600}{}{Status}
+{601}{}{Clues}  # Clues gone
+{602}{}{Automaps}
+{603}{}{Archives}
+{604}{}{Alarm Clock}
+{605}{}{Doo da doo da dum dee dee doodly doo dum dum dum doo da doo da doo da doo da doo da doo da doo}
+{606}{}{Welcome to the Rob Co Pipboy 2000.}
+{607}{}{So long.}
+{608}{}{Wake up.}
+
+
+#
+# Status thread text
+#
+# Format:
+# 0 City name
+# 1 ... 9 thread/quest name
+#
+{700}{}{Vault 13}
+{701}{}{Calm rebel faction.}
+{702}{}{Destroy the Mutant leader.}
+{703}{}{Destroy the source of the Mutants.}
+{704}{}{Find the Water Chip.}
+{705}{}{Find the Water Thief.}
+{706}{}{Help the medic to synthesize Antidote.}
+{707}{}{Learn about the outside world at the library terminals.}
+#
+{710}{}{Vault 15}
+{711}{}{Search for clues and information.}
+{712}{}{Locate a Water Chip.}
+#
+{720}{}{Shady Sands}
+{721}{}{Cure Jarvis of radscorpion poison.}
+{722}{}{Help Razlo find a cure for radscorpion venom.}
+{723}{}{Rescue Tandi from the Raiders.}
+{724}{}{Stop the radscorpions.}
+{725}{}{Help Shady Sands improve their crop yield.}
+{726}{}{Talk to Katrina.}
+{727}{}{Hire Ian.}
+#
+{730}{}{Junktown}
+{731}{}{Help Saul.}
+{732}{}{Kill Killian.}
+{733}{}{Rescue Sinthia.}
+{734}{}{Save Trish.}
+{735}{}{Stop Gizmo.}
+{736}{}{Bust the Skulz gang.}
+#
+{740}{}{Raiders}
+{741}{}{Join the Khans.}
+{742}{}{Destroy the Khans.}
+{743}{}{Kill the slaves.}
+{744}{}{Save the slaves.}
+#
+{750}{}{Necropolis}
+{751}{}{Destroy the Super Mutants at the Watershed.}
+{752}{}{Fix the Necropolis water pump.}
+#
+{760}{}{The Hub}
+{761}{}{Find and destroy the Deathclaw.}
+{762}{}{Dispose of Jain.}
+{763}{}{Dispose of merchant.}
+{764}{}{Find the missing caravans.}
+{765}{}{Steal necklace from the Merchants.}
+#
+{770}{}{Brotherhood of Steel}
+{771}{}{Become an Initiate.}
+{772}{}{Rescue Initiate from the Hub.}
+#
+{780}{}{Mariposa Military Base}
+#
+{790}{}{The Glow}
+{791}{}{Disarm traps for the facility.}
+{792}{}{Turn on power for the Glow.}
+#
+{800}{}{Boneyard}
+{801}{}{Become a Blade.}
+{802}{}{Deliver Locket for Romero.}
+{803}{}{Deliver package from the Gun Runners.}
+{804}{}{Find Children spy in the Followers.}
+{805}{}{Fix hydroponic farms in Adytum.}
+{806}{}{Rescue Jason Zimmerman.}
+{807}{}{Stop the Gangs from attacking Adytum.}
+#
+{810}{}{Children's Cathedral}
+
+
+#
+# Holodisk text
+#
+
+# *** FEV Experiment Disk ***
+{1000}{}{•  Log Date March 21, 2075:}
+{1001}{}{Major Barnett has ordered experiments with batch}
+{1002}{}{10-011 of  panimmunity virion, which has been }
+{1003}{}{renamed FEV, for Forced Evolutionary Virus. His}
+{1004}{}{main concern is with the side effects of the }
+{1005}{}{quad-helix structure rather than its main effect}
+{1006}{}{of replicative stability. He believes the new }
+{1007}{}{structure is the next logical step for mammalian}
+{1008}{}{nuclei. Experiments with single-celled organisms}
+{1009}{}{is a great success. While their basal metabolism}
+{1010}{}{appears unchanged, their immunity to infection }
+{1011}{}{and radiation is exceeding all earlier expectations.}
+{1012}{}{Addendum: chloroplasts seem unaffected by the virion.}
+{1013}{}{Further experiments on plant cells have been }
+{1014}{}{canceled by order of Major Barnett.}
+{1015}{}{**END-PAR**}
+# -
+{1016}{}{•  Log Date May 9, 2075}
+{1017}{}{We infected several species of flatworm with FEV. }
+{1018}{}{Within hours the worms had increased in size by }
+{1019}{}{28%, and 39 separate viral contagions were resisted}
+{1020}{}{by the population. Each sample was allowed to }
+{1021}{}{continue for several generations, and the new DNA }
+{1022}{}{structure was successfully passed on to worm's }
+{1023}{}{progeny, although only asexual reproduction was }
+{1024}{}{noticed in the samples. Experiments with insects }
+{1025}{}{have had less success. Major Barnett has postponed }
+{1026}{}{these experiments until further notice.}
+{1027}{}{**END-PAR**}
+# -
+{1028}{}{•  Log Date June 30, 2075}
+{1029}{}{Several lab strains of white mice have been infected}
+{1030}{}{successfully with FEV. Again, an increase in size }
+{1031}{}{was noted within hours, and after 9 days all mice }
+{1032}{}{had stabilized at 31% larger than the control group. }
+{1033}{}{Dissection revealed the most increase in size in }
+{1034}{}{striated muscle tissue and certain internal organs, }
+{1035}{}{such as the liver, heart, and kidneys. In a }
+{1036}{}{surprising finding, the infected mice were found to}
+{1037}{}{run mazes in less than half of the time of the }
+{1038}{}{control group. More testing will be needed to }
+{1039}{}{confirm this finding as significant.}
+{1040}{}{**END-PAR**}
+# -
+{1041}{}{•  Log Date November 9, 2075}
+{1042}{}{We have infected 218 rabbits with FEV. Half of }
+{1043}{}{the subjects were implanted with electrodes to }
+{1044}{}{monitor EEG activity before and after the }
+{1045}{}{infection. Increased electrical activity in the }
+{1046}{}{brain was noted in 3.2 seconds (on average) after}
+{1047}{}{injection. Again, the typical size increase was }
+{1048}{}{noted. However, increased aggression and posturing}
+{1049}{}{(especially among males) was noted as well. }
+{1050}{}{Sacrifice and profusion yielded brain tissue that}
+{1051}{}{showed increased dendritic connection, especially }
+{1052}{}{in the limbic system and frontal cortex.}
+{1053}{}{**END-PAR**}
+# -
+{1054}{}{•  Log Date January 12, 2076}
+{1055}{}{With batch 11-011, we have improved the mitotic }
+{1056}{}{cycle efficiency by 43%. We have infected 53 }
+{1057}{}{raccoons with the new strain. In addition to the}
+{1058}{}{now expected size increase, behavioral tests }
+{1059}{}{confirmed an increase in intelligence and manual }
+{1060}{}{dexterity by 19 points on the Schuler-Kapp index. }
+{1061}{}{Unfortunately, several subjects escaped confinement}
+{1062}{}{and had to be hunted down and dispatched. Major }
+{1063}{}{Barneet ordered the remaining subjects terminated.}
+{1064}{}{Two pairs were unaccounted for.}
+{1065}{}{**END-PAR**}
+# -
+{1066}{}{•  Log Date May 13, 2076}
+{1067}{}{We have spliced several new gene sequences, supplied}
+{1068}{}{by Major Barnett's advisory team, into FEV. With }
+{1069}{}{batch 11-101a, we infected 23 dogs of both pure and }
+{1070}{}{mixed breed, and all experienced nearly immediate }
+{1071}{}{growth. The larger size was accompanied with }
+{1072}{}{increased aggressiveness, while no significant }
+{1073}{}{intelligence increase was noted. We plan to attempt}
+{1074}{}{crossover of 92 allele pairs with batch 11-011. All}
+{1075}{}{subjects were terminated after 14 weeks of study.}
+{1076}{}{**END-PAR**}
+# -
+{1077}{}{•  Log Date October 4, 2076}
+{1078}{}{The crossover has been completed and 15 chimpanzees}
+{1079}{}{were infected with batch 11-111. Growth and immunity}
+{1080}{}{levels are unprecedented. Attempts to induce cancers}
+{1081}{}{in the subjects through radiological and chemical }
+{1082}{}{agents were not successful. Increased aggressiveness}
+{1083}{}{have led to isolating the subjects. Two subjects }
+{1084}{}{suffered violent epileptic seizures and died. All }
+{1085}{}{remaining subjects terminated.}
+{1086}{}{**END-PAR**}
+# -
+{1087}{}{•  Log Date January 7, 2077}
+{1088}{}{Major Barnett has ordered transfer of all FEV }
+{1089}{}{research  to the Mariposa Military Base. He }
+{1090}{}{plans to continue the project experiments on }
+{1091}{}{volunteer subjects. I am against this, and would }
+{1092}{}{like it noted here that research on human subjects}
+{1093}{}{is not recommended by myself or my staff.}
+{1094}{}{**END-DISK**}
+
+# *** Security Disk ***
+{2000}{}{Security override Gamma Omicron Delta.}
+{2001}{}{**END-DISK**}
+
+# *** Ancient Brotherhood Disk ***
+{3000}{}{Capt. Maxson was right. This place is death.}
+{3001}{}{I'm writing this so that if we don't make it}
+{3002}{}{back, someone, some day, might find out what}
+{3003}{}{happened to us.}
+{3004}{}{**END-PAR**}
+# -
+{3005}{}{We made it to West Tek Research Facility after}
+{3006}{}{20 days of hell. But that was the easy part. The}
+{3007}{}{radiation levels began to shoot up as soon as we}
+{3008}{}{could see the giant crater. We checked our supplies}
+{3009}{}{and figured with our armor and our anti-rad supplies}
+{3010}{}{we'd be fine for a least a day of exploring. We }
+{3011}{}{felt it was a calculated risk, but the technology }
+{3012}{}{we had the potential of recovering was worth it.}
+{3013}{}{**END-PAR**}
+# -
+{3014}{}{We climbed down the crater to the first level and }
+{3015}{}{everything seemed to be according to plan. The power}
+{3016}{}{was off, so we didn't need to circumvent the }
+{3017}{}{security. Or so we thought. There wasn't much of }
+{3018}{}{value on this level, so we pushed on.}
+{3019}{}{**END-PAR**}
+# -
+{3020}{}{The second level was more of the same.}
+{3021}{}{**END-PAR**}
+# -
+{3022}{}{When Jensen dropped to the third level, all hell }
+{3023}{}{broke loose. The security sensors had been burned}
+{3024}{}{out on the first two levels, but not on the third.}
+{3025}{}{Jensen was cut to ribbons before he knew what had }
+{3026}{}{happened. We'd never seen weapons cut through power}
+{3027}{}{armor like that. Men started dropping right and }
+{3028}{}{left, and the ones who were still alive lost it. I}
+{3029}{}{tried to regroup, but only Soto and Camarillo made}
+{3030}{}{it back up here to the first level with me.}
+{3031}{}{**END-PAR**}
+# -
+{3032}{}{ The fact that I can smell Soto's burning flesh }
+{3033}{}{where his arm was taken off means that my power }
+{3034}{}{armor is no longer air tight, so I'm sucking up }
+{3035}{}{a lot more rads than I had planned on. I'm leaking}
+{3036}{}{hydraulics at an alarming rate. We need to get far}
+{3037}{}{enough away from this place before my armor dies. }
+{3038}{}{**END-PAR**}
+# -
+{3039}{}{Camarillo seemed fine physically, but he wandered}
+{3040}{}{off about an hour ago, mumbling something about }
+{3041}{}{Gehenna. That bastard has all the anti-rad. }
+{3042}{}{**END-PAR**}
+# -
+{3043}{}{That leaves Soto and myself. We can't make }
+{3044}{}{it far enough away from here without the }
+{3045}{}{anti-rads, so I've got to try to find Camarillo}
+{3046}{}{before it's too late.}
+{3047}{}{**END-PAR**}
+# -
+{3048}{}{Sgt. D. Allen , United States Armed Forces.}
+{3049}{}{**END-DISK**}
+
+
+#  *** Alpha Experiment Disk ***
+{4000}{}{Prototype Panimmunity Virion Project}
+{4001}{}{**END-PAR**}
+# -
+{4002}{}{In the hopes of countering the current }
+{4003}{}{bacteriological and viral agents employed by the}
+{4004}{}{Chinese government, we have manufactured a virus }
+{4005}{}{fragment consisting of ribonucleic acid (RNA) }
+{4006}{}{encased in a protein-lipid sheath. This virion }
+{4007}{}{contains a specially arranged sequence of irradiated}
+{4008}{}{amino acids (IAA) that are capable of attaching to}
+{4009}{}{non-specific binding sites on deoxyribonucleic }
+{4010}{}{acid (DNA) and force a non-replicating mitosis to}
+{4011}{}{occur. The resulting host cell is left with a }
+{4012}{}{quadruple-helix DNA structure.}
+{4013}{}{**END-PAR**}
+# -
+{4014}{}{Early tests are promising. The virion easily }
+{4015}{}{penetrates the cell membrane and attaches directly}
+{4016}{}{to the host DNA in the nucleus. Mitosis of the }
+{4017}{}{structure begins almost immediately. Cytokinesis }
+{4018}{}{is prevented by controlling the disposition of the}
+{4019}{}{kinetochore fibers during anaphase. The entire }
+{4020}{}{mitotic cycle lasts approximately 2 hours, although}
+{4021}{}{phenotypical expression of the new structure may }
+{4022}{}{take days or weeks to become apparent.}
+{4023}{}{**END-PAR**}
+# -
+{4024}{}{In addition to an effective immunity to bacterial }
+{4025}{}{and viral agents, the quad-helix structure is }
+{4026}{}{almost entirely immune to errors introduced in }
+{4027}{}{base pairing during replication due to multiplicity}
+{4028}{}{of the base sequence. Radiation-exposed tissue }
+{4029}{}{showed no mutation in the base sequence and protein}
+{4030}{}{synthesis mechanisms in the ribosomes were unimpaired.}
+{4031}{}{**END-PAR**}
+# -
+{4032}{}{Over 80% of the samples tissue contained quad-helix}
+{4033}{}{DNA. Most affected was muscle and bone tissue, as }
+{4034}{}{well as secretory cells, as these cells seem most }
+{4035}{}{receptive to virion. Sensory cells are the least }
+{4036}{}{affected. Surprisingly, even normally non-replicating}
+{4037}{}{nerve cells and nonsomatic cell were induced to }
+{4038}{}{begin mitosis. Further experiments will be necessary}
+{4039}{}{to determine the results on these cells.}
+{4040}{}{**END-DISK**}
+
+
+# *** Delta Experiment Disk ***
+{5000}{}{The Military has deemed it necessary for us to research}
+{5001}{}{further in depth the effects of wave technology upon}
+{5002}{}{living organisms. We have taken light and sound as the}
+{5003}{}{basis of our studies. By manipulating the amplitude of}
+{5004}{}{the light waves and magnifying the frequency, we have}
+{5005}{}{been able to get lasers which will cut through a few}
+{5006}{}{feet a steel. Unfortunately, the power to do such a}
+{5007}{}{task has not been fully developed. We have other}
+{5008}{}{scientists looking into this.}
+{5009}{}{**END-DISK**}
+
+
+# *** Vree's Experiment Disk ***
+{6000}{}{•  Initial Observations}
+{6001}{}{**END-PAR**}
+# -
+{6002}{}{This is truly amazing. Some of the Knights on a patrolling}
+{6003}{}{expedition came across an unusual creature. This creature}
+{6004}{}{appears to be humanoid, and quite possibly was once of a}
+{6005}{}{human state. However, there are many differences in the }
+{6006}{}{structure of this creature than that of normal humans.}
+{6007}{}{**END-PAR**}
+# -
+{6008}{}{In the initial investigation of this creature, it is }
+{6009}{}{discovered to have a cellular structure akin to that of}
+{6010}{}{humans. Before any possible decomposition can take }
+{6011}{}{place, I am taking down the statistics of this subject.}
+{6012}{}{**END-PAR**}
+# -
+{6013}{}{•  Statistics of Subject A:}
+{6014}{}{**END-PAR**}
+# -
+{6015}{}{Height: 3.2 meters}
+{6016}{}{Mass:    363.21 kg}
+{6017}{}{Gender: Indeterminate}
+{6018}{}{Skin Color: Predominantly gray with tints of green }
+{6019}{}{under the current lighting system. Uncertain if this is}
+{6020}{}{due to decay or exposure in the wastelands. Note: The }
+{6021}{}{skin is extremely tough with respect to scalpels.}
+{6022}{}{**END-PAR**}
+{6023}{}{Mass Breakdown:}
+{6024}{}{    Muscle Mass: 77.41% }
+{6025}{}{    Bone Mass:   10.23%}
+{6026}{}{    Fat Mass:     3.02%}
+{6027}{}{    Tissue Mass:  9.34%}
+{6028}{}{**END-PAR**}
+{6029}{}{Cellular Structure:}
+{6030}{}{    Cells undergo cellular division at an increased rate.}
+{6031}{}{    Mitosis occurs at a rate 15% quicker than that of }
+{6032}{}{    normal humans.}
+{6033}{}{    Cellular structure appears to be highly similar to}
+{6034}{}{    humans.}
+{6035}{}{    Genetic structure shows a strong correlation}
+{6036}{}{    between the subject and Homo Sapiens. Possible a}
+{6037}{}{    mutation from the nuclear and biochemical agents}
+{6038}{}{    remnant from the war.}
+{6039}{}{    DNA strands appear to be very complete. All recessive}
+{6040}{}{    genes for ailments appear to have been eradicated}
+{6041}{}{    from the system.}
+{6042}{}{    The RNA strands also appear to have been manipulated}
+{6043}{}{    to allow for a greater transmission of signals.}
+{6044}{}{**END-PAR**}
+# -
+{6045}{}{•  Initial Hypothesis}
+{6046}{}{**END-PAR**}
+# -
+{6047}{}{Based on the increased size of the neural transmitters}
+{6048}{}{and synaptic receivers, I would hypothesize the}
+{6049}{}{subject had acute reflexes and heightened senses. Based}
+{6050}{}{on the reports the Knights gave of the area in which the}
+{6051}{}{subject was discovered, barren, high radiation, extremely}
+{6052}{}{high concentrations of chemical agents, it is a wonder}
+{6053}{}{the subject survived as long as it did. Performing tests}
+{6054}{}{to determine possible cause of death.}
+{6055}{}{**END-PAR**}
+# -
+{6056}{}{•  Results from tests conducted upon Subject A:}
+{6057}{}{**END-PAR**}
+# -
+{6058}{}{Visual Inspection:}
+{6059}{}{    Black powder burns near the area suggest possible }
+{6060}{}{    bullet.}
+{6061}{}{    1.2 cm lacerations upon the calf of the right leg }
+{6062}{}{    appear to have been made by teeth. Will conduct}
+{6063}{}{    test for possible Rabies.}
+{6064}{}{    Skin dried out and flaking. Possible exposure.}
+{6065}{}{**END-PAR**}
+# -
+{6066}{}{Test Results:}
+{6067}{}{    Radiation Count:  12 rads.}
+{6068}{}{    Rabies test:      Clean.}
+{6069}{}{    Hydrochloric Gas: Clean}
+{6070}{}{    Chlorine Level:   .07%}
+{6071}{}{    Sulfuric Content: .02%}
+{6072}{}{    Phase Shifting Virus:    Clean}
+{6073}{}{    Gamma Cyclotronic Virus: Clean}
+{6074}{}{    Forced Evolutionary Virus II: Severe overdose}
+{6075}{}{**END-PAR**}
+# -
+{6076}{}{•  Hypothesis}
+{6077}{}{**END-PAR**}
+# -
+{6078}{}{Based on my observations, I would hypothesize that the }
+{6079}{}{test subject has been killed in a severe fight of at }
+{6080}{}{least two people and three animals the size of dogs.}
+{6081}{}{What is truly astonishing is the extent of viral}
+{6082}{}{infection in the subject. I had read once that some}
+{6083}{}{pre-War scientists were conducting experiments with}
+{6084}{}{such a virus, but all of the research notes were}
+{6085}{}{destroyed.}
+{6086}{}{**END-PAR**}
+# -
+{6087}{}{Research into this virus has lead to many interesting }
+{6088}{}{discoveries. The test subject has gained many of its }
+{6089}{}{mutations from the overdose of the virus. This would }
+{6090}{}{account for the enhanced muscle and bone structure. }
+{6091}{}{Additionally, the recessive genes which are commonly }
+{6092}{}{found in humans have been manipulated in such a way as }
+{6093}{}{to bring about the best possible combination. While}
+{6094}{}{the process by which this happens is uncertain, it does}
+{6095}{}{have some severe side effects. Chief among them is}
+{6096}{}{sterility. The test subject would have been unable to}
+{6097}{}{reproduce with any creature, whether clean or mutated.}
+{6098}{}{Other side effects include an alteration of pigment of}
+{6099}{}{the epidermis. The life expectancy is increased by 10%.}
+{6100}{}{Intellect is decreased by this strain by 30%.}
+{6101}{}{**END-PAR**}
+# -
+{6102}{}{•  Conclusions}
+{6103}{}{**END-PAR**}
+# -
+{6104}{}{Based on this information, I would extrapolate that we }
+{6105}{}{could simply outlive these mutants. However, based on}
+{6106}{}{the fact that these mutants have a super high}
+{6107}{}{concentration of the virus, it stands to reason that}
+{6108}{}{there could be some place which is creating them.}
+{6109}{}{As to where they could be, I cannot hazard a guess.}
+{6110}{}{**END-DISK**}
+
+
+# *** Brotherhood of Steel Honor Code ***
+# *** Maxson's History ***
+{7000}{}{My father was a security guard at a secret military base}
+{7001}{}{in the desert of southern California. A typical MP, I}
+{7002}{}{remember mostly his strength. When it came time for}
+{7003}{}{the revolution, I respected his convictions. He stayed}
+{7004}{}{behind, to help those who were disabled and wounded,}
+{7005}{}{even the scis. He put the well being of myself and my}
+{7006}{}{mother into the hands of his best friend, and ordered}
+{7007}{}{us into the desert with the other rebels.}
+{7008}{}{**END-PAR**}
+# -
+{7009}{}{We, very few, marched into the wastes. The only thought}
+{7010}{}{on my mind was that I would never see my father again.}
+{7011}{}{He knew that to stay behind was death. And still, he}
+{7012}{}{stayed. He respected the flag, the CIC and the badge}
+{7013}{}{that he wore.}
+{7014}{}{**END-PAR**}
+# -
+{7015}{}{What an idiot.}
+{7016}{}{**END-PAR**}
+# -
+{7017}{}{He died for the sins of others. That will never happen}
+{7018}{}{again to us. We will become self-sufficient. We will}
+{7019}{}{become keepers of knowledge and lore. We will survive}
+{7020}{}{the end of civilization. We will take responsibility}
+{7021}{}{for our actions, and we will hold accountable the}
+{7022}{}{actions of others.}
+{7023}{}{**END-PAR**}
+# -
+{7024}{}{This I pledge to you, Maxson, my son. The Brotherhood}
+{7025}{}{of Steel is justly named. We are a Brotherhood. Unlike}
+{7026}{}{my father, we will stand back to back with those that}
+{7027}{}{share our convictions and beliefs. We are Steel. We are}
+{7028}{}{hard. We have been sharpened to and edge.}
+{7029}{}{**END-PAR**}
+# -
+{7030}{}{Always remember the fires that we were forged in.}
+{7031}{}{Never forget. }
+{7032}{}{The motto from a previous time, and our motto now.}
+{7033}{}{**END-DISK**}
+
+# *** Mutant disk *** (bug fixed by WG: 8011 was duplicated)
+{8000}{}{Message sent: Base, this is Scavenger Team Two,}
+{8001}{}{come in. Over.}
+{8002}{}{**END-PAR**}
+# -
+{8003}{}{Message received: We're reading you loud 'n clear, go}
+{8004}{}{ahead. Over.}
+{8005}{}{**END-PAR**}
+# -
+{8006}{}{Message sent: We've collected four males, two females,}
+{8007}{}{and their supplies from the caravan. They're not too }
+{8008}{}{badly contaminated. Tell the Lieutenant that we're }
+{8009}{}{sending them ahead, they might be able to survive the}
+{8010}{}{process. We're going to wait here for a few more days}
+{8011}{}{ to try to intercept another caravan. Over.}
+{8012}{}{**END-PAR**}
+# -
+{8013}{}{Message received: Roger that. Were any casualties }
+{8014}{}{accrued during the acquisition? Over.}
+{8015}{}{**END-PAR**}
+# -
+{8016}{}{Message sent: Negative. Another clean sweep. Base, also}
+{8017}{}{tell Scavenger Team one that we've run across the same}
+{8018}{}{problem they had. One of our scouts did not return from}
+{8019}{}{his perimeter patrol. A few of our other scouts reported}
+{8020}{}{that there was something big and fast seen near our}
+{8021}{}{camp last night. I'll be checking it out personally }
+{8022}{}{at 0600 hours tomorrow morning. Over.}
+{8023}{}{**END-PAR**}
+# -
+{8024}{}{Message received: Copy. We'll be awaiting your report.}
+{8025}{}{If possible, try to capture it. The Master would be}
+{8026}{}{very pleased. Over.}
+{8027}{}{**END-DISK**}
+
+
+# *** Military Base Disk***
+{9000}{}{•  Oct 10, 2077}
+{9001}{}{I, Roger Maxson, Captain, serial number 072389 have}
+{9002}{}{started this log because it doesn't look good for any of}
+{9003}{}{us, and I'd like for people to know what really happened}
+{9004}{}{here. }
+{9005}{}{**END-PAR**}
+# -
+{9006}{}{All hell broke loose when we finally discovered what}
+{9007}{}{those scientist bastards were up to. The Colonel has}
+{9008}{}{locked himself in his office and seems to be having}
+{9009}{}{some sort of breakdown. The men are screaming for blood.}
+{9010}{}{They're looking to me for answers, and I'm not sure}
+{9011}{}{what to do. Someone has to do something, though, before}
+{9012}{}{this place sinks into an anarchistic bloodbath.}
+{9013}{}{**END-PAR**}
+# -
+{9014}{}{•  Oct. 12 2077}
+{9015}{}{Every time we get a report from higher up things get}
+{9016}{}{worse here. The war is going in a very bad direction and}
+{9017}{}{this place is about to go into full mutiny, with all the}
+{9018}{}{chaos it brings. I stopped one of the men from executing}
+{9019}{}{a scientist today, and demanded that we interrogate}
+{9020}{}{them to find out what their orders were.}
+{9021}{}{**END-PAR**}
+# -
+{9022}{}{•  Oct.13 2077 }
+{9023}{}{I killed a man today. I was interrogating Chief}
+{9024}{}{Scientist Anderson and he was giving me the full details}
+{9025}{}{of their inhuman experiments. He said his orders came}
+{9026}{}{from the government, but I didn't buy it. He started}
+{9027}{}{screaming about how he was following orders, how he was}
+{9028}{}{a military man, and I just shot him. I tell myself it}
+{9029}{}{was to keep him from causing a full mutiny among the}
+{9030}{}{men, but I'm not so sure.}
+{9031}{}{**END-PAR**}
+# -
+{9032}{}{•  Oct.15 2077}
+{9033}{}{I tried again to speak to the colonel through the door, }
+{9034}{}{but he seems to have completely lost touch with reality.}
+{9035}{}{I broke down the door with several of the men just in }
+{9036}{}{time to watch him blow his head off. Right before he }
+{9037}{}{pulled the trigger he said he was sorry.}
+{9038}{}{**END-PAR**}
+# -
+{9039}{}{•  Oct. 18 2077}
+{9040}{}{By killing the egghead, I seem to have confirmed my }
+{9041}{}{position as leader of the men. They follow me without }
+{9042}{}{question now. The interrogations invariably end up being}
+{9043}{}{executions. Shellman held out the longest, but the end }
+{9044}{}{result was the same. Her arguments about orders were}
+{9045}{}{a bit too specific to be completely made up. I'm getting}
+{9046}{}{a real bad feeling in my gut about how this is all going}
+{9047}{}{to end up. I don't even lie to myself anymore about my }
+{9048}{}{reasons for executing the scientists.}
+{9049}{}{**END-PAR**}
+# -
+{9050}{}{•  Oct.20 2077}
+{9051}{}{I finally replied to the outside world over our radio.}
+{9052}{}{I don't know why they never sent anyone here to see}
+{9053}{}{what was happening when we stopped responding to their}
+{9054}{}{transmissions. It doesn't make any sense. Well, they'll}
+{9055}{}{come now. I declared ourselves seceded from the union.}
+{9056}{}{Jefferson Davis, eat your heart out.}
+{9057}{}{**END-PAR**}
+# -
+{9058}{}{•  Oct.22 2077}
+{9059}{}{What the hell is going on? We declare ourselves to be in}
+{9060}{}{full desertion from the army and no longer under the}
+{9061}{}{Government's command and what happens? Nothing.}
+{9062}{}{Something bad is coming down.}
+{9063}{}{**END-PAR**}
+# -
+{9064}{}{•  Oct. 23 2077}
+{9065}{}{I can't believe those bastards finally did it. Damn them}
+{9066}{}{all to hell. They finally let the A-bombs fly. We were }
+{9067}{}{right in the middle of trying to pry the real story out }
+{9068}{}{of von Felden when we completely lost contact. I have a }
+{9069}{}{feeling the research center was hit hard. I don't know }
+{9070}{}{why, just call it a gut feeling. It seems inconceivable }
+{9071}{}{that we were not targeted. I'm sure China will make up }
+{9072}{}{for that oversight real soon. Luckily, we had moved our }
+{9073}{}{families from outside into the facility the day before }
+{9074}{}{yesterday. We do not yet know if the fallout has reached }
+{9075}{}{this area.}
+{9076}{}{**END-PAR**}
+# -
+{9077}{}{•  Oct. 25 2077}
+{9078}{}{Sgt. Platner volunteered to go outside today to take }
+{9079}{}{specific readings on the atmosphere. It seems the }
+{9080}{}{radiation has not spread this far. Since he was wearing }
+{9081}{}{his Power Armor, there was no threat to him from }
+{9082}{}{radiation, but if he had been exposed he would have had }
+{9083}{}{to be exiled. We don't have adequate decontamination }
+{9084}{}{facilities here.}
+{9085}{}{**END-PAR**}
+# -
+{9086}{}{•  Oct. 26 2077}
+{9087}{}{I convinced the men that we should bury the scientists. }
+{9088}{}{I don't know why... perhaps it was to ease my conscience.}
+{9089}{}{I finally started to believe their stories when the last }
+{9090}{}{one was dying. }
+{9091}{}{**END-PAR**}
+# -
+{9092}{}{My God, what have I become? }
+{9093}{}{**END-PAR**}
+# -
+{9094}{}{•  Oct. 27 2077}
+{9095}{}{We're leaving this godforsaken place today. I'm leading }
+{9096}{}{the exodus to the old government bunker at Lost Hills. }
+{9097}{}{I'm leaving this log behind to be buried when this place}
+{9098}{}{goes in the next exchange. Who knows, maybe someone}
+{9099}{}{will find it someday...}
+{9100}{}{**END-DISK**}
+
+# *** Sophia Disk ***
+{10000}{}{The Exodus from the accursed base was a trying time for}
+{10001}{}{the men and their families. While there was no }
+{10002}{}{radioactive fallout to contend with, they were }
+{10003}{}{frequently beset by the fallout of humanity. Roving }
+{10004}{}{bands of psychotic marauders attempted several attacks }
+{10005}{}{on that noble group. }
+{10006}{}{**END-PAR**}
+# -
+{10007}{}{The company itself was in no danger, for they wore the }
+{10008}{}{Armor of Power. Members of their families were not so }
+{10009}{}{lucky. Once the vermin found out they were easily }
+{10010}{}{repelled, they began to fire on the unarmed civilians }
+{10011}{}{from a distance. }
+{10012}{}{**END-PAR**}
+# -
+{10013}{}{They took a great many casualties, yet for every member}
+{10014}{}{of the Exodus that was struck down in this way, our }
+{10015}{}{noble brethren took two lives from the wasteland.}
+{10016}{}{**END-PAR**}
+# -
+{10017}{}{Finally, the forefathers came to the safety of the }
+{10018}{}{bunker. Capt. Maxson, the great deliverer, decreed this}
+{10019}{}{to be our new home, and all was well.}
+{10020}{}{**END-PAR**}
+# -
+{10021}{}{In the fullness of time the bunker became our home, our}
+{10022}{}{temple and our salvation from the terrors of the outside}
+{10023}{}{world. We began to build and shape our fortress into }
+{10024}{}{something glorious, the beauty of which the }
+{10025}{}{technologically bereft world had never seen before.}
+{10026}{}{**END-PAR**}
+# -
+{10027}{}{Yet there were those who sought still more. These }
+{10028}{}{restless souls demanded we look to the southeast for }
+{10029}{}{the advanced technology that was supposedly housed }
+{10030}{}{there.}
+{10031}{}{**END-PAR**}
+# -
+{10032}{}{Capt. Maxson warned these impetuous youths that the}
+{10033}{}{research facility was doubtlessly destroyed when we}
+{10034}{}{were spared, but they would not hear his words. They}
+{10035}{}{took their sanctified armor and headed off to find their}
+{10036}{}{Holy Grail, but not before they spoke the Deliverer's}
+{10037}{}{name in vain, questioning his very bravery!}
+{10038}{}{**END-PAR**}
+# -
+{10039}{}{These men were never heard from again.}
+{10040}{}{**END-DISK**}
+
+# *** Maxson Disk ***
+{11000}{}{By my orders, as active commanding officer following}
+{11001}{}{the untimely death of Colonel Robert Spindel during}
+{11002}{}{this time of crisis, the full base security team has}
+{11003}{}{been deployed to the security bunker at Lost Hills.}
+{11004}{}{**END-PAR**}
+# -
+{11005}{}{This directive also includes the families of the}
+{11006}{}{officers and enlisted men.}
+{11007}{}{**END-PAR**}
+# -
+{11008}{}{Unless otherwise directed, from a proper }
+{11009}{}{representative of the War Department, this order }
+{11010}{}{will stand as written.}
+{11011}{}{**END-PAR**}
+# -
+{11012}{}{Operative 1: All military personnel, and their }
+{11013}{}{families, are to vacate the base by 0800, 25 Oct 2077.}
+{11014}{}{All personnel, traveling under command, will make }
+{11015}{}{their way to Lost Hills base. No leave has been granted.}
+{11016}{}{**END-PAR**}
+# -
+{11017}{}{Operative 2: All civilian personnel are directed to }
+{11018}{}{remain at base, pending orders from their legal command}
+{11019}{}{structure.}
+{11020}{}{**END-PAR**}
+# -
+{11021}{}{Operative 3: Equipment deemed necessary to the survival}
+{11022}{}{of base military personnel is to be immediately drawn }
+{11023}{}{from stores. Proper authorization will follow, time }
+{11024}{}{permitting.}
+{11025}{}{**END-PAR**}
+# -
+{11026}{}{Operative 4: All codes of military justice will be }
+{11027}{}{harshly enforced, on military personnel and civilian }
+{11028}{}{personnel in joint military operations.}
+{11029}{}{**END-PAR**}
+# -
+{11030}{}{Operative 5: Until such time as consistent and }
+{11031}{}{authorized communication can be established with the }
+{11032}{}{War Department, these orders will have precedence over}
+{11033}{}{any previously established orders.}
+{11034}{}{**END-PAR**}
+# -
+{11035}{}{Captain Maxson}
+{11036}{}{24 Oct 2077}
+{11037}{}{**END-DISK**}
+
+# *** Richard Grey audio diary ***
+{12000}{}{                                                    0000 - 0004}
+{12001}{}{I'm dying. I need to get this down before the pain}
+{12002}{}{overwhelms me.}
+{12003}{}{**END-PAR**}
+# -
+{12004}{}{                                                    0005 - 0020}
+{12005}{}{I can't believe that I was finally able to drag myself}
+{12006}{}{out of that vat. The slime did not affect me, but I}
+{12007}{}{nearly drowned. I don't know what happened to Harold-}
+{12008}{}{he was standing right next to me when the crane}
+{12009}{}{knocked me into the vat. He must have been killed or}
+{12010}{}{he would have tried to help me. Francine is dead, killed}
+{12011}{}{by one of those robots.}
+{12012}{}{**END-PAR**}
+# -
+{12013}{}{                                                    0021 - 0027}
+{12014}{}{I have no idea how much time has passed. I was able to}
+{12015}{}{hack the computer to turn off the robots and record}
+{12016}{}{this, but now my mind is slipping away.}
+{12017}{}{There is much pain...}
+{12018}{}{**END-PAR**}
+# -
+{12019}{}{                                                    0028 - 0043}
+{12020}{}{The green slime that I was immersed in is the source}
+{12021}{}{of all the mutations we traced to here. My skin is}
+{12022}{}{to fester and peel. In other areas it is bubbling and}
+{12023}{}{starting expelling a green mucus-like substance. Some}
+{12024}{}{days the pain is almost tolerable.}
+{12025}{}{**END-PAR**}
+# -
+{12026}{}{                                                    0044 - 0057}
+{12027}{}{I can actually walk a few steps again... It seems}
+{12028}{}{inconceivable that I dragged myself all the way up here}
+{12029}{}{from the vat room. Strangely, I'm actually feeling}
+{12030}{}{stronger, though I'm still in a lot of pain. Everything}
+{12031}{}{seems to be getting smaller.}
+{12032}{}{**END-PAR**}
+# -
+{12033}{}{                                                    0058 - 0096}
+{12034}{}{I think I consumed one of the mutated things scurrying}
+{12035}{}{around here today. Before I knew what was happening,}
+{12036}{}{some sort of tendril had sprang from my stomach and}
+{12037}{}{covered the poor creature. As soon as it had sucked}
+{12038}{}{the rodent into my gut I could actually feel its mind.}
+{12039}{}{I think. There is the very real possibility that I'm}
+{12040}{}{going slowly insane and can no longer differentiate}
+{12041}{}{between what is real and what is a hallucination.}
+{12042}{}{Maybe I'm still slowly dying in the vat and I've}
+{12043}{}{imagined all this.}
+{12044}{}{**END-PAR**}
+# -
+{12045}{}{                                                    0097 - 0111}
+{12046}{}{Things are becoming more clear to me every day.}
+{12047}{}{This toxin has actually improved my mind. I feel that}
+{12048}{}{I can understand even the most complex philosophical}
+{12049}{}{questions simply and directly. It's as if all the layers}
+{12050}{}{of artifice have been stripped away. I wonder what}
+{12051}{}{would happen if I submerged an animal in the vats for}
+{12052}{}{a prolonged period of time? Would it gain awareness?}
+{12053}{}{**END-PAR**}
+# -
+{12054}{}{                                                    0112 - 0134}
+{12055}{}{The strangest thing is happening to the animals.}
+{12056}{}{They actually become smarter and more aware of their}
+{12057}{}{surroundings. I dipped a dog and a rat at the same}
+{12058}{}{time today, and they were fused together. It's not}
+{12059}{}{quite two creatures anymore, but it's more than one.}
+{12060}{}{Perhaps this is the future - a coming together of}
+{12061}{}{different creatures in some sort of harmonious unity.}
+{12062}{}{I no longer consume the different animals I create}
+{12063}{}{simply for sustenance - I have become the instrument}
+{12064}{}{through which unity will be achieved. I am so much}
+{12065}{}{more than a human being now.}
+{12066}{}{**END-PAR**}
+# -
+{12067}{}{                                                    0135 - 0139}
+{12068}{}{It is time to bring others into the glory that is the}
+{12069}{}{Unity.}
+{12070}{}{**END-PAR**}
+# -
+{12071}{}{                                                    0140 - 0153}
+{12072}{}{A lost soul has finally strayed into my home. I was so}
+{12073}{}{surprised I consumed him before dipping, a mistake I}
+{12074}{}{shall not make again. His mind was so primitive as to}
+{12075}{}{be repulsive to my refined cognitive abilities.}
+{12076}{}{**END-PAR**}
+# -
+{12077}{}{                                                    0154 - 0172}
+{12078}{}{I've begun to modify myself to be more pleasing to the}
+{12079}{}{Unity by injecting small doses of the virus into my body.}
+{12080}{}{The slime in the vats is a man-made virus called the}
+{12081}{}{Forced Evolutionary Virus. This information was acquired}
+{12082}{}{from my newly grown neurolink with the base computer.}
+{12083}{}{**END-PAR**}
+# -
+{12084}{}{                                                     0173 - 0198}
+{12085}{}{The few wanderers that have found their way here have}
+{12086}{}{been a disappointment to me. They can't seem to mutate}
+{12087}{}{correctly. The best I've been able to create are some}
+{12088}{}{big and dumb mutants. Most can recall nothing from}
+{12089}{}{before I initiated them into the wondrous Unity. I only}
+{12090}{}{feed on them for fuel, now. Their minds are nothing}
+{12091}{}{to me.}
+{12092}{}{**END-PAR**}
+# -
+{12093}{}{                                                     0199 - 0236}
+{12094}{}{Oh glorious creator!! I have succeeded in spreading the}
+{12095}{}{complete joy of unification to another soul! Unlike the}
+{12096}{}{others, his total radiation count was low. I believe this}
+{12097}{}{is the factor we have been overlooking all this time,}
+{12098}{}{as it seems the conversion is more successful in the}
+{12099}{}{cases with less radiation damage. I have never known}
+{12100}{}{such glory as I felt when taking his mind into our own.}
+{12101}{}{**END-PAR**}
+# -
+{12102}{}{                                                     0237 - 0281}
+{12103}{}{We are beginning to create an army dedicated to}
+{12104}{}{unifying the wonderful diversity of life. We have trained}
+{12105}{}{them to continue our work here while we search out}
+{12106}{}{more populated areas to take into ourselves. We are}
+{12107}{}{beginning to feel the limitations of a body that is mobile.}
+{12108}{}{We must find a permanent home, with a greater store of}
+{12109}{}{knowledge, and a steady supply of bio-mass.}
+{12110}{}{**END-PAR**}
+# -
+{12111}{}{                                                     0282 - 0303}
+{12112}{}{We have stopped increasing ourself until we can find this}
+{12113}{}{new unification center. When we have arrived we will}
+{12114}{}{continue to grow and feed until we have brought peace}
+{12115}{}{and unity to the entire world.}
+{12116}{}{**END-DISK**}
+
+# *** Captain Maxson's Diary ***
+{13000}{}{•  Oct 10, 2077}
+{13001}{}{I, Roger Maxson, Captain, serial number 072389 have}
+{13002}{}{started this log because it doesn't look good for any}
+{13003}{}{of us, and I'd like for people to know what really}
+{13004}{}{happened here. All hell broke loose when we finally}
+{13005}{}{discovered what those scientist bastards were up to.}
+{13006}{}{The Colonel has locked himself in his office and seems}
+{13007}{}{to be having some sort of breakdown. The men are}
+{13008}{}{screaming for blood. They're looking to me for answers,}
+{13009}{}{and I'm not sure what to do. Someone has to do}
+{13010}{}{something, though, before this place sinks into an}
+{13011}{}{anarchistic bloodbath.}
+{13012}{}{**END-PAR**}
+# -
+{13013}{}{•  Oct. 12 2077}
+{13014}{}{Every time we get a report from higher up things get}
+{13015}{}{worse here. The war is going in a very bad direction}
+{13016}{}{and this place is about to go into full mutiny, with all}
+{13017}{}{the chaos that entails. I stopped one of the men from}
+{13018}{}{executing a scientist today, and demanded that we}
+{13019}{}{interrogate them to find out what their orders were.}
+{13020}{}{**END-PAR**}
+# -
+{13021}{}{•  Oct.13 2077}
+{13022}{}{I killed a man today. I was interrogating  chief scientist}
+{13023}{}{Anderson  and he was giving me the full details of their}
+{13024}{}{inhuman experiments. He said his orders came from the}
+{13025}{}{Gov't., but I didn't buy it. He started screaming about}
+{13026}{}{how he was following orders, how he was a military man,}
+{13027}{}{and I just shot him. I tell myself it was to keep him from}
+{13028}{}{causing a full mutiny among the men, but I'm not so sure.}
+{13029}{}{**END-PAR**}
+# -
+{13030}{}{•  Oct.15 2077}
+{13031}{}{I tried again to speak to the colonel through the door,}
+{13032}{}{but he seems to have completely lost touch with reality.}
+{13033}{}{I broke down the door with several of the men just in}
+{13034}{}{time to watch him blow his head off. Right before he}
+{13035}{}{pulled the trigger he said he was sorry.}
+{13036}{}{**END-PAR**}
+# -
+{13037}{}{•  Oct. 18 2077}
+{13038}{}{By killing the egghead, I seem to have confirmed my}
+{13039}{}{position as leader of the men. They follow me without}
+{13040}{}{question now. The interrogations invariably end up being}
+{13041}{}{executions. Shellman held out the longest, but the end}
+{13042}{}{result was the same. Her arguments about her orders}
+{13043}{}{were a bit to specific to be completely made up. I'm}
+{13044}{}{getting a real bad feeling in my gut about how this is}
+{13045}{}{all going to end up. I don't even lie to myself anymore}
+{13046}{}{about my reasons for executing the scientists.}
+{13047}{}{**END-PAR**}
+# -
+{13048}{}{•  Oct.20 2077}
+{13049}{}{I finally replied to the outside world over our radio.}
+{13050}{}{I don't know why they never sent anyone here to see}
+{13051}{}{what was happening when we stopped responding to}
+{13052}{}{their transmissions. It doesn't make any sense.}
+{13053}{}{Well, they'll come now. I declared ourselves seceded}
+{13054}{}{from the union. They remember Jefferson Davis. What}
+{13055}{}{will history say about me?}
+{13056}{}{**END-PAR**}
+# -
+{13057}{}{•  Oct.22 2077}
+{13058}{}{What the hell is going on? We declare ourselves to be}
+{13059}{}{in full desertion from the army and no longer under the}
+{13060}{}{Government's command and what happens? Nothing.}
+{13061}{}{Something bad is coming down.}
+{13062}{}{**END-PAR**}
+# -
+{13063}{}{•  Oct. 23 2077}
+{13064}{}{I can't believe those bastards finally did it. Damn them}
+{13065}{}{all to hell. They finally let the A-Bombs fly. We were}
+{13066}{}{right in the middle of trying to pry the real story out}
+{13067}{}{of von Felden when we completely lost contact. I have}
+{13068}{}{a feeling the research center was hit hard. I don't know}
+{13069}{}{why, just call it a gut feeling. It seems inconceivable}
+{13070}{}{that we were not targeted. I'm sure China will make up}
+{13071}{}{for that oversight real soon. Luckily, we had moved our}
+{13072}{}{families from outside into the facility the day before}
+{13073}{}{yesterday. We do not yet know if the fallout has}
+{13074}{}{reached this area.}
+{13075}{}{**END-PAR**}
+# -
+{13076}{}{•  Oct. 25 2077}
+{13077}{}{Sgt. Platner volunteered to go outside today to take}
+{13078}{}{specific readings on the atmosphere. It seems the}
+{13079}{}{radiation has not spread this far. Since he was wearing}
+{13080}{}{his Power Armor, there was no threat to him from}
+{13081}{}{radiation, but if he had been exposed he would have had}
+{13082}{}{to be exiled. We don't have adequate decontamination}
+{13083}{}{facilities here.}
+{13084}{}{**END-PAR**}
+# -
+{13085}{}{•  Oct. 26 2077}
+{13086}{}{I convinced the men that we should bury the scientists.}
+{13087}{}{I don't know why... perhaps it was to ease my conscience.}
+{13088}{}{I finally started to believe their stories when the last}
+{13089}{}{one was dying. My God, what have I become?}
+{13090}{}{**END-PAR**}
+# -
+{13091}{}{•  Oct. 27 2077}
+{13092}{}{We're leaving this godforsaken place today. I'm leading}
+{13093}{}{the exodus to the old government bunker at Lost Hills.}
+{13094}{}{I'm leaving this log behind to be buried when this place}
+{13095}{}{goes in the next exchange. Who knows, maybe someone}
+{13096}{}{will find it someday...}
+{13097}{}{**END-DISK**}
+
+
+# Computer Download of the FEV project
+# Uses the global var COMPUTER_FEV
+
+{14000}{}{FEV Summary Digest:}
+{14001}{}{**END-PAR**}
+{14002}{}{2073. As China became increasingly aggressive with}
+{14003}{}{their use of biological weapons, the United States}
+{14004}{}{government felt that a countermeasure was needed. }
+{14005}{}{The Pan-Immunity Virion Project (PVP) was officially}
+{14006}{}{formed September 15, 2073. }
+{14007}{}{**END-PAR**}
+#
+{14008}{}{2075. It became clear that the best way to combat the}
+{14009}{}{newly created biological weapons was to alter }
+{14010}{}{uninfected DNA so that it was no longer susceptible }
+{14011}{}{to standard viral infection.}
+{14012}{}{**END-PAR**}
+#
+{14013}{}{2076. Unforeseen side effects began surfacing in early}
+{14014}{}{2076 with the PVP. Animal test subjects began showing}
+{14015}{}{an abnormal growth rate accompanied by increased }
+{14016}{}{brain activity. The U.S. government took notice of }
+{14017}{}{these discoveries, and in the interests of national }
+{14018}{}{security, moved a team on-site to secure and oversee }
+{14019}{}{the project, which was now dubbed the FEV (Forced }
+{14020}{}{Evolutionary Virus) project.}
+{14021}{}{**END-PAR**}
+#
+{14022}{}{2077. FEV nears completion. Test on lab animals are at }
+{14023}{}{a near 100% success rate. Size and muscle density }
+{14024}{}{increase approximately 60%, and the potential }
+{14025}{}{intelligence increase by 200%. Effects upon human }
+{14026}{}{subjects remain unknown; although they are theoretically}
+{14027}{}{promising. The military, wishing to continue further }
+{14028}{}{testing, builds a large facility at the Mariposa }
+{14029}{}{military installation in central California. At this }
+{14030}{}{new facility, testing of the FEV virus continues on }
+{14031}{}{volunteer subjects from the military.}
+{14032}{}{**END-DISK**}
+
+
+# Computer Download of the Power Armor Specs
+# Uses the global var COMPUTER_POWER_ARMOR
+
+{15000}{}{The T-51b Powered Infantry Armor is designed with the}
+{15001}{}{latest passive defense features for both civilian and}
+{15002}{}{military disturbances. The back-mounted TX-28}
+{15003}{}{MicroFusion Pack generates 60,000 Watts to power the}
+{15004}{}{HiFlo hydraulic systems built into the frame of the suit.}
+{15005}{}{Made of the latest poly-laminate composite, the T-51b}
+{15006}{}{shell is lightweight and capable of absorbing over 2500}
+{15007}{}{Joules of kinetic impact. The 10 micron silver ablative}
+{15008}{}{coating can reflect laser and radiation emissions without}
+{15009}{}{damage to the composite subsurface.}
+{15010}{}{**END-DISK**}
+
+
+# Computer Download of the Personnel Records
+# Uses the global var COMPUTER_RECORDS
+
+{16000}{}{Robert Anderson}
+{16001}{}{Research Assistant}
+{16002}{}{Security Clearance: Blue}
+{16003}{}{Stationed: Mariposa Military Base}
+{16004}{}{**END-PAR**}
+#
+{16005}{}{Nick Davis}
+{16006}{}{Director of NBC Division of West Tech Research}
+{16007}{}{Security Clearance: Blue}
+{16008}{}{Stationed: West Tech Research Facility}
+{16009}{}{**END-PAR**}
+#
+{16010}{}{John Isaac}
+{16011}{}{Research Assistant}
+{16012}{}{Security Clearance: Red}
+{16013}{}{Stationed: West Tech Research Facility}
+{16014}{}{**END-PAR**}
+#
+{16015}{}{Steve Remco}
+{16016}{}{Research Assistant}
+{16017}{}{Security Clearance: Blue}
+{16018}{}{Stationed: Mariposa Military Base}
+{16019}{}{**END-PAR**}
+#
+{16020}{}{Charles Rignhold}
+{16021}{}{Research Head of Laser Development}
+{16022}{}{Security Clearance: Blue}
+{16023}{}{Stationed: West Tech Research Facility}
+{16024}{}{**END-PAR**}
+#
+{16025}{}{Michele Santose}
+{16026}{}{Lab Technician}
+{16027}{}{Security Clearance: Red}
+{16028}{}{Stationed: West Tech Research Facility}
+{16029}{}{**END-PAR**}
+#
+{16030}{}{Erin Shellman}
+{16031}{}{Lab Technician}
+{16032}{}{Security Clearance: Blue}
+{16033}{}{Stationed: Mariposa Military Base}
+{16034}{}{**END-PAR**}
+#
+{16035}{}{Leon VonFelden}
+{16036}{}{Research Head of FEV}
+{16037}{}{Security Clearance: Blue}
+{16038}{}{Stationed: Mariposa Military Base}
+{16039}{}{**END-DISK**}
+
+# Holodisk from the Brotherhood Library
+# uses the global var BROTHERHOOD_LIBRARY_DISK
+
+{17000}{}{Vault Locations v34.129 Revision C (Western USA)}
+{17001}{}{**END-PAR**}
+#
+{17002}{}{Hello, and welcome to the latest edition of the Western}
+{17003}{}{United States Vault Location Pamphlet. We hope you}
+{17004}{}{enjoy the pamphlet, and remember Vaults are not just}
+{17005}{}{for the upwardly mobile, but for everyone.}
+{17006}{}{**END-PAR**}
+#
+{17007}{}{Vault 12}
+{17008}{}{**END-PAR**}
+#
+{17009}{}{Under the sprawling metropolis of Bakersfield, lies the}
+{17010}{}{technological magnificence of Vault 12. Built with every}
+{17011}{}{amenity in mind for the prospective Vault Dweller,}
+{17012}{}{Vault 12 was given the "Pressed Vault Suit"}
+{17013}{}{award for attention to preparedness. Buried far}
+{17014}{}{underground, the protection that Vault Dwellers will}
+{17015}{}{receive will be unprecedented. Much like all other}
+{17016}{}{Vaults, Vault 12 has been fitted with the newest in}
+{17017}{}{Vault Water Purification Systems. Able to take even}
+{17018}{}{the waste located in the sewers of Bakersfield, this}
+{17019}{}{system is able to deliver over 15000 gallons of pure,}
+{17020}{}{refreshing drinking water every day.}
+{17021}{}{**END-PAR**}
+#
+{17022}{}{Vault 13}
+{17023}{}{**END-PAR**}
+#
+{17024}{}{Located in a scenic mountainous region northwest of}
+{17025}{}{Vault 12, this Vault offers an endless supply of pure}
+{17026}{}{water to the inhabitants. The rumors that the water}
+{17027}{}{table in the area could easily be contaminated in the}
+{17028}{}{event of a Nuclear War have been found by the}
+{17029}{}{Department of Water and Power to be completely}
+{17030}{}{unfounded. In the event that the water in the area}
+{17031}{}{were to become tainted, remote even though it may be,}
+{17032}{}{Vault 13 has been fitted with the government approved}
+{17033}{}{Vault Water Purification System. Rated to work without}
+{17034}{}{significant loss of output for over 250,000 hours,}
+{17035}{}{prospective Vault Dwellers should  have nothing to fear.}
+{17036}{}{**END-PAR**}
+#
+{17037}{}{Vault 15}
+{17038}{}{**END-PAR**}
+#
+{17039}{}{Due east from Vault 13, construction on this Vault has}
+{17040}{}{gone extremely smoothly. Much work was done to}
+{17041}{}{reinforce the walls of the third level of this Vault, to}
+{17042}{}{make all the future Vault Dwellers more secure in the}
+{17043}{}{knowledge that in the event of even a major earthquake,}
+{17044}{}{the regulatory computers of the Vault would continue}
+{17045}{}{to function. Recent tours of the newly constructed}
+{17046}{}{Vault, have had many a potential Vault Dweller walking}
+{17047}{}{away with a newfound awe of the improvements done to}
+{17048}{}{this already impressive Vault.}
+{17049}{}{**END-DISK**}
+
+#Regulator Disk from Blades
+
+{18000}{}{Regulator's radio transmission log:}
+{18001}{}{**END-PAR**}
+{18002}{}{Message sent: Regulator patrol to Regulator base come}
+{18003}{}{in. Do you copy? Over.}
+{18004}{}{**END-PAR**}
+{18005}{}{Message received: This is Regulator base, Principle}
+{18006}{}{Regulator Caleb responding. Report? Over.}
+{18007}{}{**END-PAR**}
+{18008}{}{Message sent: Sir, yes sir! Regulator Second Class}
+{18009}{}{Cravotta reporting. We've apprehended the Zimmerman}
+{18010}{}{boy trying to infiltrate our perimeter again, sir.}
+{18011}{}{Orders? Over.}
+{18012}{}{**END-PAR**}
+{18013}{}{Message received: Shit! Again! Well that's the last time}
+{18014}{}{that's going to happen. Eliminate him and make it look}
+{18015}{}{like the Blades did it.}
+{18016}{}{Understand? Over.}
+{18017}{}{**END-PAR**}
+{18018}{}{Message sent: Sir? Uh... did I copy correctly, sir?}
+{18019}{}{Kill him? Over.}
+{18020}{}{**END-PAR**}
+{18021}{}{Message received: : That's affirmative, Regulator.}
+{18022}{}{And I want it messy. Impale him on the guard posts}
+{18023}{}{out front- oh, and cut out his tongue so he }
+{18024}{}{can't talk while we 'attempt' to save his sorry ass.}
+{18025}{}{Over?}
+{18026}{}{**END-PAR**}
+{18027}{}{Message sent: It shall be done, sir, Over.}
+{18028}{}{**END-PAR**}
+{18029}{}{End transmission.}
+{18030}{}{**END-DISK**}
