diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/game/PRO_CRIT.MSG b/Fallout2/Fallout1in2/mods/fo1_base/text/english/game/PRO_CRIT.MSG
index 9e0dacbef..1d8a33a30 100644
--- a/Fallout2/Fallout1in2/mods/fo1_base/text/english/game/PRO_CRIT.MSG
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/game/PRO_CRIT.MSG
@@ -468,3 +468,4 @@
 {34401}{}{A huge albino rat.}
 {34500}{}{Floating Eye}
 {34600}{}{Rae}
+{34700}{}{Vasquez}
