diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/game/QUESTS.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/game/QUESTS.msg
new file mode 100644
index 000000000..5aa2477d0
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/game/QUESTS.msg
@@ -0,0 +1,286 @@
+#
+# Quest descriptons
+#
+#    Please leave space between the areas (like 100 or so)
+#
+
+
+#
+# Vault 13 Quests
+#
+{100}{}{Find the Water Chip.} 				#fo1 vanilla
+{101}{}{Calm the rebel faction.}			#fo1 vanilla
+{102}{}{Find the Water Thief.}				#fo1 vanilla
+{103}{}{Destroy the Mutant leader.}			#fo1 vanilla
+{104}{}{Destroy the source of the Mutants.}	#fo1 vanilla
+{105}{}{Help the medic to synthesize Antidote.}
+{106}{}{Learn about the outside world at the library.}
+{107}{}{Gain entry to the Armory.}
+{108}{}{Gain entry back into the Vault.}
+{109}{}{Kill the Overseer.}
+{110}{}{Kill all inhabitants of Vault 13.}
+{120}{}{Find the Water Chip.}
+#
+# Vault 15 Quests
+#
+{200}{}{Search for clues and information.}
+{201}{}{Search for a Water Chip.}
+{202}{}{Find a way down the elevator shaft.}
+{203}{}{Find the hidden items.}
+{204}{}{Kill all the rats.}
+#
+# Shady Sands Quests
+#
+{300}{}{Talk to Katrina.}
+{301}{}{Hire Ian.}
+{302}{}{Stop the radscorpions.}							#fo1 vanilla
+{303}{}{Help Razlo find a cure for radscorpion venom.}	#fo1 vanilla
+{304}{}{Cure Jarvis of radscorpion poison.}				#fo1 vanilla
+{305}{}{Rescue Tandi from the Khans.}					#fo1 vanilla
+{306}{}{Help Shady Sands improve their crop yield.}
+{307}{}{Lead Seth and the guards to destroy the Khans.}
+
+#
+# Junktown Quests
+#
+{400}{}{Help Saul.}										#fo1 vanilla
+{401}{}{Kill Killian.}									#fo1 vanilla
+{402}{}{Rescue Sinthia.}								#fo1 vanilla
+{403}{}{Save Trish.}									#fo1 vanilla
+{404}{}{Stop Gizmo.}									#fo1 vanilla
+{405}{}{Bust the Skulz gang.}							#fo1 vanilla
+{406}{}{Join the Skulz.}
+{407}{}{Steal Neal's urn from Skum Pitt.}
+{408}{}{RESERVED}
+
+#
+# Raiders Quests
+#
+{500}{}{Join the Khans.}
+{501}{}{Destroy the Khans.}
+{502}{}{Kill the slaves.}
+{503}{}{Save the slaves.}
+{504}{}{Shame Garl by beating him in hand-to-hand combat.}
+{505}{}{Free the prisoner.}
+{506}{}{Free Tandi.}
+{507}{}{Join the Khans on a raid of Shady Sands.}
+{508}{}{Abuse the slaves.}
+{509}{}{Replace Garl as leader of the Khans.}
+
+#
+# Necropolis Quests
+#
+{600}{}{Destroy the Super Mutants at the Watershed.}
+{601}{}{Fix the Necropolis water pump.}
+{602}{}{Save the prisoner.}
+{603}{}{Retrieve the Water Chip.}
+{604}{}{Allow the Super Mutants at the Watershed to leave peacefully.}
+{605}{}{Seduce Harry.}
+{606}{}{Retrieve reward from Set for removing the Super Mutants.}
+{607}{}{Sneak into Set's Hall of The Dead.}
+{608}{}{Find an entrance into the Watershed.}
+
+#
+# The Hub Quests
+#
+{700}{}{Find and destroy the Deathclaw.}			#fo1 vanilla
+{701}{}{Dispose of Jain.}							#fo1 vanilla
+{702}{}{Dispose of merchant.}						#fo1 vanilla
+{703}{}{Find the missing caravans.}					#fo1 vanilla
+{704}{}{Steal necklace from the Merchants.}			#fo1 vanilla
+{705}{}{Join the Thieves' Circle.}
+{706}{}{Dispose of Decker.}
+{707}{}{Send water to Vault 13.}
+{708}{}{Become a guard for the Water Merchants.}
+{709}{}{Become a guard for the Far Go Traders.}
+{710}{}{Become a guard for the Crimson Caravan.}
+{711}{}{Get rid of the raiders at Irwin's Farm.}
+
+#
+# Brotherhood of Steel Quests
+#
+{800}{}{Become an Initiate.}						#fo1 vanilla
+{801}{}{Rescue Initiate from the Hub.}				#fo1 vanilla
+{802}{}{Become a Knight.}
+{803}{}{Learn from the Library Computers.}
+{804}{}{Receive operations from Lori.}
+{805}{}{Seduce Jennifer.}
+{806}{}{Seduce Lorri.}
+{807}{}{Steal the Systolic Motivator.}
+{808}{}{Help Sophia and Michael.}
+{809}{}{Gain access to the Armory.}
+{810}{}{Find the Ruins Of The Ancient Order.}
+{811}{}{Scout the area to the North.}
+{812}{}{Get the Brotherhood to help at the Military Base.}
+{813}{}{RESERVED}
+{814}{}{RESERVED}
+{815}{}{RESERVED}
+{816}{}{RESERVED}
+{817}{}{RESERVED}
+{818}{}{RESERVED}
+
+#
+# Military Base Quests
+#
+{900}{}{Gain entry to the base.}
+{901}{}{Gain entry to the base undetected.}
+{902}{}{Trick the Mutants into leaving their guard posts.}
+{903}{}{Disable the forcefields.}
+{904}{}{Kill the Lieutenant.}
+{905}{}{Destroy the Vats.}
+
+#
+# The Glow Quests
+#
+{1000}{}{Gain entrance into the Glow.}
+{1001}{}{Disarm traps for the facility.}		#fo1 vanilla
+{1002}{}{Turn on power for the Glow.}			#fo1 vanilla
+
+#
+# L.A. Boneyard / Adytum Quests
+#
+{1100}{}{Become a Blade.}						#fo1 vanilla
+{1101}{}{Deliver Locket for Romero.}			#fo1 vanilla
+{1102}{}{Deliver package from the Gun Runners.}	#fo1 vanilla
+{1103}{}{Find Children spy in the Followers.}	#fo1 vanilla
+{1104}{}{Fix hydroponic farms in Adytum.}		#fo1 vanilla
+{1105}{}{Rescue Jason Zimmerman.}				#fo1 vanilla
+{1106}{}{Stop the Gangs from attacking Adytum.}	#fo1 vanilla
+{1107}{}{Kill the Mother Deathclaw.}
+{1108}{}{Convince the Gun Runners to supply the Blades.}
+{1109}{}{Kill the leader of the mutant army in Necropolis.}
+{1110}{}{RESERVED}
+{1111}{}{RESERVED}
+
+#
+# Cathedral / Master's Vault Quests
+#
+{1200}{}{Gain entry into the Master's Vault.}
+{1201}{}{Convince Rae to join you against the Master.}
+{1202}{}{Kill Morpheus.}
+{1203}{}{Kill the Master.}
+{1204}{}{Set the timer for the bomb.}
+
+#
+# Caravan Run Quests
+#
+{1300}{}{Help unload the Caravan.}
+
+#
+# 
+#
+{1400}{}{RESERVED}
+{1401}{}{RESERVED}
+{1402}{}{RESERVED}
+{1403}{}{RESERVED}
+
+#
+# 
+#
+{1500}{}{RESERVED}
+{1501}{}{RESERVED}
+{1502}{}{RESERVED}
+{1503}{}{RESERVED}
+{1504}{}{RESERVED}
+{1505}{}{RESERVED}
+{1506}{}{RESERVED}
+{1507}{}{RESERVED}
+{1508}{}{RESERVED}
+{1509}{}{RESERVED}
+{1510}{}{RESERVED}
+{1511}{}{RESERVED}
+{1512}{}{RESERVED}
+{1513}{}{RESERVED}
+{1514}{}{RESERVED}
+{1515}{}{RESERVED}
+{1516}{}{RESERVED}
+
+#
+# 
+#
+{1600}{}{RESERVED}
+{1601}{}{RESERVED}
+{1602}{}{RESERVED}
+
+#
+# 
+#
+{1700}{}{RESERVED}
+
+#
+# 
+#
+{1800}{}{RESERVED}
+
+#
+# 
+#
+{1900}{}{RESERVED}
+
+#
+# 
+#
+{2000}{}{RESERVED}
+
+#
+# 
+#
+{2100}{}{RESERVED}
+
+#
+# 
+#
+{2200}{}{RESERVED}
+
+#
+# 
+#
+{2300}{}{RESERVED}
+
+#
+# 
+#
+{2400}{}{RESERVED}
+
+#
+# 
+#
+{2500}{}{RESERVED}
+
+#
+# 
+#
+{2600}{}{RESERVED}
+
+#
+# Desert Encounter Quests
+#
+{2700}{}{RESERVED}
+
+#
+# Mountain Encounter Quests
+#
+{2800}{}{RESERVED}
+
+#
+# Ruined City Encounter Quests
+#
+{2900}{}{RESERVED}
+
+#
+# Coast Encounter Quests
+#
+{3000}{}{RESERVED}
+
+#
+# 
+#
+{3100}{}{RESERVED}
+
+#
+# 
+#
+{3200}{}{RESERVED}
+{3201}{}{RESERVED}
+{3202}{}{RESERVED}
+{3203}{}{RESERVED}
