diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/game/SCRNAME.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/game/SCRNAME.msg
index 24954968d..e33871675 100644
--- a/Fallout2/Fallout1in2/mods/fo1_base/text/english/game/SCRNAME.msg
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/game/SCRNAME.msg
@@ -958,7 +958,7 @@
 {1058}{}{Door}                                          #  RaidDoor.int
 {1059}{}{Damaged Force Field Emitter}                   #  GenEmit.int
 {1060}{}{Test}                                          #
-{1061}{}{Cheater Item}                                  #  Chitem.int
+{1061}{}{Dog}                                           #  Peasdog.int    ; Dog victim for random encounters.
 {1062}{}{Caravan Customer}                              #  carcust.int
 {1063}{}{Boxer}                                         #  jboxer.int
 {1064}{}{}                                              #  stapbox.int
