diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/game/TRAIT.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/game/TRAIT.msg
new file mode 100644
index 000000000..657afba03
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/game/TRAIT.msg
@@ -0,0 +1,41 @@
+#
+#   Messages from trait.c
+#
+#
+#   TRAIT NAMES
+#
+{100}{}{Fast Metabolism}
+{101}{}{Bruiser}
+{102}{}{Small Frame}
+{103}{}{One Hander}
+{104}{}{Finesse}
+{105}{}{Kamikaze}
+{106}{}{Heavy Handed}
+{107}{}{Fast Shot}
+{108}{}{Bloody Mess}
+{109}{}{Jinxed}
+{110}{}{Good Natured}
+{111}{}{Chem Reliant}
+{112}{}{Chem Resistant}
+{113}{}{Night Person}  # was: Sex Appeal
+{114}{}{Skilled}
+{115}{}{Gifted}
+#
+#   TRAIT DESCRIPTIONS
+#
+{200}{}{Your metabolic rate is twice normal.  This means that you are much less resistant to radiation and poison, but your body heals faster.}
+{201}{}{A little slower, but a little bigger.  You may not hit as often, but they will feel it when you do!  Your total action points are lowered, but your Strength is increased.}
+{202}{}{You are not quite as big as the other Vault-Dwellers, but that never slowed you down.  You can't carry as much, but you are more agile.}
+{203}{}{One of your hands is very dominant.  You excel with single-handed weapons, but two-handed weapons cause a problem.}
+{204}{}{Your attacks show a lot of finesse.  You don't do as much damage, but you cause more critical hits.}
+{205}{}{By not paying attention to any threats, you can act a lot faster in a turn.  This lowers your armor class to just what you are wearing, but you sequence much faster in a combat turn.}
+{206}{}{You swing harder, not better.  Your attacks are very brutal, but lack finesse.  You rarely cause a good critical, but you always do more melee damage.}
+{207}{}{You don't have time to aim for a targeted attack, because you attack faster than normal people.  It costs you one less action point to use a weapon.}
+{208}{}{By some strange twist of fate, people around you die violently.  You always see the worst way a person can die.}
+{209}{}{The good thing is that everyone around you has more critical failures in combat, the bad thing is so do you!}
+{210}{}{You studied less-combative skills as you were growing up.  Your combat skills start at a lower level, but First Aid, Doctor, Speech and Barter are substantially improved.}
+{211}{}{You are more easily addicted to chems.  Your chance to be addicted by chem use is twice normal, but you recover faster from their ill effects.}
+{212}{}{Chems only affect you half as long as normal, but your chance to be addicted is also only 50% of normal.}
+{213}{}{As a night-time person, you are more awake when the sun goes down.  Your Intelligence and Perception are improved at night, but dulled during the day.}
+{214}{}{Since you spent more time improving your skills than a normal person, you start with better skill levels.  The tradeoff is that you do not gain as many extra abilities.  You will gain a perk every four levels.}
+{215}{}{You have more innate abilities than most, so you have not spent as much time honing your skills.  Your primary statistics are each +1, but you lose -10% on all skills to start, and receive 5 less skill points per level.}
