diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/game/WORLDMAP.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/game/WORLDMAP.msg
index 1e437752f..382df7c07 100644
--- a/Fallout2/Fallout1in2/mods/fo1_base/text/english/game/WORLDMAP.msg
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/game/WORLDMAP.msg
@@ -159,7 +159,7 @@
 # Worldmap Encounter messages
 #
 # Original is "You encounter:"
-{2998}{}{Encounter begins. --- disabled by Sfall ---}
+{2998}{}{}
 # Original is "Do you wish to avoid:"
 {2999}{}{Encounter! Investigate?}
 
