diff --git a/Fallout2/Fallout1in2/mods/fo1_base/text/english/game/WORLDMAP.msg b/Fallout2/Fallout1in2/mods/fo1_base/text/english/game/WORLDMAP.msg
new file mode 100644
index 000000000..1e437752f
--- /dev/null
+++ b/Fallout2/Fallout1in2/mods/fo1_base/text/english/game/WORLDMAP.msg
@@ -0,0 +1,383 @@
+# Text labels and messages used by town and world map
+#
+#Darek regarding dream sequences:
+#
+# The intervals are 23 Oct 2241, 21 Jan 2242, 21 Apr 2242 and 20 Jul 2242 For Fallout 2, or adjusted for Fallout 1 the first one comes 05 Mars 2162, and you can just count the rest from there.
+#
+#If done on exit in map script of first map the movie will be played upon leaving map to worlmap, and will be hardly noticed (just a quick fade out).
+#Should also add a "" in the map script for the new location, or the world map will display "unknown" and the fast travel button will be missing..
+#
+#
+
+# Town map entry-point selection names
+# Format:
+#	city type.
+#	Up to 10 selection names
+#
+# Vault 13
+{200}{}{Cavern Entrance}
+{201}{}{Entrance}
+{202}{}{Living Quarters}
+{203}{}{Command Center}
+#
+# Vault 15
+{210}{}{Desert Entrance}
+{211}{}{Caverns/Entrance}
+{212}{}{Living Quarters}
+{213}{}{Command Center}
+#
+# Shady Sands
+{220}{}{Entrance}
+{221}{}{Garden}
+{222}{}{Brahmin Pen}
+#
+# Junktown
+{230}{}{Entrance}
+{231}{}{Crash House}
+{232}{}{Casino}
+#
+# Raiders
+{240}{}{Base}
+#
+# Necropolis
+{250}{}{Hotel}
+{251}{}{Hall}
+{252}{}{Watershed}
+#
+# Hub
+{260}{}{Entrance}
+{261}{}{Downtown}
+{262}{}{Heights}
+{263}{}{Old Town}
+{264}{}{Merchants}
+{265}{}{Cave}
+{266}{}{Irwin's Farm}
+#
+# Brotherhood
+{270}{}{Entrance}
+{271}{}{Level 1}
+{272}{}{Level 2}
+{273}{}{Level 3}
+{274}{}{Level 4}
+#
+# Military Base
+{280}{}{Entrance}
+{281}{}{Stronghold}
+{282}{}{Vats}
+{283}{}{Ruins}
+#
+# Glow
+{290}{}{Crater}
+{291}{}{Level 1}
+#
+# Boneyard
+{300}{}{Adytum}
+{301}{}{Downtown}
+{302}{}{Library}
+{303}{}{Fortress}
+{304}{}{Warehouse}
+#
+# Cathedral
+{310}{}{Entrance}
+{311}{}{Interior}
+{312}{}{CHILDRN2}
+{313}{}{MSTRLR12}
+{314}{}{MSTRLR34}
+{315}{}{Crater}
+
+# Cathedral Crater
+{320}{}{Crater}
+
+# Military Base Ruins
+{330}{}{Ruins}
+
+# Vipers
+{340}{}{Camp}
+
+#
+# Car Out of Gas
+{410}{}{Motorcycle out of gas}
+
+# Misc messages
+{500}{}{You gained 10,000 experience points!}
+{501}{}{Ruins}
+
+# Random Encounter Desert - triggered by player
+{680}{}{Desert}
+# Random Encounter Mountain - triggered by player
+{690}{}{Mountain}
+# Random Encounter City - triggered by player
+{700}{}{City}
+# Random Encounter Coast - triggered by player
+{710}{}{Coast}
+
+# Special Encounter FOOT
+{720}{}{Desert}
+# Special Encounter TARDIS
+{730}{}{Desert}
+# Special Encounter USEDCAR
+{740}{}{Desert}
+# Special Encounter COLATRUK
+{750}{}{Desert}
+# Special Encounter FSAUSER
+{760}{}{Desert}
+# Special Encounter TALKCOW
+{770}{}{Desert}
+# Special Encounter Guardian of Forever
+{780}{}{Guardian of Forever}
+
+# Scrapheap
+{790}{}{Scrapheap}
+
+# Copy of Caves (Shady sands), CAVES.MAP - used for optional location circle feature
+{800}{}{Caves}
+# Copy of DeathClaw's Lair, DETHCLAW.MAP - used for optional location circle feature
+{810}{}{Irwin's Farm}
+# Copy of Irwine's Farm, HUBMIS0.MAP - used for optional location circle feature
+{820}{}{Cave}
+
+#
+# Terrain type names for world map hover button
+#
+{1000}{}{Desert}
+{1001}{}{Mountains}
+{1002}{}{City}
+{1003}{}{Coast}
+{1004}{}{Unknown}  # Unknown/visited city
+{1005}{}{Radiated Mountains}
+{1006}{}{Radiated Desert}
+
+# Misc messages
+{1500}{}{You gained 10,000 experience points!}
+{1501}{}{Ruins}
+{1502}{}{The car is out of power.}
+{1503}{}{This is a car.  It doesn't look like it's running.}
+
+
+
+#
+# Worldmap Encounter messages
+#
+# Original is "You encounter:"
+{2998}{}{Encounter begins. --- disabled by Sfall ---}
+# Original is "Do you wish to avoid:"
+{2999}{}{Encounter! Investigate?}
+
+# Encounter Table 0
+{3000}{}{unused}
+
+# Encounter Table 1
+{3050}{}{unused}
+
+# Encounter Table 2
+{3100}{}{unused}
+
+# Encounter Table 3
+{3150}{}{unused}
+
+# Encounter Table 4
+{3200}{}{unused}
+
+# Encounter Table 5
+{3250}{}{unused}
+
+# Encounter Table 6
+{3300}{}{unused}
+
+# Encounter Table 7
+{3350}{}{unused}
+
+# Encounter Table 8
+{3400}{}{unused}
+
+# Encounter Table 9
+{3450}{}{unused}
+
+# Encounter Table 10
+{3500}{}{unused}
+
+# Encounter Table 11
+{3550}{}{unused}
+
+# Encounter Table 12
+{3600}{}{unused}
+
+# Encounter Table 13
+{3650}{}{unused}
+
+# Encounter Table 14
+{3700}{}{unused}
+
+# Encounter Table 15
+{3750}{}{unused}
+
+# Encounter Table 16
+{3800}{}{unused}
+
+# Encounter Table 17
+{3850}{}{unused}
+
+# Encounter Table 18
+{3900}{}{unused}
+
+# NOTE: Empty brackets are special encounters. These need to be empty!
+# Encounter Table 19 (Fo1in2: Encounter Table 0)
+{3950}{}{ }
+{3951}{}{ }
+{3952}{}{ }
+{3953}{}{ }
+{3954}{}{ }
+{3955}{}{ }
+{3956}{}{ }
+{3957}{}{ }
+
+# Encounter Table 20 (Fo1in2: Encounter Table 1)
+{4000}{}{ }
+{4001}{}{ }
+{4002}{}{ }
+{4003}{}{ }
+{4004}{}{ }
+{4005}{}{ }
+{4006}{}{ }
+{4007}{}{ }
+
+# Encounter Table 21 (Fo1in2: Encounter Table 2)
+{4050}{}{ }
+{4051}{}{ }
+{4052}{}{ }
+{4053}{}{ }
+{4054}{}{ }
+{4055}{}{ }
+{4056}{}{ }
+{4057}{}{ }
+
+# Encounter Table 22 (Fo1in2: Encounter Table 3)
+{4100}{}{ }
+{4101}{}{ }
+{4102}{}{ }
+{4103}{}{ }
+{4104}{}{ }
+{4105}{}{ }
+{4106}{}{ }
+{4107}{}{ }
+{4108}{}{ }
+
+# Encounter Table 23 (Fo1in2: Encounter Table 4)
+{4150}{}{ }
+{4151}{}{ }
+{4152}{}{ }
+{4153}{}{ }
+{4154}{}{ }
+{4155}{}{ }
+{4156}{}{ }
+{4157}{}{ }
+
+# Encounter Table 24 (Fo1in2: Encounter Table 5)
+{4200}{}{ }
+{4201}{}{ }
+{4202}{}{ }
+{4203}{}{ }
+{4204}{}{ }
+{4205}{}{ }
+{4206}{}{ }
+{4207}{}{ }
+
+# Encounter Table 25 (Fo1in2: Encounter Table 6)
+{4250}{}{ }
+{4251}{}{ }
+{4252}{}{ }
+{4253}{}{ }
+{4254}{}{ }
+{4255}{}{ }
+{4256}{}{ }
+{4257}{}{ }
+
+# Encounter Table 26 (Fo1in2: Encounter Table 7)
+{4300}{}{ }
+{4301}{}{ }
+{4302}{}{ }
+{4303}{}{ }
+{4304}{}{ }
+{4305}{}{ }
+{4306}{}{ }
+{4307}{}{ }
+
+# Encounter Table 27 (Fo1in2: Encounter Table 8)
+{4350}{}{ }
+{4351}{}{ }
+{4352}{}{ }
+{4353}{}{ }
+{4354}{}{ }
+{4355}{}{ }
+{4356}{}{ }
+{4357}{}{ }
+
+# Encounter Table 28 (Fo1in2: Encounter Table 9)
+{4400}{}{ }
+{4401}{}{ }
+{4402}{}{ }
+{4403}{}{ }
+{4404}{}{ }
+{4405}{}{ }
+{4406}{}{ }
+{4407}{}{ }
+
+# Encounter Table 29 (Fo1in2: Encounter Table 10)
+{4450}{}{ }
+{4451}{}{ }
+{4452}{}{ }
+{4453}{}{ }
+{4454}{}{ }
+{4455}{}{ }
+{4456}{}{ }
+{4457}{}{ }
+
+# Encounter Table 30 (Fo1in2: Encounter Table 11)
+{4500}{}{ }
+{4501}{}{ }
+{4502}{}{ }
+{4503}{}{ }
+{4504}{}{ }
+{4505}{}{ }
+{4506}{}{ }
+{4507}{}{ }
+
+# Encounter Table 31 (Fo1in2: Encounter Table 12)
+{4550}{}{ }
+{4551}{}{ }
+{4552}{}{ }
+{4553}{}{ }
+{4554}{}{ }
+{4555}{}{ }
+{4556}{}{ }
+{4557}{}{ }
+
+# Encounter Table 32 (Fo1in2: Encounter Table 13)
+{4600}{}{ }
+{4601}{}{ }
+{4602}{}{ }
+{4603}{}{ }
+{4604}{}{ }
+{4605}{}{ }
+{4606}{}{ }
+{4607}{}{ }
+
+# Encounter Table 33 (Fo1in2: Encounter Table 14)
+{4650}{}{ }
+{4651}{}{ }
+{4652}{}{ }
+{4653}{}{ }
+{4654}{}{ }
+{4655}{}{ }
+{4656}{}{ }
+{4657}{}{ }
+
+# Encounter Table 34 (Fo1in2: Encounter Table 15)
+{4700}{}{ }
+{4701}{}{ }
+{4702}{}{ }
+{4703}{}{ }
+{4704}{}{ }
+{4705}{}{ }
+{4706}{}{ }
+{4707}{}{ }
